{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 1.Read Input"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [],
   "source": [
    "import textacy\n",
    "\n",
    "training_records = textacy.io.read_json(\"datasets/five_training_data.json\", lines=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 2. List of Text and Labels"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [],
   "source": [
    "X_train = []\n",
    "y_train = []\n",
    "\n",
    "for training_record in training_records:\n",
    "    \n",
    "    X_train.append(training_record[\"text\"])\n",
    "    y_train.append(training_record[\"class\"])\n",
    "\n",
    "#print(X_train)\n",
    "#print(y_train)\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "#### 2.1 Tokening data with spaCy"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [],
   "source": [
    "import string\n",
    "import spacy\n",
    "from spacy.lang.en.stop_words import STOP_WORDS\n",
    "from spacy.lang.en import English\n",
    "\n",
    "punctuations = string.punctuation\n",
    "\n",
    "nlp = spacy.load('en')\n",
    "stop_words = spacy.lang.en.stop_words.STOP_WORDS\n",
    "\n",
    "parser = English()\n",
    "\n",
    "def spacy_tokenizer(sentence):\n",
    "    mytokens = parser(sentence)\n",
    "    mytokens = [ word.lemma_.lower().strip() if word.lemma_ != \"-PRON-\" else word.lower_ for word in mytokens ]\n",
    "    mytokens = [ word for word in mytokens if word not in stop_words and word not in punctuations ]\n",
    "    \n",
    "    return mytokens"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 3. transforms(): Extract Noun Phrases from Text"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Custom transformer function"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "metadata": {},
   "outputs": [],
   "source": [
    "from sklearn.base import TransformerMixin\n",
    "\n",
    "class transforms(TransformerMixin):\n",
    "    def transform(self, X, **transform_params):\n",
    "        # Cleaning Text\n",
    "        #for text in X:\n",
    "            # print(\"TEXT________________________________________\")\n",
    "            # print(text)\n",
    "        return [transform_text(text) for text in X]\n",
    "    \n",
    "    def fit(self, X, y=None, **fit_params):\n",
    "        return self\n",
    "    \n",
    "    def get_params(self, deep=True):\n",
    "        return {}\n",
    "\n",
    "def transform_text(text):\n",
    "    doc = textacy.make_spacy_doc((str(text), {\"class\": \"\"}))\n",
    "    nps = textacy.extract.noun_chunks(doc,\n",
    "                                      drop_determiners = True,\n",
    "                                      min_freq = 1)\n",
    "    doc_nps = [str(np).replace(' ','_') for np in nps]\n",
    "    nps = ' '.join(doc_nps) # because countvectorizer needs a string\n",
    "    return (nps)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 4.Vectorizer (Bag of Words / TfIdf)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "metadata": {},
   "outputs": [],
   "source": [
    "from sklearn.feature_extraction.text import CountVectorizer, TfidfVectorizer\n",
    "\n",
    "bow_vector = CountVectorizer(tokenizer = spacy_tokenizer, ngram_range=(1,1))\n",
    "tfidf_vector = TfidfVectorizer(tokenizer = spacy_tokenizer)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 5. Classifier for Training (Logistic Regression)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [],
   "source": [
    "from sklearn.linear_model import LogisticRegression\n",
    "classifier = LogisticRegression(solver='newton-cg',multi_class='multinomial')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 6. Training Pipeline"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "Pipeline(memory=None,\n",
       "     steps=[('transformer', <__main__.transforms object at 0x7f6b0eb29860>), ('vectorizer', CountVectorizer(analyzer='word', binary=False, decode_error='strict',\n",
       "        dtype=<class 'numpy.int64'>, encoding='utf-8', input='content',\n",
       "        lowercase=True, max_df=1.0, max_features=None, min_df=1,\n",
       "      ...ty='l2', random_state=None, solver='newton-cg',\n",
       "          tol=0.0001, verbose=0, warm_start=False))])"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "from sklearn.feature_extraction.text import CountVectorizer, TfidfVectorizer\n",
    "from sklearn.base import TransformerMixin\n",
    "from sklearn.pipeline import Pipeline\n",
    "\n",
    "transforms()\n",
    "\n",
    "pipe = Pipeline([(\"transformer\", transforms()),\n",
    "                ('vectorizer', bow_vector),\n",
    "                ('classifier', classifier)])\n",
    "\n",
    "pipe.fit(X_train,y_train)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "['pipeUSECASE.joblib']"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "from joblib import dump\n",
    "\n",
    "dump(pipe, 'pipeUSECASE.joblib')"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## 7. Classification"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "Read Test data"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "metadata": {},
   "outputs": [],
   "source": [
    "test_records = textacy.io.read_json(\"datasets/five_test_data.json\", lines=True)\n",
    "\n",
    "X_test = []\n",
    "y_test = []\n",
    "\n",
    "for test_record in test_records:\n",
    "    \n",
    "    X_test.append(test_record[\"text\"])\n",
    "    y_test.append(test_record[\"class\"])\n",
    "\n",
    "#print(X_test)\n",
    "#print(y_test)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "results:\n",
      " Abstract Background  Non-small cell lung cancer (NSCLC) is a heterogeneous group of disorders with a number of genetic and proteomic alterations. c-CBL is an E3 ubiquitin ligase and adaptor molecule important in normal homeostasis and cancer. We determined the genetic variations of c-CBL, relationship to receptor tyrosine kinases (EGFR and MET), and functionality in NSCLC.  Methods and Findings  Using archival formalin-fixed paraffin embedded (FFPE) extracted genomic DNA, we show that c-CBL mutations occur in somatic fashion for lung cancers. c-CBL mutations were not mutually exclusive of MET or EGFR mutations; however they were independent of p53 and KRAS mutations. In normal/tumor pairwise analysis, there was significant loss of heterozygosity (LOH) for the c-CBL locus (22%, n = 8/37) and none of these samples revealed any mutation in the remaining copy of c-CBL. The c-CBL LOH also positively correlated with EGFR and MET mutations observed in the same samples. Using select c-CBL somatic mutations such as S80N/H94Y, Q249E and W802* (obtained from Caucasian, Taiwanese and African-American samples, respectively) transfected in NSCLC cell lines, there was increased cell viability and cell motility.  Conclusions  Taking the overall mutation rate of c-CBL to be a combination as somatic missense mutation and LOH, it is clear that c-CBL is highly mutated in lung cancers and may play an essential role in lung tumorigenesis and metastasis.  Go to: Introduction In the US alone, each year approximately 219,400 people are diagnosed with lung cancers, out of which more than 145,000 of them succumb to the disease [1]. This number is roughly equivalent to the combined mortality rates of cancers of the breast, prostate, colon, liver, kidney and melanoma [1]. In addition the prognosis is usually poor and the five-year survival rate is less than 15%. There are also significant ethnic differences for lung cancer, and the outcome is worse for blacks compared to whites. Gender differences are also striking with women having significantly better prognosis as compared to men. There are a number of genetic alterations that can occur in lung cancer. As an example, in NSCLC, mutations in KRAS, p53, EGFR and MET have been identified. Many of these pathways, especially Receptor Tyrosine Kinases (RTKs) are controlled by c-CBL.  CBL (Casitas B-lineage lymphoma) is a mammalian gene located on human chromosome 11q23.3 [2] and is involved in cell signaling and protein ubiquitination [3]. CBL proteins belong to the RING finger class of ubiquitin ligases (E3) and there are three homologues c-CBL, CBL-b, CBL-3 [4]. The c-CBL and CBL-b genes are ubiquitously expressed with the highest levels in hematopoietic tissues [5]. c-CBL consists of four regions encoding for functionally distinct protein domains: the N-terminal tyrosine kinase binding (TKB) domain, the linker region, the catalytic RING finger domain, the proline-rich region and the c-terminal ubiquitin-associated (UBA) domain that also overlaps with a leucine-zipper (LZ) domain [3]. Both TKB and RING finger domains are essential for ligand-induced ubiquitination of RTKs [6], [7], [8], [9]. The RING finger domain is required for the recruitment of E2 ubiquitin-conjugating enzymes. The TKB domain includes four-helix bundle (4H), a calcium-biding EF hand, and a modified SH2 domain, which binds to phosphotyrosine residues [3], [10], [11], [12]. In addition, the proline-rich region of c-CBL can associate with the SH3 domain of Grb2, which can indirectly recruit c-CBL to RTKs via the GRB2 adaptor protein [7], [13], [14].  c-CBL also binds to EGFR and acts as the E3 that targets EGFR for ubiquitination and degradation. Furthermore, CBL desensitizes EGF signaling and opposes cellular proliferation induced by EGF [15]. EGF activation also appears to activate the tyrosine kinase SRC, which phosphorylates c-CBL and in turn activates the ubiquitination and degradation of EGFR [16], [17], [18]. A recent study shows that defective endocytosis of EGFR is characterized by a deletion mutant and the point mutation L858R, whereby its association with c-CBL and subsequent ubiquitination are impaired [19]. Recently, the first human c-CBL mutations were reported in acute myeloid leukemia (AML) patients [20]. The mutation R420Q inhibits FMS-like tyrosine kinase 3 (FLT3) internalization and ubiquitination [20].  Not only can E3 activity be important in oncogenesis, c-CBL has a dual but separate function as a signal transduction molecule. We have previously shown that c-CBL is important in binding CRKL and BCR/ABL in hematopoietic cells. Also, it can bind and modulate functions of cytoskeleton by binding to proteins like talin and paxillin. The TKB domain is important in binding to a number of molecules, and they then function in signal transduction.  Given the critical role of CBL in normal homeostasis and cancer, we hypothesized that it might be mutated in lung cancers. In this study, we report novel c-CBL somatic mutations S80N/H94Y, Q249E and W802* in Caucasian, Taiwanese and African-American lung cancer patients, respectively. Expressing these mutations in NSCLC cell lines lead to increased proliferation and cell motility. We show that c-CBL mutations occur with or without MET or EGFR mutations but are mutually exclusive of a LOH at the c-CBL locus. Additionally, c-CBL LOH is associated with either MET or EGFR mutations. We thus hypothesize that c-CBL mutations might contribute to the oncogenic potential of MET and EGFR in lung cancer.  Go to: Methods Ethics Statement  Written consent on all research on human subjects has been obtained from the Institutional Review Board, University of Chicago and covers all research performed in the laboratory. The following is their contact information: Institutional Review Board, The University of Chicago, McGiffert Hall, 5751 S. Woodlawn Ave., 2nd floor, Chicago, IL 60637. Written informed consents were received from all patients whose tissue samples were used for this study.  Tissue Samples  Lung cancer tissue and paired adjacent normal lung tissues were obtained from 50 Caucasian, 29 African-Americans and 40 Taiwanese NSCLC patients who were recruited at the University of Chicago Hospital (Chicago, USA) (Caucasian and African-American patients) and Taipei Veterans General Hospital of Taiwan (Taiwanese patients) after obtaining appropriate Institutional Review Board permission and informed consent from the patients. Out of 119 samples, 77 were men, 38 were women and 4 were unknown with age at diagnosis ranging from 47 to 90 years. In terms of tumor types, 53 were adenocarcinoma, 32 were squamous cell carcinoma and 34 were large cell carcinoma. 49 were stage I, 14 were stage II, 34 were stage III, and 13 were stage IV (Table S1).  Cell Culture  Human non-small cell lung carcinoma cells A549 and H358 were maintained in DMEM and RPMI-1640, respectively. Human embryonic kidney 293T cells were cultured in DMEM. Media were supplemented with 10% fetal bovine serum, 100 units/ml of penicillin, and 100 µg/ml of streptomycin (Invitrogen, Carlsbad, CA). Cells were cultured at 37°C in a humidified incubator containing 5% CO2.  c-CBL Gene Mutational Analysis  Exons 2 to 16 of c-CBL gene were individually amplified by polymerase chain reaction (PCR). Primers are listed in Table S2. PCR conditions were 1 cycle of 95°C for 5 minutes; 35 cycles of 94°C for 30 seconds, 58°C for 30 seconds and 72°C for 2 minutes; and one cycle of 72°C for 10 minutes. PCR products were treated with ExoSAP-IT (USB Corporation, Cleveland, OH) and sequenced by Big-Dye Terminator Chemistry (Applied Biosystems, Foster City, CA). Sequencing was performed on the forward coding strand with confirmation of c-CBL alterations performed by sequencing the reverse strand as well. Chromatograms were analyzed for mutations using Mutation Surveyor v2.61 (Softgenetics, State College, PA).  Plasmid Constructs and Site-Directed Mutagenesis  The wild-type c-CBL cDNA insert was subcloned into the pAlterMax expression vector using XhoI and SalI restriction enzyme sites (Promega, Madison, WI). Using this parental plasmid pAlterMax-c-CBL, the TKB domain double mutation (S80N/H94Y), the point mutation (Q249E), and the C-terminal point mutation W802* of c-CBL were created using the following primers: 5′-GCTGGCGCTAAAGAATAACCCACCTTATATCTTAGAC-3′ and 5′-CTACCAGATACCTACCAGTATCTCCGTACTATCTTGTC-3′ for the double mutation S80N/H94Y; 5′-CTTTACCCGACTCTTTGAGCCCTGGTCCTCTTTGC-3′ for Q249E, and 5′-CAGCTCCTCCTTTGGCTGATTGTCTCTGGATGGTGATC-3′ for W802* along with their complementary primers using the QuickChange Site-Directed Mutagenesis XL kit (Stratagene, La Jolla, CA) according to the manufacturer's instructions. The constructs were confirmed for the point mutations by standard DNA sequencing of both strands.  Loss of Heterozygosity (LOH) Analysis  Five microsatellites on chromosome 11 (3 on 11q at or within 200 kb up or downstream of the c-CBL gene and 2 control markers on 11p) were selected for analysis (Table S3). Established microsatellite markers and respective primer sequences were selected from the GeneLoc database (http://genecards.weizmann.ac.il/geneloc/index.shtml, Weizmann Institute of Science, Rehovot, Israel). Primers were custom designed and each forward primer was fluorescently labeled at the 5′ end with FAM, PET, NED, or VIC (Applied Biosystems). Primer annealing temperatures and duplex scores were evaluated with NIST Primer Tools (http://yellow.nist.gov:8444/dnaAnalysis/primerToolsPage.do; National Institute of Standards and Technology, Gaithersburg, MD). Primers were verified by performing PCR with control DNA (isolated from TK6 cells) and resolving the products on agarose gels. Bands were visualized with an UV transilluminator. Genomic DNA was extracted from tumor samples and paired normal lung tissue. Primers were grouped into multiplex combinations shown in Table S4. Marker D11S929 served as an internal control to check for consistency in PCRs and of peaks from capillary electrophoresis. Multiplex PCRs were carried out in a volume of 10 µL that contained 1 µL genomic DNA (20–50 ng), 0.5 µM of each primer (1.0 µM total for each primer pair), 400 µM dNTPs, 1X PCR buffer containing MgCl2, and 0.2 U Taq DNA polymerase. PCR was performed on the ABI GeneAmp 9700 PCR System under the following conditions: 5 min at 94°C; 30 cycles of 30 sec at 94°C, 1 min at 60°C, 1 min at 72°C; and 5 min at 72°C. The PCR products were separated by capillary electrophoresis on an ABI 3130XL DNA Analyzer. Chromatograms were analyzed with Peak Scanner 1.0 and GeneMapper 3.7 software (Applied Biosystems) for allelic alterations. The area of the peaks produced by the DNA PCR products was quantified for each allele. The ratio of the allelic areas was calculated for each tumor and paired normal DNA sample. When the qLOH (allelic ratio for the tumor peaks divided by the allelic ratio of paired normal sample) was ≤0.5 or ≥2.0 for c-CBL and at least one other 11q marker in at least two separate experiments, the sample was considered as having an allelic imbalance and interpreted as LOH. Samples were evaluated in at least two separate experiments and samples showing prospective LOH at c-CBL repeated a third time which included a new control marker at the BAX locus (data not shown) on chromosome 19 to verify integrity of sample DNA.  Transfection of c-Cbl Constructs  The A549 cell line was transfected using the Fugene HD (Roche, Nutley, NJ) reagent according to the manufacturer's instructions. Eight µg of plasmid DNA, containing either no insert (empty vector), wild-type c-CBL, S80N/H94Y c-CBL, Q249E c-CBL or W802* CBL was used for transfection in a 6-well culture plate. Cells were harvested 48 h after transfection and analyzed for expression.  c-CBL Knockdown  c-CBL knockdown was performed using lentiviral transduction using MISSION lentiviral transduction particles (Sigma-Aldrich, St. Louis, MO) as per manufacturer's instructions. Briefly, 1×105 H358 cells/well were seeded in 6-well plates and infected the following day with c-CBL lentiviral shRNA constructs. To generate stable c-CBL knockdown cell lines, cells were selected for 2 days with 1 µg/ml puromycin. c-CBL levels were determined using whole cell lysates by immunoblotting with anti-CBL antibody (Santa Cruz Biotechnologies, Santa Cruz, CA).  Cell Viability Assay  Cells were transfected as described above in the transfection assay. Forty-eight hours after transfection, viability of cells was assessed using Trypan Blue exclusion.  Wound Healing Assay  A549 cells were seeded in 6-well plates and cultured for 48 h until 100% confluent. The medium was then changed and the cells were transfected as described in the transfection assay. Twelve hours after transfection, a straight scratch was made across the cell layer using a 1 ml pipette tip. The cells were then gently washed with 1× PBS to remove cellular debris and the media was replaced. Photographs were taken of the wound region every 12 h until 48 h.  Western Blot Analysis  Forty eight hours after transfection, cells were collected and washed twice in 1X PBS, then lysed in ice-cold lysis buffer (0.5M Tris-HCl with pH 7.4, 1.5 M NaCl, 2.5% deoxycholic acid, 10 mM EDTA, 10% NP-40, 0.5 mM DTT, 1 mM phenylmethylsulfonyl fluoride, 5 µg/mL leupeptin, and 10 µg/mL aprotinin) for 5 minutes. The lysate was centrifuged at 13,000 rpm for 20 minutes at 4°C, and protein content of the supernatant was measured. Total cell lysates (50 µg/well) were separated by SDS-PAGE electrophoresis and the gels transferred onto nitrocellulose membranes (Whatman, Piscataway, NJ). Membranes were blocked with 5% non-fat dry milk in phosphate-buffered saline containing Tween-20 (PBST) (1X PBS, 0.1% Tween-20) for 1 h at room temperature and incubated with the appropriate primary antibody at 4°C overnight. Membranes then were washed three times with PBST and probed with appropriate horseradish peroxidase (HRP)-conjugated secondary antibody for 1 h at room temperature. The membranes were again washed three times in PBST and bands were visualized using Western blot chemiluminescence reagent (BioRad, Valencia, CA) on a Chemidoc Gel documentation system (BioRad, Valencia, CA). Antibodies were obtained from Santa Cruz Biotechnologies and used at the following dilutions (c-CBL, 1∶5000; c-MET, 1∶5000; EGFR, 1∶5000; ubiquitin, 1∶1000; HA, 1∶5000 and β-actin, 1∶10,000).  Flow Cytometry  Cell cycle analysis was carried out by flow cytometry. Approximately 2×106 cells were grown in media containing 10% FBS. Cells were harvested by trypsin/EDTA treatment, washed with 1X PBS three times and fixed with ice-cold 70% ethanol for 2 h. Cells were washed again with cold PBS and stained with a solution containing 25 µg/mL propidium iodide, 200 µg/mL RNase A, and 0.1% Triton X-100 for 30 minutes in the dark. Cell cycle analysis was performed using a Guava PCA-96 flow cytometer (Guava Technologies, Millipore, Billerica, MA).  Ubiquitin Ligase Activity  293T cells were maintained in culture in DMEM supplemented with 10% FBS and 1% penicillin (100 units/mL) and streptomycin (100 µg/mL) were transfected with 0.2 µg EGFR-pcDNA3 and 2 µg HA-tagged c-CBL constructs as indicated using calcium phosphate according to manufacturer's protocol (Profection, Promega, Madison, WI). Twenty-four hours post-transfection, cells were starved overnight in DMEM supplemented with 0.5% FBS, and then treated with or without EGF (100 ng/ml) for 15 min. The cells were collected and washed two times in ice-cold PBS containing 0.2 mM sodium orthovanadate then lysed in ice-cold lysis buffer (10 mM Tris HCl, pH 7.5, 150 mM NaCl, 5 mM EDTA, 1% Triton X100, 10% Glycerol, 2 mM sodium orthovanadate and protease inhibitors). Lysates were cleared of debris by centrifugation at 16,000 g for 10 min at 4°C. EGFR immunoprecipitations were performed on 200 µg of cleared lysate using 250 ng of rabbit-anti-EGFR and Protein A/G Plus Sepharose overnight at 4°C. Precipitations were washed 5 times in lysis buffer before boiling in Laemmli buffer. Elutions were immunoblotted with anti-ubiquitin and EGFR. Twenty micrograms of cleared lysate were immunoblotted for each of the c-CBL constructs using anti-HA.  Statistical Analysis  Mutation rates between different groups were compared using Fisher's exact test. For continuous variables, group comparisons were performed using analysis of variance (ANOVA) followed by Sidak's adjustment for multiple comparisons. Experiments involving repeated measurements over time were analyzed using repeated measures ANOVA with the Greenhouse-Geisser adjustment to the degrees of freedom. Analyses were conducted using STATA (v10.1) software (Stata Corporation, College Station, TX).  Go to: Results c-CBL Gene Mutations in Lung Cancer  To investigate the role of c-CBL in lung cancer, we analyzed its genomic DNA in tumor and paired normal samples drawn from multiple ethnicities. The lung tumor samples represented Caucasians (n = 50), African-Americans (n = 29), and Taiwanese (n = 40) lung cancer patients. We designed 12 pairs of primers to sequence the coding region of c-CBL gene that spans exons 2 to 16 (Table S2). We identified 8 unique somatic mutations in c-CBL exons among 8 different patients. A variation L620F, a known SNP (rs2227988) in exon 11 was also detected. Importantly, the eight novel non-synonymous mutations were confirmed by sequencing both strands of c-CBL genomic DNA obtained from lung tumor samples (Table 1). Moreover, none of the 8 mutations were detected in the corresponding normal tissue, indicating that these were somatic mutations. Four synonymous single nucleotide variations (SNVs) were also identified but were not used further in this study.  Table 1 Table 1 c-CBL mutation analysis in 119 lung cancer patient tumor tissues. Three of the 8 novel non-synonymous mutations were located in the TKB (tyrosine kinase binding) domain (S80N, H94Y, and Q249E), one in the RING finger domain (V391I), one in the proline-rich region (72515_72517 del ATG), and three in the C-terminal region (W802*, R830K, and A848T) of the c-CBL protein (Figure 1A and Figure S1). In Figure 1B, we show model chromatograms of representative samples.  Figure 1 Figure 1 c-CBL mutations and LOH in non-small cell lung cancer. 11q LOH of c-CBL Gene  Paired lung tumor and normal lung tissue samples from Taiwanese patients (n = 37) were investigated for LOH. Eight (21.6%) showed LOH at the c-CBL locus on chromosome 11 while 29 samples (78.4%) revealed normal allelic contribution at the microsatellite markers (Figures 1C and D).  c-CBL Mutations in Different Ethnic Groups  The c-CBL double mutant S80N/H94Y was found in the same patient and the overall mutation rate for c-CBL in lung tumors was 6.7% (8/119). The frequency of c-CBL mutation was highest in large cell carcinoma (14.7%; 5 of 34 patients) followed by squamous carcinoma (6.3%; 2 of 32 patients) and the least was observed in adenocarcinoma (AD) (1.8%; 1 of 53 patients), although these rates were not statistically significant (p = 0.292). Mutation rates were 6.0% among Caucasians (0 of 20 in AD; 0 of 10 in SQ; and 3 of 20 in LC), 13.8% in African-Americans (1 of 10 in AD; 1 of 10 in SQ; and 2 of 9 in LC), and 2.5% (0 of 23 in AD; 1 of 12 in SQ; and 0 of 5 in LC) in the Taiwanese population. Additionally two Taiwanese patients with lung cancer (one squamous and one adenocarcinoma) had the known SNP L620F. Ethnic differences were not statistically significant, however the power to detect differences was low.  Mutations in MET and EGFR Can Be Co-Associated with c-CBL Alterations  Since East Asians with lung cancer have a higher frequency of EGFR and MET mutations in lung tumors [21], [22], we also determined mutations in EGFR and MET in the same Taiwanese cohort samples and compared the results with the observed c-CBL alterations (LOH and/or mutations). In the 37 samples tested, we did not find any overlap between c-CBL mutations and c-CBL LOH (Figure 2). Of the three c-CBL mutants (including the known L620F SNP, rs2227988), one of the samples had a MET mutation (N375S) and the other had an EGFR mutation (L858R). Among the 8 samples that had a LOH at the c-CBL locus 5 had an additional mutation in MET (N375S) and 2 had an EGFR exon 19 deletion. Twenty-six samples had neither c-CBL mutation nor c-CBL LOH (3 patients had a c-CBL mutation but no c-CBL LOH). Among these 26 samples 9 had a MET mutation (8 N375S, 1 L211W), 13 had an EGFR mutation (7 exon 9 deletion, 6 L858R) and 4 had no other MET or EGFR mutation. Thus the rate of MET or EGFR mutations among patients with LOH at the c-CBL locus (7 of 8) was similar to that seen in patients without c-CBL mutation or LOH (22 of 26 patients) (p = 0.99). These 4 patients with no identifiable mutation in c-CBL, MET or EGFR represented 10.8% of the 37 patients analyzed in the Taiwanese patient cohort. Conversely, 89.2% Taiwanese lung cancer patients have an identifiable mutation in either c-CBL, MET or EGFR or a combination of the three genes (Figure 2). Additionally, we determined p53 and KRAS mutations in these Taiwanese cohorts. Two p53 and 1 KRAS mutation were detected. The single KRAS mutation overlapped with one p53 mutation. This patient also had the EGFR exon 19 deletion but had no c-CBL mutation. The other p53 mutation sample had a c-CBL LOH with concurrent MET N375S mutation. Thus, in the Taiwanese samples analyzed, p53/KRAS mutations and c-CBL mutations were mutually exclusive (data not shown).  Figure 2 Figure 2 c-CBL mutations and relationship to MET and EGFR mutations in lung cancer. Cellular Functions of c-CBL Alterations in the Context of Lung Tumorigenesis  A. E3 activity is intact in the mutant c-CBL proteins To investigate whether the different c-CBL mutations affect the E3 activity, EGFR was chosen as a model substrate for c-CBL E3 function. All of the c-CBL mutants tested enhanced ubiquitination of the activated EGFR similar to the wild-type c-CBL protein. This result demonstrates that the catalytic activity of the c-CBL mutants is not impaired when EGFR was the substrate. (Figure 3A).  Figure 3 Figure 3 Ubiquitination, viability, expression and cell cycle analysis of various c-CBL mutants. B. Effect on lung cancer cell viability The effect of a representative c-CBL mutant from each of the three ethnic backgrounds on lung cancer cell viability in cell lines was determined. S80N/H94Y double mutation, Q249E, and W802* were identified in lung tumor samples obtained from a Caucasian, a Taiwanese and an African-American, respectively. As described in methods, the c-CBL wild-type (WT) and the above three mutants were expressed after cloning them into pAlterMax vector in A549 cells. These cells express relatively low basal levels of endogenous c-CBL (data not shown). Transfection efficiency was comparable between different groups and the number of cells transfected with c-CBL wild-type construct was about 70% compared to control cells that were transfected with the empty vector. Cells transfected with S80N/H94Y, Q249E and W802* c-CBL mutant constructs resulted in increased number of viable cells that was 132.3%, 120.8% and 147.9% higher respectively, relative to the empty vector control transfected cells and significantly different from the wild-type construct (p = 0.022, p = 0.049, and p = 0.008, respectively) (Figure 3B). Relative levels of c-CBL protein in whole cell lysates prepared from samples obtained from a parallel experiment was determined. The c-CBL protein levels in samples representing untransfected and empty vector transfected cells were comparable and those representing the c-CBL WT and the three c-CBL mutants were comparable (Figure 3C).  C. Effect on cell cycle To investigate if the increases in cell viability in different c-CBL mutants are due to increased cellular proliferation, a cell cycle analysis was performed. A549 cells were transfected with the c-CBL WT or the three different mutants: S80N/H94Y, Q249E and W802*. The empty vector transfectant was used as a control. Forty-eight hours after transfection cell cycle analysis was performed as described in materials and methods. There was no significant change in the subG1, G1 or the S phase of the cell cycle among the different mutants compared to the WT construct (p = 0.64, p = 0.40, and p = 0.28, respectively). The G2/M phase of the cell cycle showed an increase in cell numbers for the three mutants, S80N/H94Y, Q249E and W802*, when compared to the WT but again the difference was not statistically significant (p = 0.25) (Figure 3D).  D. Effect on cell motility To investigate the effect of the expression of the above three c-CBL mutants on cell migration, we carried out wound healing assay as described in materials and methods. The closing of the scratch or the wound was monitored at 0, 12, 24, 36, and 48 h. (Figure 4A). In all the samples, that represented cells transfected with mutants, the wound gap was much smaller than that seen in the sample that represented cells transfected with c-CBL WT (p<0.001). We also determined the rate of wound closure for all the five groups. At 48 h, wild-type c-CBL transfectants showed 61.1% open wound while the S80N/H94Y, Q249E and W802* mutants showed 18.7%, 23.9% and 34.3% open wound respectively (p<0.001) (Figure 4B).  Figure 4 Figure 4 c-CBL mutations affect wound healing in A549 cells. E. c-CBL knockdown increases cell viability It is hypothesized that the LOH seen in our samples could lead to decreased expression of c-CBL. Thus we tested the effect of c-CBL knockdown in lung cancer cells. Compared to A549, H358 lung cancer cells express relatively high levels of endogenous c-CBL (data not shown). c-CBL expression was knocked down using lentiviral construct that expressed c-CBL specific shRNA and compared the results with those that were transduced with scrambled shRNA and the results are shown in Figure 5. We identified several clones that revealed varying degrees of c-CBL knockdown showing different sets of c-CBL lentiviral shRNA knockdown efficiency (Figure 5A). Of all the clones tested, Clone 27 was chosen for further experiments. Equal amount of cells were seeded in a 6-well plate and the cell proliferation was measured at various times and the results are depicted in Figure 5B. As expected, number of cells increased in a time dependent fashion from 100 to 190% relative to scrambled shRNA as control in a span of 48 h (p = 0.0002) (Figure 5B). The cell cycle phases in H358 cells that were knocked down with c-CBL shRNA were looked at and compared with the scrambled shRNA. There were no discernable differences between these two constructs in the different phases of the cell cycle (data not shown).  Figure 5 Figure 5 Knockdown of c-CBL using an shRNA increases cell proliferation. Go to: Discussion Our results demonstrate that c-CBL is somatically mutated (or has LOH) in lung cancers, and can significantly contribute to enhanced cell viability and motility. There was also a high prevalence of LOH with respect to c-CBL in lung tumors that harbored MET or EGFR mutation.  In the present study, we have demonstrated the occurrence of c-CBL mutations in lung cancer patients, especially with different ancestral variations. Mutations in c-CBL have been recently reported in juvenile myelomonocytic leukemia and myeloid malignancies. In the AML study, the mutation R420Q located in the junction of the RING finger and the linker region inhibited FMS-like tyrosine kinase 3 (FLT3) internalization and ubiquitination [20], thus contributing to the gain-in-function for the RTK. In addition, mutations such as H398Y, C384R, and L380P mapped to the RING finger domain and the linker region of c-CBL that is required for its E3 activity [23], [24], [25], [26], [27]. Additionally, homozygous mutations in the RING finger domain of the c-CBL gene were described as a result of acquired Uniparental Disomy (UPD) [26]. It is important to note that our results indicate LOH at 11q23 locus and these are mutually exclusive from missense mutations of c-CBL. The somatic mutations were all heterozygous. The mutations in AML led to abrogation of the E3 activity leading to prolonged RTK activation. In addition mutants located on the linker region surrounding the RING finger domain exhibited enhanced AKT signaling in response to cytokine stimulation [26]. In addition, it was shown in NH3T3 cells, that neither mutations in the RING finger nor the linker region causes transformation, however while certain mutations perturbs the ubiquitination, others affect receptor recycling and prolong kinase activity [28].  We report here c-CBL mutations that mapped not only to the RING finger domain, but also to the TKB domain, proline-rich domain and the C-terminal region, but none mapped to the linker region as reported in the AML studies described above [23], [24], [25], [26], [29]. In addition, 8 mutants that we detected were found in different ethnic backgrounds. For example, S80N/H94Y, Q249E, W802* were detected in Caucasians, Taiwanese and African-Americans, respectively. The results point out not only the difference between lung cancer and other cancers, but also genetic polymorphism among different races in the same cancer. Interestingly, there is a large disparity between African-American and other ethnic populations with lung cancer [30]. We have previously shown that there was a low frequency of EGFR and MET mutation in African-Americans as compared with Taiwanese and Caucasians [31]. In this study, the number of African-American samples analyzed was relatively fewer and we found 3 mutations that are unique to this ethnicity. It would now behoove us to further study the genetic alterations that can occur and determine the targeted therapeutics for African-Americans. Our results provide evidence of the importance of c-CBL in tumorigenesis and potential signaling. Our prediction, based on the AML data, would be that the V391I RING finger domain mutation would affect the E3 activity. Also, it will be important to determine the binding partners for c-CBL in the TKB domain and proline rich domain mutations. It has previously been shown that the TKB domain can bind to growth factor receptors and it will be important to determine the cross-binding of these mutants to MET and or EGFR. It would also be important in the future to look at fluorescence in situ hybridization/copy number changes in c-CBL in lung cancer.  c-CBL plays an important role in down regulating RTK-mediated signaling through K63 poly-ubiquitination and subsequent downregulation of RTKs followed by lysosomal degredation [3]. Mono-ubiquitination or ubiquitinated with K63-linked chains of substrates by c-CBL may lead to enhancement of biological and biochemical functions (reviewed in Hermann et al, 2007 [32]). The mutations that we analyzed in our studies all point out to the fact that the E3 activity of c-CBL on EGFR is intact; the EGFR levels in the various mutants remain same (Figure S2). Multiple kinases, both RTKs and non-RTKs could be acted upon by c-CBL, including ERBs, PDGFR, FMS, MET, c-Kit, VEGFR, FLT-1, RON, FGFR, IR, as well as SYK, FYN, LCK, FGR, LYN and c-ABL [3]. In lung cancers the relevant substrates of c-CBL in terms of degradation or signal transduction are yet to be identified.  The observation that c-CBL somatic mutations, especially S80N/H94Y, Q249E and W802* showed increased cell viability and cell motility that is in agreement with the physiological role for D-cbl in the regulation of apoptosis and differentiation identified in Drosophila is very significant [33]. It has been previously shown that activating c-CBL mutation downregulates EGFR signaling and decreases cellular proliferation and migration in breast cancer cell lines [34]. Although the role of c-CBL in the negative regulation of RTKs is well substantiated, thereby suggesting that it is a natural tumor suppressor, studies in cancer cells have revealed both tumor suppressor and tumor promoting activities depending on the type of c-CBL mutation and the number of alleles at the c-CBL locus [24]. In agreement with the above, the three c-CBL mutants described here appear to have tumor growth and metastasis promoting properties. Although these mutants are outside of the RING finger or the linker region of c-CBL, their downstream effects are significant so as to cause increased proliferation and migration, but the substrate affected by these mutations are not known yet. This raises the possibility that some of the cellular functions of c-CBL are independent of its ubiquitin-ligase activity, an area that we are currently investigating. The oncogenic nature of RTKs, addiction of cancers to growth signals and given the clustering of c-CBL, EGFR and MET mutations, it is possible that the transforming effect of c-CBL mutations is most likely a combinatorial effect of the three. We also show that LOH for c-CBL was found in a significant number of samples that harbored MET or EGFR mutations. The fact that about 7% of lung tumor samples are likely to have c-CBL mutations and an additional 22% are likely to harbor c-CBL-related LOH, makes c-CBL a highly mutated molecule in lung cancer. Since LOH alone is not enough to cause a transforming event [35], [36], [37], associated mutation in the MET or EGFR locus or yet another RTK discussed above may play a role in carcinogenesis. We predict that this LOH in c-CBL results in haploinsufficiency that downplays RTK ubiquitination leading to hyperactivity of the RTKs. However, whether this is sufficient cause for tumorigenesis remains to be determined. Consistent with our hypothesis is the fact that c-cbl-/- mice have increased kinase activity in lymphocytes, but is not sufficient for tumor formation [35], [36], [37]. c-CBL LOH could also lead to increased expression of c-CBL from the other allele to compensate for a loss of an allele. Alternately, there could be some form of synergy working with reduced c-CBL levels and mutated receptors that exacerbate the phenotype of each alone.  Previous studies from our lab and others have shown that East Asians with lung cancers have relatively high frequencies of gain-of-function of mutations in RTKs such as EGFR and MET [31]. In a cohort of Japanese patients an activating MET mutation was identified in the splice region that deletes the juxtamembrane domain that is involved E3 activity of c-CBL [38]. This study also found that activation of MET is mutually exclusive of EGFR, KRAS and HER2 gene mutations [38]. We failed to detect such mutations in significant numbers in lung tumor samples obtained from African-Americans (n = 29) and Caucasian (n = 50) patients. One MET mutation was identified in each of the groups whereas 1 and 3 EGFR mutations were identified in the African-American and Caucasian cohorts respectively. EGFR mutations have earlier been identified as one of the key mutations affecting lung adenocarcinoma patients in a comprehensive study of 188 patients [39]. Our study encompasses different histologies of NSCLC. However, the published series did not find any mutations in c-CBL or MET unlike our study that encompassed different subtypes of NSCLC. It is important to note, we have recently shown that MET mutations in lung cancer are in majority germline [31]. We have reported earlier c-CBL mutations in a small cohort of Taiwanese lung cancer samples [40]. In our efforts to understand the ethnic differences in the lung oncogenome, we also looked at PAX transcription factors such as PAX5 and PAX8 that are highly expressed in lung cancers; however there was no preferential expression or mutations of the above genes in lung tumor samples of African-Americans. In this study, we show relatively high frequency of c-CBL mutations in lung cancers, especially in the large cell type among Caucasians and particularly among African-Americans. We therefore propose c-CBL as an efficacious target for lung cancers in African-Americans that needs to be further substantiated. This is all the more important because the prognosis for African-Americans with lung cancer, especially for men is much poorer compared to their Caucasian counterparts [41].  In conclusion, the results presented in this study demonstrate that c-CBL is frequently mutated or even lost in lung cancers. Our results support a role for c-CBL mutants that are independent of its ubiquitination activity. Given the relatively high mutation rates in c-CBL as well as RTKs such as MET and EGFR, it is likely that their combined effect could be synergistic in promoting tumorigenesis. Oncogenic mutations in the monomeric Casitas B-lineage lymphoma (Cbl) gene have been found in many tumors, but their significance remains largely unknown. Several human c-Cbl (CBL) structures have recently been solved depicting the protein at different stages of its activation cycle and thus provide mechanistic insight underlying how stability-activity tradeoffs in cancer-related proteins may influence disease onset and progression. In this study, we computationally modeled the effects of missense cancer mutations on structures representing four stages of the CBL activation cycle to identify driver mutations that affect CBL stability, binding, and activity. We found that recurrent, homozygous, and leukemia-specific mutations had greater destabilizing effects on CBL states than did random non-cancer mutations. We further tested the ability of these computational models assessing the changes in CBL stability and its binding to ubiquitin conjugating enzyme E2, by performing blind CBL-mediated EGFR ubiquitination assays in cells. Experimental CBL ubiquitin ligase activity was in agreement with the predicted changes in CBL stability and, to a lesser extent, with CBL-E2 binding affinity. Two-thirds of all experimentally tested mutations affected the ubiquitin ligase activity by either destabilizing CBL or disrupting CBL-E2 binding, whereas about one-third of tested mutations were found to be neutral. Collectively, our findings demonstrate that computational methods incorporating multiple protein conformations and stability and binding affinity evaluations can successfully predict the functional consequences of cancer mutations on protein activity, and provide a proof of concept for mutations in CBL.  Keywords: CBL, driver mutations, protein interactions Go to: Introduction Whole exome sequencing of cancer patients has produced unprecedented amounts of data to analyze and interpret; these studies report a very large fraction of missense mutations which can potentially be implicated in tumorigenensis (1). Although some missense mutations can provide selective growth advantage to tumor cells (driver mutations), the large majority of them are considered to be neutral (passenger mutations). The mechanisms by which the driver variants may affect protein stability, interactions, and function remain largely unknown. Various computational methods have been developed to estimate the impacts of disease mutations on proteins but most of them exclusively use sequence features and do not explicitly utilize the protein three-dimensional structures, their physico-chemical properties and dynamics (2, 3). Many cancers are characterized by (de)activation of certain proteins which may be a result of missense mutations (4). The interconversion between active and inactive states is highly regulated in proteins and it is not well understood how these regulatory mechanisms are disrupted in cancer. The development of in silico approaches to estimate the effects of disease mutations on protein activity, stability and binding will help to define which are likely to be driver or passenger mutations. Moreover, understanding the mechanisms of their actions would allow for prioritization of potential driver candidates for better targeted therapies to design drugs which might in turn compensate for the reduced/enhanced protein stability or activity.  The monomeric Casitas B-lineage lymphoma (Cbl) RING finger ubiquitin ligase (E3) represents an exceptionally difficult yet important system to study the mechanisms of cancer mutations (5, 6). Strikingly, proteins from this family play both positive and negative regulatory roles in tyrosine kinase signaling which is aberrantly activated in many cancers (5). Oncogenic mutations in the c-Cbl gene (referred to as CBL thereafter) were found in human myeloid neoplasms and other tumors (5) but the significance of these mutations and their impacts on CBL function were studied only for very few mutants (7). The mechanistic aspects of CBL cancer mutations can now be adequately addressed as several CBL structures have become available which represent the snapshots of different stages of the CBL activation cycle (Fig. 1). All CBL proteins share a highly conserved N-terminus which includes a tyrosine kinase–binding domain (TKBD), a linker helix region (LHR) and a RING finger domain, while the C-terminus comprises a proline-rich region (8). The RING domain of CBL has E3 activity and ubiquitinates activated receptor tyrosine kinases which subsequently targets them for degradation (8). At the same time, since CBL proteins can bind to activated receptor tyrosine kinases via the TKBD domain, they can serve as adaptors by recruiting downstream signal transduction components such as SHP2 and P13K (9, 10).  Figure 1 Figure 1 CBL activation cycle. The structures representing the activation cycle of CBL are shown. In the inactive closed state (nCBL) the protein exists in the cytosol. Upon activation of the RTK, CBL can bind to the phosphorylated RTK (orange peptide) via TKBD ... Another aspect of CBL function which should be accounted for in modeling the effects of cancer mutations is that it can bind to the ubiquitin-conjugating enzyme (E2) in complex with ubiquitin (Ub) and a substrate protein, thereby facilitating the transfer of Ub from E2 to a lysine residue of the substrate (11). The crystal structure of the inactive complex of CBL with E2 (UbcH7) was solved more than ten years ago (12) while the active phosphorylated CBL-E2 (UbcH5B) complex was resolved fairly recently (13). According to the latter study, substrate binding and Tyr371 phosphorylation activates CBL by producing a large conformational change in order to place the RING domain and E2 in close proximity to the substrate. It was further confirmed that phosphorylation-induced conformational change is required for positioning of ubiquitin for effective catalysis (14).  Here we present a new approach, which aims to assess the effects of cancer mutations on stability, binding and activity of cancer related proteins. We apply computational models to four different stages of the CBL activation cycle (Fig. 1) and perform blind in vivo experiments of CBL-mediated EGFR ubiquitination. We show a rather remarkable agreement of experimental EGFR ubiquitination by CBL mutants with the computed changes in CBL thermodynamic stability and to a lesser extent with CBL-E2 binding affinity. The computational models not only quantitatively predict the magnitude of the effects of mutations but also shed light on the mechanisms of their action. Namely, we find that cancer mutations have greater destabilizing effects on four CBL states than random non-cancer mutations for recurrent, homozygous and leukemia mutations. Most damaging cancer mutations happen in the sites involved in Zn-coordination and in the formation of salt bridges and hydrogen bonds within CBL or between CBL and E2. Overall cancer driver mutations affect different or multiple stages of the CBL activation cycle either completely abolishing its E3 activity or partially attenuating it. The computational models based on stability and binding affinity calculations can discriminate experimentally validated driver from passenger mutations (with the exception of two mutations) and outperform several state-of-the-art bioinformatics methods aiming to predict phenotypic impacts of mutations.  Go to: Materials and Methods Computational modeling and analysis  Mapping of CBL mutations The COSMIC database (15) stores data on somatic cancer mutations and integrates the experimental data from full-genome sequencing studies. We extracted 103 missense mutations for the CBL gene from the COSMIC database (15) that could be mapped to four available CBL structures in its activation cycle (Fig. 1). Cancer mutations were classified into different classes according to the frequency of observed samples (single and recurrent mutations), zygosity (homo- and heterozygous mutations), types of cancer (leukemia and sarcoma) and the involvement of mutations in Zn-coordination (Table S1). In addition, all possible single nucleotide substitutions resulting in amino-acid changes in the CBL gene were performed to obtain a “Random” missense mutation reference set. After excluding mutations observed in COSMIC and mutations occurring on residues without known coordinates in crystal structures, we obtained 2,102 random missense mutations (Table S1). We also searched the dbSNP database (16) but found very few benign variations in the CBL gene. Detailed description is provided in Supplementary Materials and Methods.  Model preparation We investigated the effects of mutations on four states of the CBL activation cycle: (1) closed CBL state (nCBL), (2) partially opened CBL state bound to substrate (CBL-S), (3) unphosphorylated autoinhibitory CBL bound to substrate and conjugating enzyme UbcH5B (CBL-E2-S) and (4) phosphorylated CBL bound to substrate and UbcH5B (pCBL-E2-S) (Fig. 1). It was previously noted that both UbcH5B and UbcH7 can bind specifically to CBL but only UbcH5B can facilitate ubiquitination (17). The crystal structures of nCBL (PDB id: 2Y1M) (13), CBL-S (PDB id: 2Y1N) (13) and pCBL-E2-S (PDB id: 4A4C) (13) were obtained directly from the Protein Data Bank (PDB) (18). Only one crystal structure of the unphosphorylated inactive state of CBL bound to E2 (UbcH7) and Zap-70 peptide (PDB id: 1FBV) (12) was available in PDB. Although sequence identity between UbcH7 and UbcH5B proteins is 38%, the structural similarity is very high with the root mean square deviation (RMSD) of 1.04 Å. The unphosphorylated autoinhibitory structure of CBL-UbcH5B-S was therefore modeled based on CBL-UbcH7-S (PDB id: 1FBV) using Chimera (19). (Detailed description is provided in Supplementary Materials and Methods.)  Minimization procedure We applied our recently developed optimization protocol for minimizing wild-type and mutant structures (20). Heavy side chain atoms without known coordinates and hydrogen atoms were added to the crystal structures using the VMD (version 1.9.1) program (21) with models immersed into rectangular boxes of water molecules extending up to 10 Å from the protein in each direction. Wild-type protein complexes were minimized for 40,000 steps using explicit TIP3P water model. The final minimized models of wild-type protein complexes were used to produce all mutant structures and then an additional 300-step minimization for all mutant structures was performed. The energy minimization was carried out with the NAMD program (version 2.9) (22) using the CHARMM27 force field (23). For unfolding free energy calculations we applied the optimization procedure implemented in the RepairPDB module of the FoldX program (24) which optimizes the side chain configurations to provide a repaired structure. (See Supplementary Materials and Methods for more information.)  Binding and unfolding free energy calculations Binding free energy and effects of mutations on binding affinity were calculated according to the approach introduced by us earlier (20). Energy calculations were based on the modified MM-PBSA method that combined the molecular mechanics terms with the Poisson–Boltzmann continuum representation of the solvent (25) and statistical scoring energy functions with parameters optimized on experimental sets of several thousand mutations (equation 3). Binding free energy calculations were performed on minimized wild-type and mutant structures of inactive CBL-E2-S and active pCBL-E2-S states. The binding energy is defined as a difference between the free energies of the complex and unbound proteins.  ΔGbind=GCBL-E2-S-GCBL-S-GE2 (1) The change of binding energy due to a mutation can be calculated as:  ΔΔGbind=ΔGbindmut-ΔGbindWT (2) The effect of mutations on binding affinity was calculated in this work using the following energy function:  ΔΔGbind=αΔΔEvdw+βΔΔGsolv+γΔSAmut+εΔΔGBM+λΔΔGFD+δ (3) Here ΔΔEvdw is the change of the van der Waals interaction energy and ΔΔGsolv is the change of the polar solvation energy of solute in water. ΔSAmut represents a term proportional to the interface area of the mutant complex. ΔΔGBM and ΔΔGFD are the binding energy changes from BeAtMuSiC and FoldX, respectively. BeAtMuSiC (26) estimates the effect of mutations on binding based on statistical potentials.  The FoldX software program (24) was used to estimate the unfolding free energy and to model the unfolding state. FoldX calculates the effects of mutations on protein stability using an empirical force field. The BuildModel module was used to introduce a mutation, optimize the configurations of the neighboring side chains and calculate the difference in stability (unfolding free energy) between mutant ( ΔGfoldmut) and repaired native structure ( ΔGfoldWT).  ΔΔGfold=ΔGfoldmut-ΔGfoldWT (4) For comparison we used three alternative methods to predict the impacts of mutations on unfolding free energy: the Eris server (27), the Rosetta program (28) and the PoPMuSiC server (29). We also applied four additional webtools to assess the impacts of amino acid substitutions on CBL function: PROVEAN (30), PolyPhen-2 (31), MutationAssessor (32) and InCa (33). (See Supplementary Methods for more information.)  Go to: Experimental procedures Expression constructs  Human CBL cDNA was originally obtained from Wallace Langdon and subsequently cloned into the pCEFL expression plasmid (34). Point mutation constructs, described above, were created from wild-type CBL using the QuikChange II Site-directed Mutagenesis Kit according to the manufacturer’s instructions (Stratagene, La Jolla, CA). All constructs were confirmed by DNA sequencing.  Cell culture and transfections  The human embryonic kidney cell line HEK293T, the human nonsmall cell lung cancer cell line A549 and the human cervical cancer cell line HeLa used in this study were originally obtained from ATCC and maintained in culture using DMEM (Gibco, Grand Island, NY) supplemented with 10% FBS, 100 U/ml penicillin and 100 μg/ml streptomycin sulfate. Cell lines were authenticated by Short Tandem Repeat (STR) analysis by using either Promega Powerplex 16 (Promega, Madison, WI) or the AmpFISTR Identifier Kit (Life Technologies, Carlsbad, CA) and compared to the ATCC or DSMZ databases. HeLa cells were received in the lab in 2013 last authenticated on 7-30-15, A549 were received in the lab in 2015 and last authenticated on 8-12-15, and HEK293T were received in the lab in 1995 and last authenticated on 10-8-15.HEK293T cells were transfected using calcium phosphate according to the instructions accompanying the reagent (Profection; Promega Corp., Madison, WI), incubated 18 h prior to media change and grown for a total of 48 h prior to harvesting. A549 and Hela cells were transfected with Lipofectamine 2000 (Invitrogen, Carlsbad, CA). Transfections were allowed to incubate 6 h prior to media change, and cells were grown an additional 48 h before being harvested. Cells were starved for 4 hours and treated with EGF (100 ng/ul). Each cell-based experiment was repeated at least two times.  Immunoblotting and immunoprecipitation  To harvest proteins, cells were washed twice in ice-cold Dulbecco’s PBS-containing 200 μM sodium orthovanadate (Fisher Chemicals, Fairlawn, NJ) then lysed in ice-cold lysis buffer (10 mM Tris-HCl pH 7.5, 150 mM NaCl, 5 mM EDTA, 1% Triton X-100, 10% glycerol, 100 mM iodoacetamide [Sigma-Aldrich Corp., St. Louis, MO], 2 mM sodium orthovanadate, and protease inhibitors [Complete tabs®, Roche Diagnostics Corp., Indianapolis, IN]). All whole cell lysates were cleared of cellular debris by centrifugation at 16,000 × g for 15 m at 4 °C. Supernatant protein concentrations were determined using the BioRad protein assay (BioRad, Hercules, CA). For immunoblotting, 20μg of whole cell lysates were boiled in a 1:1 dilution of 2X loading buffer (62.5 mM Tris-HCl pH 6.8, 10% glycerol, 2% SDS, 1 mg/ml bromophenol blue, 0.3573 M β-mercaptoethanol) for 5 m. For immunoprecipitations, 150 μg of each of the whole cell lysates were incubated with rabbit anti-EGFR (Ab-3; Millipore) and with Protein A/G+ agarose beads (sc-2003; Santa Cruz Biotechnology, Santa Cruz, CA). All immunoprecipitations were incubated overnight at 4°C with tumbling. Immune complexes were washed five times in 1 mL cold lysis buffer, then resuspended in 2X loading buffer, boiled for 5 m, then resolved by SDS-PAGE and transferred to nitrocellulose membranes (Protran BA85; Whatman, Sanford, MA). For immunoblot detection of proteins, the following antibodies were used: rabbit anti-EGFR (2232L; Cell Signaling), rat monoclonal high affinity anti-HA-peroxidase, (clone 3F10; Roche), rabbit anti-Cbl (sc-C-15; Santa Cruz) and mouse anti-Hsc70 (sc-7298; Santa Cruz). Horseradish peroxidase linked donkey anti-rabbit IgG (NA934V; GE Healthcare, Piscataway, NJ), or donkey anti-mouse IgG (NA931: GE Healthcare, Piscataway, NJ) immunoglobulin was used with SuperSignal (Pierce Biotechnology Inc., Rockford, IL) to visualize protein detection.  Densitometric Analysis  Immunoblots were developed on HyBlot CL Autoradiography Film (Denville Scientific Inc., South Plainfield, NJ) with an X-OMAT automated processor (Eastman Kodak, Rochester, NY). Protein expression levels were then recorded using an Epson Perfection V750 PRO scanner (Epson Inc., Long Beach, CA) and densitometric analysis was performed using Adobe Photoshop software version 7.0 (Adobe Systems Inc., U.S.A). EGFR ubiquitination signal intensity was determined by optical density in a set area normalized against the EGFR band intensity in parallel immunoblots and expressed as a densitometric ratio of ubiquitination/EGFR levels. The mean densitometric ratio of EGFR ubiquitination, in the presence of each CBL mutation, was then assessed relative to wild-type CBL where the ratio was set at 1.0.  Go to: Results Cancer mutations impact CBL stability  We estimated the thermodynamic stability (unfolding free energy) changes, ΔΔGfold, upon mutation for closed (nCBL) and partially opened (CBL-S) CBL states (Fig. 1). These states did not involve binding to E2. The examination of the ΔΔGfold distribution for random non-cancer mutations (Fig. 2A) showed that it was very similar to experimental distributions produced by random mutagenesis on a set of different proteins (35, 36). Namely, the distribution for random non-cancer mutations was asymmetrically centered at positive energy values and there were about 10% of random mutations with highly damaging effects on CBL structures by more than 5 kcal mol−1 (Fig. 2A). Despite the presence of highly damaging random mutations, recurrent cancer mutations overall produced significantly larger destabilizing effects compared to random (p-value ≪ 0.01) for both nCBL and CBL-S states (Fig. 2A and S1, Table S2). This was not the case for single cancer mutations observed in only one patient (Table S2). Overall, the form of the ΔΔGfold distribution for recurrent cancer mutations was different from the ΔΔGfold distribution of random mutations (Kolmogorov-Smirnov test p-value ≪ 0.01) whereas the distribution for single cancer mutations was indistinguishable from a random mutation distribution. However, it does not mean that all single cancer mutants can be considered passenger, as we show later, this is not the case for some of them.  Figure 2 Figure 2 (A) The distribution of unfolding (ΔΔGfold) and binding (ΔΔGbind) free energy changes for Single, Recurrent and Random class mutations on nCBL and pCBL-E2-S states (see SI for other states). Positive and negative values ... The top 25% cancer mutations with the largest damaging effects are presented in Table S3. Many of them occurred in Zn-coordinating sites while many others did not involve Zn-coordination sites. Some of these latter mutations (G415V, G413R and G415S) introduced large Van der Waals clashes with neighboring residues which could not be accommodated by side chain rearrangements, whereas other mutations (S376F, Y371D and L405P) affected disulfide or hydrogen bonds. For example, L405P located in the middle of an α-helix affected the CBL structure by introducing an energetically unfavorable kink in the helix due to its inability to donate an amide hydrogen bond.  Effects of cancer mutations on CBL-E2 binding  The outcome of mutations can be assessed by the extent of structural changes they induce. We calculated the local root mean squared deviation (RMSD) between the minimized wild-type and mutant structures around the mutated site. We found that the protein backbone in the vicinity of a mutation of the CBL-E2 complex underwent larger local conformational changes upon recurrent cancer mutations, especially for mutations occurring in Zn-coordinating site clusters, compared to random mutations (p-value ≪ 0.01, Fig. S2 and S3). In the previous section we analyzed the original closed (nCBL) state and partially opened (CBL-S) state induced by substrate binding. In this section we study the effects of cancer mutations on two other CBL states: an autoinhibitory CBL state bound to substrate and conjugating enzyme UbcH5B (CBL-E2-S) and phosphorylated active CBL state bound to substrate and UbcH5B (pCBL-E2-S) (Fig. 1) (13). Effects of mutations on binding can be in general linked with their structural locations, therefore we examined the locations of mutations and found that about one third of all cancer mutations were located on the CBL-E2 interface (Fig. S4). This preference was found to be statistically significant (p-value ≪ 0.01) (Fig. S4).  It was previously experimentally shown that binding between CBL and E2 was rather weak with a micromolar dissociation constant (13). Despite the fact that the interface between CBL and E2 in the inactive state is several residues larger than in the active state, consistent with experiments, we found somewhat stronger binding between CBL and E2 for the active state of CBL (Table S4). Similarly to the impacts of cancer mutations on stability, recurrent (but not single) mutations destabilized CBL-E2 binding significantly (Fig. 2B, S1 and Table S2). Importantly, cancer mutations reduced the CBL-E2 binding in the active state considerably more than in the inactive state (p-value = 0.002) (Fig. 2B) and the effects of cancer mutations on CBL-E2 binding of the active state were noticeably larger compared to random mutations even if we excluded mutations in Zn-coordinating sites (Table S2). This latter observation does not hold true for the autoinhibitory CBL-E2-S conformation.  Next, all cancer mutations were ranked with respect to their effects on CBL-E2 binding. About half of all highly damaging mutations (top 25% of most damaging mutations, Table S3) impacted both stability and CBL-E2 binding for all four CBL states. Among them, several mutations occurred in Zn-coordinating clusters and M400 and L405 sites. Another class of mutations (involving W408 and R420 sites) mostly influenced CBL-E2 binding. For example, the W408S mutation caused the largest perturbation in CBL-E2 binding of the active state by decreasing binding affinity up to 3.5 kcal mol−1 but having a moderate effect on stability of the nCBL and CBL-S states. It was previously suggested that amino acids in positions W408 and I383 constituted specificity of CBL-E2 binding and corresponded to binding hot spots (10, 12). Mutations in the I383 site are not recorded in the COSMIC database although they have a profound destabilizing effect on CBL-E2 binding for both states (see data for random mutations on the ftp site ftp://ftp.ncbi.nih.gov/pub/panch/CBL). Another site affecting binding of CBL (R420) is a highly conserved site in the CBL family, it strongly interacts with the Q92, W93 and S94 residues of E2. It was previously experimentally verified that mutations in this site disrupted the CBL activity and could be associated with cytokine-independent growth (37). There are 24 cancer patient samples where mutations of this site (R420Q, R420L and R420P) are found and they all produce strong destabilizing effects on the active pCBL-E2-S state.  According to the zygosity annotation, there are 49 heterozygous (“Hetero”) and 27 homozygous (“Homo”) mutations (Table S1) and for the rest of cancer mutations their zygocity status is undetermined. Overall, we found that destabilizing mutations were enriched among homozygous compared to heterozygous mutations for all stages of CBL activation cycle (p-value = 0.021 – 0.035) and were more prevalent in leukemia compared to sarcoma patients (p-value = 0.000 – 0.003) (Fig. 2B, Table S2). This observation could not be attributed to the prevalence of homozygous mutations in Zn-coordinating clusters since these clusters had almost equal numbers of heterozygous and homozygous mutations. It was reported earlier that many homozygous mutations could be connected to uniparental disomy when germline heterozygosity would lead to neoplasia upon reduction to homozygosity (6, 38, 39). If some patients with myeloproliferative neoplasms had germline heterozygous mutations in CBL, then lost the wild-type CBL allele and duplicated mutant allele, it would be possible that the damaging mutant allele would be duplicated in a cancer cell with a higher probability. Indeed, human leukemia samples show the loss of the normal CBL allele.  CBL-mediated EGFR ubiquitination: comparing experiments with computational models  Next we randomly picked fifteen cancer mutations based on their predicted damaging status, trying to equally sample highly damaging and benign mutations irrespective to their frequencies in cancer samples, and experimentally tested the E3 activity of wild-type and mutant CBL proteins (see Methods, Table 1). As can be seen in Fig. 3A, in the presence of wild-type CBL, activation of the EGFR induced more than a 10-fold increase in the ubiquitination of the EGFR compared to empty vector (endogenous CBL levels) in HEK293T cells (compare lane 4 to lane 2 in top panel of Fig. 3A). The E3 activity of wild-type CBL was mirrored by a decrease in the levels of immunoprecipitated EGFR consistent with the targeting of ubiquitinated EGFR for degradation (compare lanes 2 and 4 in second panel of Fig. 3A).  Figure 3 Figure 3 (A) EGFR ubiquitination. HEK293T cells were transfected with plasmids encoding wild-type or mutant CBL proteins along with the EGFR and hemagglutinin (HA)-epitope tagged ubiquitin. The vector control and wild-type, CBL transfectant cells were incubated ... Table 1 Table 1 Experimental relative densitometry values for HEK293T cells, predicted changes in stability (ΔΔGfold kcal mol−1), binding affinity (ΔΔGbind kcal mol−1) and comparison with other methods for 15 experimentally ... We classified all mutations into three groups (damaging, attenuating and benign) according to their experimental relative densitometry data and compared experimental data with the estimates produced by the computational models. The first group of damaging mutants included C396R, H398Q, Y371H, K382E and C381A which completely abolished CBL activity (relative densitometry was less than 10%) (Fig. 3B) and the levels of total EGFR were not decreased by these mutants (Fig. 3A). Only two of these mutants belonged to the Recurrent class of mutations. Table 1 shows predicted and experimentally verified effects of mutations on all four CBL states. All five mutants were predicted to be damaging by stability and binding affinity calculations but the mechanisms of their action were different. C396R, H398Q and C381A mutations disrupted Zn-coordinating clusters and had very damaging consequences according to the stability model for both nCBL and CBL-S states and damaging effects on CBL-E2 binding even though none of these mutations were located on the CBL-E2 interface. On the other hand, the K382E mutation did not affect Zn-coordination but destabilized the nCBL state. This, in turn, could be explained by the charge substitution that led to the disruption of the K382-E373 salt bridge within the closed state of nCBL (Figure 4 and Figure S5A). Moreover, K382E mutation had a significant impact on CBL-E2 binding due to the disruption of a salt bridge between pY371 and K382 affecting the stability of CBL in the active state (Figure 4 and Figure S5D). No significant changes were observed for CBL-S and CBL-E2-S states (Figure S5B,C). K382E mutation was previously observed in Noonan syndrome and was speculated to affect CBL stability or binding (40). Finally, the Y371H mutation not only abolished phosphorylation at the Y371 site but also had a profound destabilizing effect on all four states as evident from Table 1.  Figure 4 Figure 4 Structures of wild type (green) and K382E mutant (tan) are shown for nCBL and pCBL-E2-S states. Salt bridges are shown as dashed lines and in both cases mutation affects the stability by disrupting the salt bridge. The second group of mutants (M374V, V430M, P428L, Q249E and double mutant S80N/H94Y) maintained the CBL activity equivalent to or greater than wild-type CBL (relative densitometry of 80% or higher) (Fig. 3B). Consistent with this, the levels of the activated EGFR were also decreased in these samples (Fig. 3A). To confirm that the retained E3 activity of these mutants was not cell type specific, we transfected these mutants into the non-small cell lung cancer cell line A549 and the cervical cancer cell line Hela (Fig. S6). The S80N/H94Y and Q249E mutations were previously identified in human non-small cell lung cancers making the A549 cell line a relevant cell type to investigate the function of these CBL mutants while other cancer mutations were found in other cancer types. As in the HEK293T cells, transfection of wild-type CBL into A549 and HeLa cells resulted in a significant increase in EGF stimulated ubiquitination of EGFR compared to the cells transfected with empty vector (Fig. S6). The increase in transfected CBL protein compared to endogenous CBL protein was less in HeLa than in either the A549 or HEK293T cells. Consistent with this, the fold increase in EGF stimulated ubiquitination of the EGFR by wild-type CBL was smaller in Hela cells (about three fold) compared to either A549 or HEK293T cells (more than 10 fold in each). As in HEK293T cells, the Y371H mutant did not stimulate ubiquitination compared to the vector transfected control for A549 or HeLa cells (Fig. S6A and C). The CBL mutants that were fully active in HEK293T cells (Fig. 3A and B) maintained CBL E3 activity in A549 and HeLa cells (Fig. S6). Two of these CBL mutants resulted in more ubiquitination of EGFR compared to wild type CBL. For example, the Q249E mutant showed an increased ubiquitination of EGFR in 293T and HeLa cells while the M374V mutant resulted in higher ubiquitination levels in A549 and HeLa cells (Figure 3 and S6). Only V430M mutant was on the borderline with the densitometry ratio of 0.8. All mutations from the second group were predicted to be benign according to our computational models, whereas V430M, consistent with experimental data, had a borderline destabilizing impact on CBL-E2 binding (Table 1).  Finally, the third group constituted mutations (L399V, G375P, P395A and V391I) which attenuated the CBL E3 activity according to the relative densitometry data (Fig. 3B). Only one of these mutants was observed in two cancer samples while other three belonged to the Single mutation class. Concordant with this, there were intermediate levels of EGFR (Fig. 3A). G375P and P395A mutations were predicted to have partially damaging effects while V391I and L399V were classified as benign in our predictions. Interestingly, mutations from this third group affected only some of the CBL states. In contrast, the highly damaging mutations from the first group had damaging impacts on almost all CBL states.  Next we tested if the reduction in CBL ubiquitination activity was directly correlated with the effects of mutations on stability (Fig. 5A). The relationship between experimental densitometry data and ΔΔGfold was better described by an exponential dependence with correlation coefficient (R) of 0.77 and 0.78 for CBL-S and nCBL states respectively (Table S5). Indeed, the change in absorption between wild-type and mutant proteins which refers to a fraction of active mutants can be described by the Boltzmann equation relating the probability of a state with the energy of this state. A mutation may lead to damaging effects and a loss of function if it impacts at least one of the CBL functional states. Taking this into consideration we calculated the correlation between densitometry and stability changes taking a maximum of ΔΔGfold values for the nCBL and CBL-S states. As shown in Fig. 5A and Table S5, changes in stability can indeed explain the effects of mutations on CBL ubiquitination activity with high correlation of R = 0.83. For comparison, several alternative methods were applied to predict the effect of mutations on unfolding free energy, they all reported correlation coefficients ranging from 0.21 to 0.52, with only one method PopMusic reporting a statistically significant correlation of 0.52 (Table S5). The relationship between densitometry and binding affinity changes was on the borderline of significance with a linear correlation coefficient of 0.48 and 0.62 (if two highly damaging mutations were excluded) (Fig. 5B and Table S5).  Figure 5 Figure 5 (A) Comparison between experimental densitometry values (exponential model) and maximal predicted changes in stability. (B) Comparison between experimental densitometry values (linear model) and predicted changes in binding affinity of pCBL-E2-S state. ... There are various methods which predict the phenotypic effects of mutations, (30–33, 41) and some of them use structural features (31, 41). We applied four state-of-the-art independent methods to predict the impacts of mutations on CBL function: PROVEAN (30), PolyPhen-2 (31), MutationAssessor (32) and InCa (33). Some of these methods outperformed our model in classifying cancer from random mutations since they were trained to distinguish disease from neutral variants (Table S6). However, all four methods had a very limited accuracy in classifying CBL cancer mutations into those that disrupted function and those that did not. As evident from Table 1 and Table S5, all methods except for PROVEAN over-predicted damaging effects of experimentally tested mutations while PROVEAN was the only method which produced a significant correlation between the densitometry data and PROVEAN’s score (R=0.58, Table S5).  Go to: Discussion Stability-activity balance in cancer related proteins  Evolutionary selection to maintain structural, foldable and functional proteins eliminates many mutations in protein sequences. On the other hand, thermodynamic stability can be compromised in evolution to ensure certain arrangements of catalytic and binding sites, which might not be energetically optimal (36, 42). In tumorigenesis, protein stability or binding may be reduced (or in some cases increased) due to cancer mutations. As a consequence it can lead to decreased fitness at the protein level, but may confer a fitness advantage for the population of tumor cells (43, 44). However, the extent of the stability-activity tradeoff in oncogenes and tumor suppressors remains largely unknown. Using the example of the CBL protein, here we tried to elucidate the stability-activity balance and to understand whether the loss or gain of activity in cancer-related proteins can be accompanied by compromised stability or binding.  In contrast to many other computational studies, which attempt to link stability with activity by mostly focusing on one protein conformation, we performed an analysis on four different stages of the CBL activation cycle. Our predictions of changes in stability and binding were further elucidated by the experimental CBL-mediated EGFR ubiquitination assays. We found a strong relationship between the effects of mutations on CBL stability and experimentally obtained densitometry data (quantifying activity), while a relatively weaker correlation was observed between changes in densitometry and CBL-E2 binding affinity. It could be explained either by a less significant impact of experimentally tested mutations on CBL-E2 binding or by a limited coupling (compared to CBL stability) between CBL-E2 binding and E3 activity.  Drivers or passengers?  According to our study, about two thirds of all experimentally tested mutations either completely abolished or attenuated E3 activity, while one third of them were neutral. Trying to assess the limitations of our models, we applied state-of- the-art methods commonly used to predict functional effects of mutations. However, these methods failed to distinguish inactivating from neutral mutations as accurately as the computational models which were solely based on accounting for the effects on stability and binding for different CBL states. Indeed, many methods are not specifically designed to discriminate driver from passenger mutations within the pool of cancer mutations, but rather they perform a task of distinguishing cancer from neutral mutations.  Certainly, the large majority of all mutations detected in cancer genomics studies are likely to be neutral although the collective burden of passenger mutations may also alter the course of tumorigenesis (45). When we analyzed the distribution of all single cancer missense mutations observed in the CBL gene, their average effects on stability or binding were not found to be significantly different from the pool of random mutations. These mutations either mostly constitute true passenger mutations or their oncogenic mechanisms are not directly connected to protein destabilization. The story is quite the opposite for the recurrent mutations as they have on average significantly higher destabilizing effects than random mutations and the majority of them should be drivers. An interesting group of mutations includes those which are highly damaging but found only in one cancer sample (10% of all single cancer mutations). These mutations may represent either rare driver or latent driver mutations (46). Several of these mutations were experimentally tested and are listed in Table 1.  CBL-E2 binding in deciphering the mechanisms of cancer  Our analysis showed that cancer mutations reduce CBL-E2 binding in the active state considerably more than in the inactive state. It is in contrast to the stability-activity tradeoff reported for EGFR and other cancer related proteins (47), where cancer mutations may disrupt autoinhibitory interactions and activate the kinase (48, 49). Importantly, all mutations with experimentally tested high inactivating effects have impacts on both CBL stability and CBL-E2 binding (Table 1). In fact, the correlation between ΔΔGfold and ΔΔGbind for CBL cancer mutations from COSMIC is positive (R = 0.20 – 0.48 depending on the CBL state, p-value < 0.05). It might seem counterintuitive as some residues maintain stability by sustaining the RING-TKBD autoinhibitory interactions within CBL. This interface overlaps with the CBL-E2 interface and competes against E2 binding (13). One might think that disruption of RING-TKBD interactions and destabilization of CBL might facilitate binding to E2 and therefore lead to CBL activation. A slight activation was actually observed for two mutants, one of which (M374V) was directly located on the CBL-E2 interface. However, as we showed through the positive high correlation between stability and activity changes, this mechanism is rarely observed and the vast majority of mutations disrupt CBL-E3 activity by either destabilizing CBL and/or CBL-E2 binding or by directly affecting phosphorylation of the critical tyrosine Y371.  Overall, our results support the idea that decrease of E3 activity, of the ability to ubiquitinate receptor tyrosine kinases, of CBL stability and/or CBL-E2 binding can give a selective advantage for tumor cells. However, there are several factors that complicate deciphering the mechanisms of action of CBL cancer mutations. CBL mutations can be dominant-negative and mutated CBL might not only change the CBL E3 activity but also might affect the concentration and activity of the wild-type CBL. According to a ratiometric method to identify driver genes in cancer (1), CBL can be regarded as an oncogene since it has several mutation hot spots. On the other hand, as shown in our study and in other studies, cancer driver mutations can inactivate the E3 activity of CBL, so it can also be regarded as a tumor suppressor. Although the latter fact complicates the development of CBL targeted therapies, understanding the delicate balance among different CBL-affected pathways may facilitate the indirect drug targeting of damaged CBL proteins. The current genetics-based frameworks to analyze cancer genome-wide sequence data are necessary but not sufficient for understanding the processes of carcinogenesis and developing informed, targeted therapies. Our approach, which can be applied in general to different proteins of interest, emphasizes the importance of the physics of binding and protein conformational ensembles in deducing the mechanisms of cancer.   : 2\n",
      "Rasopathies are phenotypically similar syndromes with overlapping features. These autosomal dominant disorders typically present with distinctive craniofacial features, a wide spectrum of congenital heart defects, short stature, and variable neurocognitive impairments [Rauen, 2013]. Rasopathies are caused by mutations that deregulate the RAS-mediated Extracellular signal-Regulated Kinases 1 and 2 (ERK1/2) signal transduction pathway [Allanson and Roberts, 2011; Tartaglia et al., 2011].  Noonan syndrome (NS; MIM #163950) is the most common Rasopathy, affecting between one in 1,000 and one in 2,500 children worldwide. Individuals with NS typically have distinct craniofacial features that may include wide-spaced, and down-slanting palpebral fissures with ptosis and epicanthal folds, a short nose with a depressed nasal bridge and anteverted nares, low-set ears with prominent helices, which may be posteriorly rotated, and a high arched palate. Cardiac defects occur in 50%–80% of individuals with pulmonic stenosis and hypertrophic cardiomyopathy (HCM) being the most common. Other findings include short stature, intellectual disability in up to 30% of patients, ophthalmologic, renal and musculoskeletal defects, lymphatic dysfunction, bleeding diathesis, cryptorchidism, and predisposition to leukemia [Allanson et al., 2010]. Mutations associated with NS are found in nonreceptor protein tyrosine phosphatase PTPN11 (50%); the Ras guanine nucleotide exchange factor SOS1 (13%); small GTPases KRAS, NRAS, and RIT1; and the serine/threonine kinase RAF1 (3%–17%). Additionally, the molecular cause for the disorder is not known for ∼20% of patients with NS [Allanson and Roberts, 2011].  Cardiofaciocutaneous syndrome (CFC; MIM #115150) is a rare Rasopathy with craniofacial features that are similar to but often coarser than those in NS [Rauen, 2012]. Cardiac findings are similar to those in NS and include cardiac arrhythmias, heart defects, and HCM in frequency similar to those of NS and Costello syndrome [Rauen, 2006; Yoon et al., 2007; Siegel et al., 2011]. Most individuals have short stature, a broad forehead, macrocephaly, bitemporal narrowing, shallow orbital ridges, cutaneous abnormalities (such as sparse hair with abnormal texture, dystrophic nails, ichthyosis, hyperpigmentation, and hyperkeratosis), and cognitive delay [Yoon et al., 2007; Allanson et al., 2011]. Brain anomalies, such as crowding of the posterior fossa or Chiari I malformations, are not uncommon. At least four genes are associated with CFC: BRAF (∼75%–80%), MAP2K1 and MAP2K2 (∼10%–15%), and KRAS (<5%) [Allanson et al., 2011; Rauen, 2012; Rauen, 2013].  SHOC2 (MIM #602775) is a scaffold protein in the ERK1/2 pathway, which tethers RAS, RAF-1, and the catalytic subunit of protein phosphatase 1c (PP1c). Recruitment of PP1c enables dephosphorylation of the S259 residue of RAF-1 resulting in the stimulation of RAF-1 kinase activity by phosphorylation of RAF-1 at Ser338 [Rodriguez-Viciana et al., 2006b]. The only previously reported pathogenic SHOC2 variant (c.4A>G; p.S2G) is associated with Noonan-like syndrome with loose anagen hair (NS/LAH). Most reported individuals with NS/LAH have a prominent forehead, macrocephaly, growth hormone deficiency, sparse, loose and slow-growing anagen hair, hyperpigmented skin with eczema or ichthyosis, mild psychomotor delays, hypernasal voices, and attention deficit hyperactivity disorder that improves with age [Mazzanti et al., 2003; Cordeddu et al., 2009]. Cardiac defects appear to include an over-representation of mitral valve dysplasia and septal defects in comparison with classic NS. Other reports have expanded the clinical phenotype to include structural brain anomalies and myelofibrosis [Gripp et al., 2013]. The p.Ser2Gly mutation creates a new recognition site for N-terminal myristoylation, causing aberrant targeting of SHOC2 protein to the plasma membrane and impaired translocation to the nucleus upon stimulation with growth factor.  We report here the identification of a novel SHOC2 functional variant (c.519G>A; p.M173I) in two individuals who do not have classic symptoms of NS, CFC or NS/LAH, but display features overlapping all three conditions. In this study, we have examined the EGF signal transduced through the SHOC2 p.M173I variant in order to demonstrate that it alters the ability of SHOC2 to accelerate ERK1/2 phosphorylation. We provide evidence that the M173I mutation leads to changes in the assembly of the SHOC2–RAS–RAF-1–PP1c complex.  The first patient is a 5-year-old female referred for evaluation of right optic nerve hypoplasia, bilateral nystagmus, speech delay, poor hair growth, and hyperactive behavior. In addition, magnetic resonance imaging showed multiple irregularities of unknown significance in the midbrain, pons, and medulla. She is the oldest child of nonconsanguineous parents of European ancestry. She is the 6 pound 9 ounce product of a 34 weeks gestation, who was delivered by emergency C-section due to cephalopelvic disproportion. Milestones, including walking at 11–12 months of age and speech onset, were normal. The family history was significant, as both parents required special education classes in high school. She had average stature (47th percentile), macrocephaly (98th percentile), sparse and slow-growing hair, which was not loose, normal skin pigmentation, bilateral ptosis and nystagmus, normal chest with wide-spaced nipples, and clubbed nails (Fig. 1A and B). A Simulconsult database query (http://www.simulconsult.com/run/index.html), using the keywords ptosis, macrocephaly, optic nerve hypoplasia, and poor hair growth, suggested CFC as a possible diagnosis. A chromosome microarray was negative for copy number variants. DNA sequencing of a Rasopathy gene panel including BRAF, CBL, HRAS, KRAS, MAP2K1, MAP2K2, NRAS, PTPN11, RAF1 SHOC2, and SOS1 identified a novel heterozygous SHOC2 variant c.519G>A; p.M173I. Nucleotide numbering reflects cDNA numbering with +1 corresponding to the A of ATG translation initiation codon in the reference sequence for human SHOC2 (GenBank: NM_007373.3), according to journal guidelines (http://www.hgvs.org/mutnomen/). The initiation codon is codon 1. This variant was not found in 6,500 asymptomatic individuals. Both parents were tested, and our proband's father had a heterozygous copy of the p.M173I variant. He was 6’3” tall, had intellectual disability, a high-arched palate, sparse hair, and clubbed nails (Fig. 1C–E). Echocardiograms and renal ultrasounds on both patients were negative (Supp. Table S1 provides a comparison of our patients with classic findings in NS, CFC, and NS/LAH). This novel c.519G>A variant has been submitted to the NCBI ClinVar database (http://www.ncbi.nlm.nih.gov/clinvar/) (SCV000147871.1).  Figure 1.  Figure 1. Open in figure viewerDownload Powerpoint slide Phenotypes of individuals with the p.M173I SHOC2 variant. A and B: Proband face and profile, showing macrocephaly, ptosis, normal skin pigmentation and hair, which has never been cut. C, D, and E: Face and, profile of proband's father, showing sparse hair, normal skin pigmentation, lack of ptosis and clubbed nails. Additional features of both individuals are described in the text. To evaluate the functional consequences of the SHOC2 p.M173I variant, we performed a series of experiments in cells lacking endogenous SHOC2 protein (see Supp. Materials and Methods). Cos1 cells with constitutive knock-down of SHOC2 (Cos1-LV1) described in our earlier studies were utilized [Galperin et al., 2012]. In these cells, phosphorylation of MEK1/2 and ERK1/2 is severely impaired upon stimulation with physiological amounts of EGF. The single-base substitution (c.519G>A) identified in both patients was introduced into SHOC2 in the tagRFP-N1 vector. In addition to p.M173I, six “silent” variants were also introduced in the SHOC2 cDNA to render it resistant to shRNA without changing its amino acid sequence [Galperin et al., 2012]. Cells were transfected with wild-type (WT) SHOC2-tRFP and the SHOC2 (M173I)-tRFP, and then stimulated with a low (physiological) concentration of EGF (0.2 ng/ml). We observed that while the constitutive depletion of SHOC2 led to a dramatic decrease in the extent of phosphorylation of MEK1/2 and ERK1/2 upon EGFR activation (Fig. 2A, lanes 4–6), transient expression of the WT SHOC2-tRFP rescued EGF-induced ERK1/2 phosphorylation to the extent of the endogenous SHOC2 (Fig. 2A, lanes 10–12). As expected, the maximum increase in ERK1/2 phosphorylation was observed 7 min after stimulation of cells with EGF. Conversely, the SHOC2 (M173I)-tRFP mutant expressed to a similar level as WT SHOC2-tRFP was only able to restore ∼10% above the basal level of EGF-induced ERK1/2 activity (Fig. 2A, lanes 7–9 and Fig. 2B). Overexpression of the SHOC2 (M173I)-tRFP in parental Cos1 cells did not lead to changes in ERK1/2 activity (Supp. Fig. S1A and B). We also did not observed changes in AKT phosphorylation. These findings indicated that the M173I mutation altered the function of the SHOC2 protein.  Figure 2.  Figure 2. Open in figure viewerDownload Powerpoint slide Molecular evaluation of the SHOC2 p.M173I variant. A: Cos-LV1 cells were transiently transfected with full-length SHOC2-tRFP or SHOC2 (M173)-tRFP truncated mutants. Cells were serum-starved for up to 10 hr and treated with 0.2 ng/ml EGF for indicated times at 37°C. The lysates were probed by immuno-blotting for EGFR, RAF-1, SHOC2, activated MEK1/2 (pMEK1/2), activated ERK1/2 (pERK1/2), total ERK1/2 (tERK1/2), and GAPDH (loading control). Dotted line shows area of the blot that was cropped to conserve space. B: Multiple blots from the experiments exemplified in A were analyzed. Bars represent the mean values (±SE, n = 3) of phosphorylated ERK1/2 activity normalized to total ERK1/2 in arbitrary units (pERK/ERK), (a vs. b, c vs. d, P < 0.05, one-way ANOVA followed by a Student's t-test was used to determine differences in phosphorylated ERK1/2). C: 293FT cells transiently cotransfected with expression vectors encoding GST-SHOC2 and HA-M-RAS. GST-SHOC2 were immunoprecipitated (IP) with glutathione sepharose beads, and the immunoprecipitates were probed with GST antibodies to detect SHOC2, RAF-1, PP1c, and HA antibodies to detect M-RAS. Cell lysates were immunoblotted with anti-SHOC2, anti-RAF-1, anti-PP1c, and anti-HA antibody. D: Cos-LV1 cells transiently expressing SHOC2 (WT)-tRFP and SHOC2(M173I)-tRFP were serum-starved for up to 10 hr and treated with 0.2 ng/ml EGF for indicated times at 37°C. The lysates were probed by immuno-blotting for SHOC2, phospho-(S338) RAF-1, activated ERK1/2 (pERK1/2), and GAPDH (loading control). Results in each panel are representative of three independent experiments. E: Multiple blots from the experiments exemplified in D were analyzed. Bars represent the mean values (±SE, n = 3) of phosphorylated RAF-1 (7 min of EGF treatment) normalized to GAPDH in arbitrary units (pRAF-1/GAPDH) (a vs. b, P = 0.029). The Met173 amino acid residue of SHOC2 is located in the fourth leucine-rich repeat (LRR4) of the SHOC2 LRR domain and is conserved among all SHOC2 vertebrate orthologues (Supp. Fig. S2A). We speculated that the disparity in the capacity of the SHOC2-M173I mutant to enhance ERK1/2 signaling was due to alterations in protein stability. To assess the physical effect of p.M173I, we analyzed a molecular model of the full LRR domain of SHOC2 [Jeoung et al., 2013]. From both the model and the alternating hydrophilic/hydrophobic pattern found in a beta-sheet, Met173 is predicted to be a solvent-facing residue located on the concave surface of the SHOC2 LRR domain. The M173I mutation can be readily accommodated without any perturbation to surrounding residues suggesting that the point mutation does not destabilize the fold of SHOC2 (Supp. Fig. S2B). To experimentally test the stability of the SHOC2 mutant in comparison with that of WT SHOC2, we measured the half-life of WT and the SHOC2 (p.M173I) mutant in Cos-LV1 cells. To accomplish this, Cos-LV1 cells were transiently transfected with WT SHOC2-tRFP or the SHOC2 (M173I)-tRFP mutant. We found that the M173I substitution in SHOC2 did not affect the half-life of this long-lived protein (∼12 h) (Supp. Fig. S2C), suggesting that protein stability was not affected by the M173I variant.  To examine whether the reduced ability of SHOC2-M173I to enhance activity of the ERK1/2 pathway is due to altered subcellular localization of SHOC2, we investigated the subcellular localization of SHOC2-M173I. Fluorescence microscopy analysis showed that SHOC2-tRFP and SHOC2 (M173I)-tRFP were distributed in both the cytosol and nucleus (Supp. Fig. S2D). We next compared the cytosolic/nuclear distribution of the SHOC2-M173I mutant to WT SHOC2 by subcellular fractionation of Cos-LV1 cells transiently expressing either WT SHOC2-tRFP or SHOC2 (M173I)-tRFP proteins (Supp. Fig. S2E). Our results suggest that the subcellular distribution of SHOC2 (M173I) variant was unaltered.  Next, we investigated whether the M173I mutation altered the capacity of SHOC2 to tether RAS and RAF-1 proteins to the scaffold complex. We expressed WT and the SHOC2 (p.M173I) variant in 293FT cells and tested their ability to interact with HA-M-RAS and RAF-1. The capacity of SHOC2-M173I to form a complex with M-RAS and RAF-1 was comparable to that of WT SHOC2 (Fig. 2C), showing that the p.M173I substitution does not affect SHOC2–RAS–RAF-1 complex assembly. These observations were not entirely surprising as we have previously determined that an N-terminal domain of SHOC2 mediates binding of RAS [Jeoung et al., 2013].  Since SHOC2 has been shown to function as a regulatory subunit of PP1c resulting in activation of RAF-1 kinase [Rodriguez-Viciana et al., 2006b], we hypothesized that the M173I mutation might affect the SHOC2 interaction with PP1c. Although SHOC2 does not contain a canonical PP1c binding motif [R/K]-x(0,1)-[V/I]-x-F, the sequences in LRR4 of SHOC2 contained a hydrophilic motif 169KKLR172 that is found in several PP1c-binding proteins [Bennett et al., 2006]. To determine whether p.M173I is part of a PP1c-binding motif and whether mutation affects SHOC2-PP1c binding affinity, we transfected 293FT cells with the WT or the M173I mutant of GST-SHOC2 and tested for their interaction with PP1c using immunoprecipitation. Results in Figure 2C (lanes 2 and 4) show that the GST-tagged M173I mutant of SHOC2 failed to precipitate endogenous PP1c effectively, suggesting that the M173I substitution alters the interaction of SHOC2 and PP1c. The function of PP1c in the SHOC2–RAS–RAF-1 complex is associated with activation of RAF1 kinase activity [Rodriguez-Viciana et al., 2006b]. Thus, we assessed whether phosphorylation of RAF-1 at Ser 338, a step that is required for efficient RAF-1 activation, and levels of phospho-ERK were altered concomitantly. In cells expressing WT SHOC2-tRFP, stimulation with EGF led to a dramatic increase in RAF-1-(S338) phosphorylation (Fig. 2D and E). Conversely, in cells expressing SHOC2 (M173I)-tRFP, only a mild increase in RAF-1-(S338) phosphorylation was observed, supporting the hypothesis that the M173I substitution in SHOC2 altered the interaction of SHOC2 with PP1c thus contributing to decreased RAF-1/ERK1/2 activity.  The c.519G>A; p.M173I substitution reported here is only the second SHOC2 mutation reported in patients with Rasopathy symptoms. Phenotypic features of patients with the M173I mutation are significantly different from those reported for patients with the c.4A>G; p.S2G SHOC2 mutation. While NS/LAH patients usually present with significant short stature, intellectual disability, cardiac anomalies, and multiple cutaneous findings [Cordeddu et al., 2009; Gripp et al., 2013], our proband has craniofacial features suggestive of NS and sparse slow-growing hair that is more characteristic of CFC. She does not have short stature, a cardiac anomaly, hyperpigmentation, ichthyosis, or loose anagen hair. Her developmental delays are relatively mild compared with many patients with CFC or NS/LAH, especially considering that both of her parents have mild intellectual disability. Her father has tall stature and very subtle clinical features that do not clearly denote a Rasopathy, other than his sparse hair and clubbed nails. We thus conclude that SHOC2 mutations can cause a spectrum of Rasopathy phenotypes of varying severity. This is not surprising, given that variable phenotypes have been reported for most other Rasopathy-associated genes. A recent study by Justino et al. (2014), re-emphasized the clinical heterogeneity of all neuro-CFC syndromes, the absence of consensus on distinct diagnostic criteria and the necessity of multigene panels for molecular diagnosis. Clinical heterogeneity associated with the SHOC2 c.4A>G; p.S2G variant has also been reported [Cordeddu et al., 2009; Hoban et al., 2012; Justino et al., 2014].  The p.M173I SHOC2 substitution causes loss of function in the ERK1/2 pathway. While the majority of mutations associated with Rasopathies are gain-of-function mutations, loss-of-function mutations in PTPN11 have been reported to cause Leopard syndrome, a Rasopathy with the added feature of lentigenes [Kontaridis et al., 2006]. CFC can be caused by both loss- and gain-of-function mutations in B-RAF. [Rodriguez-Viciana et al., 2006a]. This implies that the deregulation of the ERK1/2 pathway rather than upregulation is the critical factor in the causation of Rasopathies.  In summary, our results agree with a substantial set of data suggesting that deregulation of the ERK1/2 pathway has a significant effect on development. Our report of patients with mild Rasopathy symptoms caused by a novel SHOC2 mutation should improve the diagnostic accuracy of patients with findings in the Rasopathy spectrum and provide new evidence for understanding the pathogenesis of these disorders. Our data show that p.M173I is a functional variant by demonstrating that it causes an alteration in ERK1/2 activity and that reduced binding of SHOC2 to the catalytic subunit of PP1c is the cause of the ERK1/2 pathway dysregulation. A recent study by Young et al. (2013) proposed two PP1c binding motifs that are closely localized on the exposed convex side of SHOC2, SLVK, and KIPF [Young et al., 2013]. An attractive hypothesis for our observations is that these motifs are alternatively utilized for efficient PP1c recruitment to the SHOC2 scaffolding complex. Future studies will be needed to confirm this conclusion, to decipher the physiological role of the p.M173I SHOC2 variant in development, and to generate a comprehensive model of SHOC2 function.  : 3\n",
      " Abstract  DICER1 plays a critical role in microRNA (miRNA) biogenesis. Recurrent somatic 'hotspot' mutations at the four metal-binding sites within the RNase IIIb domain of DICER1 were identified in ovarian sex cord-stromal tumours and have since been described in other paediatric tumours. In this study, we screened the RNase IIIb domain of DICER1 in 290 endometrial tumours and identified six cases with hotspot mutations, including two cases affected by an atypical G1809R mutation directly adjacent to a metal-binding site. Using Illumina and Sanger targeted resequencing, we observed and validated biallelic DICER1 mutations in several cases with hotspot mutations. Through in vitro DICER1 cleavage assays, small RNA deep sequencing and real-time PCR, we demonstrated that mutations adding a positively charged side chain to residue 1809 have similar detrimental effects on 5p miRNA production to mutations at the metal-binding sites. As expected, 5p miRNAs were globally reduced in tumours and cell lines with hotspot mutations. Pathway analysis of gene expression profiles indicated that genes de-repressed due to loss of 5p miRNAs are strongly associated with pathways regulating the cell cycle. Using a Dicer1-null mouse cell line model, we found that expression of DICER1 hotspot mutants promoted cell proliferation, whereas wild-type (WT) DICER1 inhibited cell proliferation. Furthermore, targets of let-7 family miRNAs are enriched among the up-regulated genes, suggesting that loss of let-7 may be impacting downstream pathways. Our results reveal that DICER1 hotspot mutations are implicated in common malignancies and may constitute a unique oncogenic pathway. Copyright © 2015 Pathological Society of Great Britain and Ireland. Published by John Wiley & Sons, Ltd. Introduction  MicroRNAs (miRNAs) are ∼22 nucleotide (nt), single-stranded, non-coding RNAs that post-transcriptionally regulate gene expression in animals and plants [1]. miRNAs play crucial roles in maintaining normal cellular functions, such as differentiation, proliferation and apoptosis [2]. The biogenesis of miRNAs starts with the transcription of long primary miRNAs (pri-miRNAs), which are then processed into ∼70 nt hairpin precursor miRNAs (pre-miRNAs) by the endoribonuclease Drosha. Pre-miRNAs are subsequently transported to the cytoplasm by Exportin 5 (XPO5), where they are further trimmed by DICER1, another endoribonuclease. The two RNase III domains of DICER1, which form an intramolecular dimer, cleave the pre-miRNA hairpin to generate miRNA duplexes composed of 5p and 3p miRNAs that emanate from the 5′ and 3′ arms, respectively, of the hairpin [3, 4]. The 'guide strand' will be preferentially incorporated into an miRNA-induced silencing complex (miRISC) and interacts with target mRNAs, whereas the 'passenger strand' is often degraded. Since a single defect in the components of the miRNA biogenesis pathway could potentially affect the entire miRNA population, it is not surprising that disruptions of this pathway are associated with a variety of human malignancies [5].  DICER1 germline mutations are known to predispose carriers to cancer [5], such as familial pleuropulmonary blastoma (PPB) [6]. Furthermore, heterozygous germline DICER1 mutations have been identified in PPB-related familial tumour–dysplasia syndrome (PPB–FTDS), multinodular goitre (MNG), Sertoli–Leydig cell tumour (SLCT), cystic nephroma (CN), Wilms' tumour, cervical embryonal rhabdomyosarcoma (cERM) and pineoblastoma [7-10]. We recently identified recurrent somatic DICER1 hotspot mutations in rare non-epithelial ovarian tumours [11]. Subsequent studies have shown that hotspot mutations at the four metal-binding sites in the RNase IIIb domain (E1705, D1709, D1810 and E1813) impair DICER1's ability to generate mature 5p miRNAs [4, 11, 12]. More recently, DICER1 hotspot mutations were identified in tumours from PPB and PPB–FTDS cases possessing germline DICER1 mutations, suggesting a two-hit hypothesis of tumourigenesis [11, 13-16].  The mutational analyses of DICER1 have previously focused on rare paediatric cancers [6-11, 13-16]. In this study, we investigated the prevalence of DICER1 mutations in a spectrum of more common malignancies. We screened for DICER1 hotspot mutations in 290 endometrial tumours, and evaluated the two-hit hypothesis in cases with hotspot mutations. We identified an atypical hotspot mutation, G1809R, which is immediately adjacent to the metal-binding residue D1810, and characterized its functional impact on miRNAs and downstream signalling pathways. Using a Dicer1-null mouse fallopian tube carcinoma-derived cell line [17], we demonstrated that DICER1 hotspot mutations abolish the inhibitory effects of DICER1 on cell proliferation. Our data also suggested that loss of let-7 family miRNAs may contribute to the oncogenic properties of DICER1 hotspot mutations. Materials and methods  Patient samples and cell culture  All the patient tumour and blood samples were collected from the Ovarian Cancer Research Programme (OvCaRe) tissue bank (Vancouver, British Columbia, Canada) under ethical approvals from the University of British Columbia and BC Cancer Agency Research Ethics Board [11].  DKO172 cells, derived from a Dicer1/Pten double-knockout mouse fallopian tube tumour model [17], were maintained in Dulbecco's modified Eagle's medium (DMEM)/F12 (Hyclone) supplemented with 10% fetal bovine serum (FBS) at 37 °C and 5% CO2. DNA and RNA extraction  DNA was extracted using DNeasy Blood & Tissue or QIAamp DNA FFPE Tissue Kits (Qiagen), and RNA was extracted using miRNeasy or miRNeasy FFPE kits (Qiagen), according to the manufacturer's protocols. PCR and Sanger sequencing  Primer sets and the protocol used to sequence the RNase IIIb domain of DICER1 have been described in previous studies [11, 18]. Primer sets for DICER1 cDNA sequencing are listed in Table S2 (see supplementary material). PCR products were sequenced on an ABI Prism 3130xl Genetic Analyser (Applied Biosystems) and visualized using 4peaks (Mek&Tosj).  To determine whether two DICER1 mutations from the same sample are in trans or in cis, a single amplicon encompassing two mutations was amplified and cloned into pCR2.1, using a TOPO TA-Cloning Kit (Life Technologies). Ten colonies of each sample were Sanger sequenced for both hotspot and non-hotspot mutations. Targeted DICER1 amplicon resequencing  The sequencing protocol was adopted from Forshew et al [19]. Target-specific amplification was performed on a microfluidic system using the 48.48 Access Array Integrated Fluidic Circuit, following the manufacturer's user guide (Fluidigm; for more information, see supplementary material, Supplementary materials and methods and Table S3). Small RNA sequencing  Total RNA was quantified using the Qubit® RNA BR Assay Kit (Life Technologies) and diluted to 200 ng/ μl. Small RNA libraries were prepared using the TruSeq Small RNA Sample Preparation Kit (Illumina), according to the manufacturer's instructions (for more information, see supplementary material, Supplementary materials and methods). Plasmid construction  A series of DICER1 mutants were generated from pDONR221-wild-type DICER1 (WT) [4] by site-directed mutagenesis (Agilent) and shuttled into pLD-hygro-EnVM vector [20] through Gateway LR-clonase reaction (Invitrogen). Sanger sequencing verified the DICER1 mutations in the vectors. GFP-expressing vectors were used as the vector controls. Western blotting  Cell lysates (30  μg) were separated on 8% sodium dodecyl sulphate–polyacrylamide gels (SDS–PAGE) and then were transferred to nitrocellulose membranes. The DICER1 antibody (Santa Cruz, sc-30226) was used to detect both mouse and human DICER1 forms. Vinculin (Sigma, V9131) was used as the loading control. DICER1 in vitro cleavage assay  Pre-let-7c miRNA was generated by in vitro transcription of a DNA template oligo (5′-GGAAAGCTAGAAGGTTGTACAGTTAACTCCCAGGGTGTAACTCTAAACCATACAACCTACTACCTCCTATAGTGAGTCGTATTACAT-3′) annealed to the 5′-ATGTAATACGAC TCACTATAG-3′ oligo using the MEGAscript ® T7 Transcription Kit (Invitrogen). The RNA transcript was gel-purified and 5′-32P end-labelled. The preparation of DICER1 enzymes and in vitro DICER cleavage assays were performed as previously described [11]. The resulting products were visualized on a storage phosphor screen, using a FLA-7000 instrument (GE Healthcare). miRNA and mRNA real-time PCR  The expression of individual miRNAs was quantified using the miRCURY LNA Universal RT microRNA PCR system (Exiqon), in triplicate, following the manufacturer's instructions. The endogenous small nuclear RNA RNU1A1 was used for normalization. The expression of mRNA transcripts was determined using cDNA converted from total RNA, followed by real-time PCR with Power SYBR ® Green Master Mix (Life Technologies) [4]. The mouse housekeeping gene Actb was used for normalization. The relative levels were calculated using the ΔΔCt method [21]. Cell growth and gene expression analysis  DKO172 cells were transfected with 2  μg pLD-hygro-EnVM vectors expressing WT DICER1, D1709N, G1809R or GFP, using the TransIT ®-LT1 transfection reagent (Mirus), followed by selection in 300  μg/ml hygromycin for 84 h. Cells were then harvested for RNA extraction or replated in 96-well plates at 2000 cells/well. Cell growth was monitored by the IncuCyte ZOOM Live Content Imaging System (Essen BioScience). The gene expression profiles were determined using the Agilent SurePrint G3 mouse gene expression microarrays, according to the manufacturer's instructions. Array data were filtered, normalized, analysed and visualized by GeneSpring GX (v. 12, Agilent). Raw data can be accessed via NCBI Gene Expression Omnibus (GSE65092). Results  DICER1 hotspot mutations in endometrial tumours  To evaluate the prevalence of DICER1 mutations in common malignancies, we examined the Cancer Genome Atlas (TCGA) cBioPortal database (http://www.cbioportal.org) and discovered that endometrial carcinomas have the highest frequency of DICER1 mutations (see supplementary material, Figure S1) [22]. Within endometrial cancers, 23/248 carcinomas and 1/56 carcinosarcoma contain 36 and one DICER1 mutations, respectively; 19% (7/37) of DICER1 mutations are hotspot mutations (see supplementary material, Table S1). Among 30 non-hotspot mutations, 28 mutations are missense, one mutation is nonsense, and one mutation is a frameshift deletion (see supplementary material, Table S4). Using a Bernoulli approximation, we estimated the probability of 37 observed DICER1 mutations occurring at random seven or more times at any one of four metal-binding sites, as 1.64E-12, suggesting that hotspot mutations are selectively enriched during endometrial cancer development.  To determine whether hotspot DICER1 mutations are a recurrent feature of endometrial carcinomas, we screened the RNase IIIb domain of DICER1 in 290 endometrial tumours from the OvCaRe tumour bank and a carcinosarcoma of the ovary reported to have a hotspot mutation [11]. The sequencing results were summarized in Table 1 and Figure 1A. In total, eight heterozygous missense mutations in the RNase IIIb domain of DICER1, affecting three sites, E1705, G1809 and E1813, were found in seven cases. Mutations at E1705 and E1813 were previously referred to as 'hotspot' mutations and their detrimental effects on 5p miRNA production have been characterized [4, 11, 23]. Two heterozygous mutations (c.5115A > C and c.5113G > C) were found at the same E1705 codon in one sample. Notably, we observed a recurrent mutation at G1809 in two cases. Although not a metal-binding site, glycine at residue 1809 is highly conserved across species and locates within close proximity to the metal ion in the catalytic valley (Figure 1B; see also supplementary material, Figure S2). The glycine-to-arginine change at residue 1809 is predicted to be damaging (by PolyPhen-2 and SIFT) and highly functional (by Mutation Assessor), suggesting that it may be a hotspot mutation and function equivalently to the alterations at the metal-binding residues.  Table 1. Summary of DICER1 mutations Sample\tTumour type\tMutation in the RNase IIIb domain\tAdditional Mutation\tMutation in cis or trans *Germline mutation imageIdentified in a previous study [11] A\tEndometrioid\t c.5115A > C (E1705D)c.5113G > C (E1705Q)   \ttrans B\tEndometrioid\tc.5425G > A (G1809R)\tc.2461C > T (R821C)\ttrans C\tCarcinosarcoma (uterus)\tc.5425G > A (G1809R)\tc.2516C > T (S839F)*\ttrans Dimage\tCarcinosarcoma (ovary)\tc.5438A > G (E1813G)\tSuspected inactivation\ttrans E\tEndometrioid\tc.5439G > T (E1813D)\t–\t– F\tEndometrioid\tc.5439G > T (E1813D)\t–\t– G\tEndometrioid\tc.5113G > A (E1705K)\t–\t– Figure 1.  Figure 1. Open in figure viewerDownload Powerpoint slide Overview of DICER1 mutations in tumours. (A) A schematic of DICER1 domains with mutations identified through Sanger and Illumina targeted resequencing. The fill colour of each mutation represents individual tumour samples; note that some samples have two mutations. (B) Similar to the four metal-binding sites (1705, 1709, 1810 and 1813) in the RNase IIIb domain of DICER1, the glycine at residue 1809 is highly conserved across species Evaluation of biallelic DICER1 mutations  To evaluate a 'two-hit' hypothesis for DICER1 in these tumours, we performed targeted sequencing for the entire DICER1 exome of all seven tumours with hotspot mutations. In addition, the entire DICER1 cDNA of those cases was analysed by Sanger sequencing. Potential biallelic DICER1 mutations were identified in four samples. In sample A, two heterozygous mutations (c.5115A > C and c.5113G > C) were shown to be in trans by deep sequencing (Figure 2A), leading to two overlapping missense mutations, E1705D and E1705Q, respectively. In sample D, the E1813G mutation was identified by initial genomic DNA (gDNA) sequencing, and subsequent cDNA sequencing of an amplicon spanning the RNase IIIb domain revealed two products, a long one containing the canonical DICER1 sequence and a shorter one with a 163 nt deletion missing exon 25 (Figure 2B). This result agreed with a previous observation that the c.5438A > G (E1813G) mutation in DICER1 promotes inefficient exon 25 exclusion, producing both a full-length transcript with c.5438A > G and an exon-25-depleted transcript [13]. Interestingly, when the longer cDNA amplicon was sequenced, only c.5438A > G but no wild-type (WT) sequence was detected (Figure 2B), suggesting that the expression of WT DICER1 is lost, probably due to either a cryptic genomic aberration or epigenetic silencing.  Figure 2.  Figure 2. Open in figure viewerDownload Powerpoint slide Potential biallelic DICER1 mutations in tumours. (A) In case A, Sanger sequencing identified two point mutations (c.5115A > C and c.5113G > C) at the same codon (on the left) and Illumina deep sequencing suggested that they are in trans (on the right). The sequencing reads were aligned to the hg19 reference sequence, which is the reverse complement of the DICER1 sequence, as the DICER1 gene is located on the negative strand. (B) In case D, PCR amplification of cDNA spanning the RNase IIIb domain and subsequent Sanger sequencing revealed two products, one containing the canonical DICER1 sequence and the other, shorter, product with a 163 nt deletion missing exon 25. Notably, only the c.5438A > G (E1813G) mutation but no WT sequence was detected in the full-length transcript. (C) In case C, two heterozygous mutations, c.5425G > A (G1809R) and c.2516C > T (S839F), were confirmed to be in trans by sequencing the cloned cDNA encompassing both mutations. WT, wild-type; gDNA, genomic DNA; cDNA, complementary DNA To determine the allelic relationship of the two heterozygous mutations in cases B (G1809R and R821C) and C (G1809R and S839F), we cloned and sequenced cDNAs spanning both mutations and confirmed that the two mutations from both cases were in trans (Figure 2C; see also supplementary material, Figure S3A). R821C, a somatic mutation, was predicted to be damaging by PolyPhen-2 and SIFT. S839F is present in constitutional DNA and thus is a germline mutation (data not shown). Although S839F was predicted to be tolerated, ectopic expression of the S839F mutant was noticeably lower at the protein level but not at the mRNA level, compared to WT or other mutants (see supplementary material, Figure S4A, B). This suggests that the G1809R mutant may be the predominantly expressed form of DICER1, despite being compound heterozygous with S839F in sample C. In samples E, F and G, the DICER1 hotspot mutations were heterozygous in both cases, suggesting the presence of an intact WT allele (see supplementary material, Figure S3B–D). Mature miRNA expression in tumours with DICER1 hotspot mutation  To evaluate the impact of DICER1 hotspot mutations on miRNA expression profiles, we performed deep sequencing on small RNAs in all seven tumours with DICER1 hotspot mutations and five endometrial tumours without hotspot mutations. As expected, overall 5p miRNA expression was significantly reduced in tumours with DICER1 hotspot mutations [mean ± standard error (SE) = 62 ± 5.2; n = 7] compared to tumours without hotspot mutations (mean ± SE = 78 ± 1.8; n = 5; p = 0.0212; Figure 3A). We also analysed the miRNA expression data from the TCGA endometrial cancer cohort, and observed similar results (see supplementary material, Figure S5): the entire 5p miRNA population was reduced in tumours with DICER1 hotspot mutations (mean ± SE = 54.9 ± 4.7; n = 6) compared to either tumours with DICER1 non-hotspot mutations (mean ± SE = 66.7 ± 3.1; n = 14; p = 0.052) or tumours without DICER1 mutations (mean ± SE = 67.7 ± 0.5; n = 375; p = 0.001). However, the reduction of 5p miRNAs in tumours with hotspot mutations was not as drastic as in DICER1-null cell lines that express hotspot mutants [4, 12], which may be due to other confounding factors in tumours, such as the presence of WT allele, DICER1 variant allele frequency, additional genetic aberrations, intratumour heterogeneity, tumour cell content and cell-free circulating miRNAs.  Figure 3.  Figure 3. Open in figure viewerDownload Powerpoint slide Small RNA sequencing of tumours with or without DICER1 hotspot mutations. (A) The entire 5p miRNA population was significantly reduced in seven tumours with hotspot mutations (mean ± SE = 62 ± 5.2) compared to five tumours without hotspot mutations (mean ± SE = 78 ± 1.8). (B) The 5p:3p ratio of miR-99b, a 5p_strand predominant miRNA, was drastically reduced in tumours harbouring DICER1 hotspot mutations (mean ± SE = 118.8 ± 60.7) compared to tumours without hotspot mutations (mean ± SE = 486.9 ± 31.9). (C) The 3p:5p ratio of miR-27b, a 3p_strand predominant miRNA, was slightly increased in tumours possessing DICER1 hotspot mutations (mean ± SE = 533.9 ± 63.1) compared to tumours without hotspot mutations (mean ± SE = 425.5 ± 53.8). Individual miRNAs were normalized by dividing raw reads by the total number of miRNA aligned reads in the library × 1 million. The p values were calculated by independent samples t-test Among specific miRNAs, the relative abundance between 5p- and 3p- strands derived from the same pre-miRNA varied greatly. In general, miRNAs can be classified into two groups: 5p_strand predominant miRNAs (eg miR-99b, miR-181a, let-7a) or 3p_strand predominant miRNAs (eg miR-27b, miR-221, miR-199a). The 5p:3p ratio of 5p_strand predominant miRNAs was drastically reduced in tumours with hotspot mutations compared to tumours without hotspot mutations (Figure 3B; see also supplementary material, Figure S6A), while the 3p:5p ratio of 3p_strand predominant miRNAs was slightly increased in tumours with hotspot mutations (Figure 3C; see also supplementary material, Figure S6B). Since DICER1 hotspot mutations severely impair the production of 5p miRNAs [4, 11, 12], it is not surprising that mainly 5p_strand predominant miRNAs were affected by the defects. Characterization of the functional impact of G1809R on miRNA biogenesis  To test the hypothesis that the G1809R mutation impairs the enzymatic function of the DICER1 RNase IIIb domain, we compared the in vitro cleavage activity of this mutant to a well-characterized hotspot mutation, D1709N [4, 11]. Both G1809R and D1709N DICER1 mutants failed to cleave the 5′-labelled pre-let-7c miRNA (Figure 4A) at the RNase IIIb processing site in vitro, as shown by the nearly absent amount of the 22 nt 5p let-7c miRNA and the accumulation of the 45 nt let-7c intermediate RNA, whereas WT DICER1 produced the expected 22 nt 5p let-7c miRNA (Figure 4B). These data suggested that the G1809R mutation adversely affects 5p miRNA production, similar to the D1709N and other mutations at metal-binding sites [4, 11].  Figure 4.  Figure 4. Open in figure viewerDownload Powerpoint slide Functional impact of DICER1 mutations at residue 1809 on miRNA production. (A) Pre-let-7c structure showing cleavage sites of DICER1 RNases IIIa and IIIb: sequence in red indicates the mature 5p let-7c miRNA released by DICER1 activity; green sequence shows the 3p miRNA product; *5′-end 32P label. (B) The DICER1 G1809R mutation impaired RNase IIIb activity in vitro. Assays were performed on a pre-let-7c RNA substrate (see part A). The G1809R and D1709N mutants lacked RNase IIIb cleavage activity, resulting in a 45 nt intermediate cleavage product (green arrow). WT DICER1 had IIIb cleavage activity, resulting in the expected 22 nt mature let-7c miRNA (red arrow). A mock control (substrate only in reaction mix) was used to assess degradation of the substrate during the reaction. The full gel containing the intervening lanes can be found in Figure S7A (see supplementary material) to establish band alignment between the lanes. (C) Small RNA sequencing revealed that the 5p miRNA population was reduced from 50% to < 20% of total miRNAs in cells expressing G1809R and G1809K, which is functionally equivalent to a bona fide hotspot mutation D1709N. The 5p population was unchanged in G1809A compared to WT. (D, E) Real-time PCR analysis of selected miRNAs. Two 5p miRNAs, miR-99b and let-7, were virtually undetectable in cells expressing D1709N, G1809R and G1809K (D). Two 3p miRNAs, miR-22 and miR-27b, were present but slightly reduced in cells expressing D1709N, G1809R and G1809K (E). As expected, no miRNA was detected in DKO172 cells or DKO172 cells expressing GFP. The endogenous small nuclear RNA RNU1A1 was used as an internal control. Each sample was measured in triplicate; data are shown as mean ± SE To further evaluate the ex vivo functional impact of DICER1 mutations at residue 1809 on miRNA production, we performed small RNA deep sequencing on Dicer1-null DKO172 cells [17] transfected with vectors expressing WT or mutant DICER1 in which the uncharged glycine at residue 1809 was replaced with alanine, arginine and lysine. The side-chain of alanine is a small uncharged methyl group, while both arginine and lysine have bulky, positively charged side-chains (see supplementary material, Figure S7B). We hypothesized that G1809A, an amino acid substitution that does not change the charge of the side-chain and was not seen in tumours, would not interfere with DICER1 function, whereas both tumour-associated G1809R and the similarly charged G1809K substitution would disrupt the enzymatic activity of the RNase IIIb domain, leading to loss of 5p miRNAs. The small RNA sequencing results revealed that ∼ 50% of total miRNAs are 5p miRNAs in DKO172 cells expressing WT DICER1 or G1809A, while the 5p miRNA population was drastically reduced to < 20% of total miRNAs in cells expressing G1809R or G1809K, similar to the cells expressing a bona fide hotspot mutant D1709N (Figure 4C). The global reduction of miRNA species in all sequencing reads was mainly due to the loss of 5p miRNAs (see supplementary material, Figure S7C). We also observed a partial decrease of 3p miRNAs in cells expressing hotspot mutants, which agrees with a previous finding [12]. To confirm the results from small RNA sequencing, we examined the expression of individual miRNAs, using real-time PCR. Complete loss of 5p miRNAs, ie let-7c and miR-99b, and partial reduction of 3p miRNAs, ie miR-22 and miR-27, were observed in DKO172 cells expressing G1809R, G1809K or D1709N mutants (Figure 4D, E). Therefore, our data demonstrated that a positively charged side-chain at residue 1809 would have detrimental effects on the cleavage of 5p miRNAs from precursor miRNAs. DICER1 hotspot mutation affects downstream pathways associated with cell proliferation  Loss of miRNAs should, in part, directly de-repress mRNA levels and therefore lead to increased target gene expression. In order to understand the impact of losing 5p miRNAs on downstream cell signalling pathways, we profiled mRNA expression in DKO172 cells expressing WT DICER1 or mutants. Consistent with previous results, two DICER1 hotspot mutants, D1709N and G1809R, clustered together by unsupervised hierarchical clustering, indicating that they have very similar expression profiles and are functionally equivalent (Figure 5A). Furthermore, DICER1 mutants were clustered further away from WT DICER1 than the GFP control, indicating that the gene expression profiles of hotspot mutants are more similar to the profile of Dicer1-null than to that of WT DICER1 (Figure 5A). To identify genes differentially expressed due to hotspot mutations, gene expression profiles of D1709N and G1809R were compared to that of WT; > 90% of up-regulated genes (fold change ≥ 2) in D1709N and G1809R were the same when compared to WT (Figure 5B), supporting the notion that the two DICER1 mutants function alike. Ingenuity Pathway Analysis was used to analyse the cellular functions of 832 genes that were up-regulated in both mutants, and revealed that the most significantly enriched gene ontology categories were associated with cell cycle progression (Table 2; see also supplementary material, Table S5).  Figure 5.  Figure 5. Open in figure viewerDownload Powerpoint slide The effects of DICER1 hotspot mutations on downstream pathways. (A) Unsupervised hierarchical clustering of gene expressing profiles: the dendrogram was generated by GeneSpring 12.0, using average linkage clustering and the uncentred correlation metric. DKO172 cells were seeded in six-well plates at 200 000 cells/well the day before transfection, and then transfected with 2  μg pLD-hygro-EnVM vectors expressing WT DICER1, D1709N, G1809R or GFP, using the TransIT ®-LT1 transfection reagent (Mirus). The cells were replated into P60 plates after 24 h and then selected with 300  μg/ml hygromycin for 84 h. Total RNA was extracted for the gene expression array. (B) Venn diagram of up-regulated genes in D1709N and G1809R compared to WT DICER1, respectively; FC, fold change. (C) The effects of DICER1 hotspot mutations on cell proliferation. Transfected and hygromycin-selected DKO172 cells were seeded at 2000 cells/well into a 96-well plate. Cell proliferation rate was assessed by phase-contrast imaging, using the IncuCyte ZOOM system. Graphs show representative results from one of three independent experiments, each performed in sextuplicate and presented as mean ± SE. (D) The top five most abundant 5p miRNA families in DKO172 cells expressing WT DICER1, and their corresponding expression in cells expressing D1709N and G1809R. The reads from members of the same miRNA family were combined and then compared to the total miRNA reads Table 2. Top canonical pathways identified by Ingenuity Pathway Analysis Top canonical pathway\tp The p value was calculated with the Fisher's Exact Test. The significance indicates the probability of association of genes from the dataset with the canonical pathway by random chance alone. Oestrogen-mediated S-phase entry\t1.42E-07 Cyclins and cell cycle regulation\t8.85E-05 Antiproliferative role of TOB in T cell signalling\t1.08E-03 DNA damage-induced 14–3–3σ signalling\t4.38E-03 GADD45 signalling\t5.09E-03 Cell cycle: G1–S checkpoint regulation\t6.39E-03 Mitotic roles of polo-like kinase\t6.81E-03 To evaluate the effects of DICER1 hotspot mutations on cell proliferation, DKO172 cells were transiently transfected with WT DICER1, mutant and GFP control constructs. WT DICER1 strongly repressed cell proliferation, whereas the D1709N and G1809R mutants displayed a moderate growth advantage over Dicer1-null cells expressing the GFP control (Figure 5C). This is in agreement with the data showing that DICER1 hotspot mutations up-regulate genes promoting cell proliferation (Table 2). Therefore, our data suggest that hotspot mutations impair the tumour-suppressive effect of DICER1 in DKO172 cells.  Studies have shown that the miRNA seed sequence, nucleotides 2–7 from the 5′ end of a miRNA, is essential for target recognition [24, 25]. Many miRNA genes have multiple isoforms that share conserved seed regions and can be grouped into specific miRNA families. Based on miRNA seed sequences, Ingenuity Pathway Analysis predicted that the let-7 family is a key regulator and directly regulates 53/832 genes that were up-regulated in cells expressing DICER1 hotspot mutants (see supplementary material, Table S6). This is not surprising, because the let-7 family, composed of let-7-a1 , −a2 , −b , −c1 , −c2 , −d , −e , −f1 , −f2 , −g , −i and miR-98 (www.targetscan.org), is the most abundant 5p miRNA family in DKO172 cells expressing WT DICER1 and is completely lost in hotspot mutant-expressing cells (Figure 5D; see also supplementary material, Figure S9). The let-7 family miRNAs are well known tumour suppressors that repress cell proliferation pathways [26]. Targets of let-7 that were increased in DICER1 hotspot mutants, such as CDC25A, CCNA2, CCNE1, CDK1, SKP2, LIN28A/B, etc., are important regulators of cell cycle progression. Discussion  In this study, we report that DICER1 hotspot mutations are present in ∼2% of endometrial tumours. This is consistent across both the TCGA dataset (7/304) and our own local tumour bank (6/290; see supplementary material, Tables S1, S7). DICER1 hotspot mutations, although rare, were also identified in brain, colorectal and thyroid cancers (see supplementary material, Table S1). Aksoy and colleagues [27] also reported similar results using data from the TCGA in a preprint when we were preparing this manuscript. Together, our findings suggest that the DICER1 hotspot mutations may impact tumourigenesis beyond the spectrum of rare paediatric cancers [8-13, 15-18]. DICER1 hotspot mutations and subsequent miRNA/mRNA dysregulation may therefore constitute a common oncogenic pathway in a small subset of endometrial tumours.  We identified a recurrent G1809R DICER1 mutation, which is located at a highly conserved site and is in close spatial proximity to the metal-binding residues (E1810 and D1813) in the RNase IIIb domain of DICER1. During the preparation of our manuscript, several studies identified G1809R mutations in both PPB and Wilms' tumour [16, 28-30] and a G1809V mutation in Wilms' tumour [30]. We demonstrated that the G1809R mutation impairs 5p miRNA production in vitro with RNA cleavage assays (Figure 4). We characterized the impact of G1809R on miRNA production in an isogenic cell line model, and confirmed that G1809R is functionally equivalent to previously characterized hotspot mutations [4, 11, 12]. We also found that not all mutations at residue 1809 will impair RNase IIIb cleavage activity; for example, substituting glycine with alanine, a small uncharged residue, did not impair miRNA processing, while the substitution of large positively charged residues, such as arginine or lysine, led to loss of 5p miRNA production (Figure 4). In one case, G1809R was compound heterozygous with a germline S839F mutation. This is of particular interest, because the heterozygous S839F DICER1 mutation has been reported to be highly penetrant in a large family with MNG [7]. Western blot results suggested that ectopic expression of S839F mutant reduces DICER1 protein levels (see supplementary material, Figure S4), and thus may produce a hypomorphic phenotype in vivo, although the detailed mechanism of this mutation warrants further investigation.  In summary, we have shown that four of seven tumours with DICER1 hotspot mutations potentially fitted the two-hit hypothesis for DICER1 involvement in tumourigenesis. In the TCGA endometrial cohort, five of six cases with hotspot mutations possessed additional DICER1 mutations (see supplementary material, Table S1). However, whether those mutations were in trans or deleterious remained to be determined. No loss of heterozygosity (LOH) was observed in tumours with DICER1 hotspot mutations (see supplementary material, Figure S8). Overall, down-regulation of 5p miRNAs was observed in tumours with DICER1 hotspot mutations (Figure 3A; see also supplementary material, Figure S5). The 5p miRNA abundance in the total miRNA population was lower in tumours with 'two-hit' (mean ± SE = 57.5 ± 7.8; n = 4) compared to its level in tumours with a single hotspot hit (mean ± SE = 68 ± 6; n = 3). However, the difference was not significant (p = 0.36), which may be due to the limited sample size. Future studies are needed to address whether hotspot alleles can compete with WT alleles for miRNA processing.  Early mouse models suggested that Dicer1 might function as a haplo-insufficient tumour-suppressor gene, in that loss of a single allele of Dicer1 promotes tumour formation but loss of both alleles stalls tumour development in an oncogenic Kras or retinoblastoma-sensitized background [31, 32]. Initial clinical studies found that PPB and PPB–FTDS patients carry heterozygous DICER1 germline mutations and no homozygous loss of DICER1 was observed [6, 33], which supported the DICER1 haplo-insufficiency hypothesis. However, this model was contradicted by recent studies showing that Dicer1-null cells (KrasG12DTrp53 −/ −Dicer1 −/ −) from a mouse sarcoma cell line are competent for tumour formation, and that Pten/Dicer1 double-knockout mice develop primary fallopian tube tumours [17, 34]. These divergent results suggest that the DICER1 haplo-insufficiency hypothesis may only apply to specific tissues or tumour models.  The two-hit hypothesis for DICER1 was initially proposed after identification of DICER1 hotspot mutations in tumours from PPB and PPB–FTDS patients with germline DICER1 mutations [11, 13, 29]. More recently, frequent biallelic DICER1 mutations were identified in sporadic (8/11) and germline PPB (12/12) from two independent studies [16, 28]. The biallelic DICER1 mutations predominantly involved a loss-of-function mutation in one allele and a hotspot mutation in the other allele, although homozygous hotspot mutations were observed in two sporadic PPB cases [28] and complete loss of functional DICER1 was reported in several pineoblastomas [10, 35]. Mosaic germline DICER1 hotspot mutations were associated with a novel human overgrowth syndrome [36], suggesting that a DICER1 hotspot mutation may be oncogenic regardless of the presence of WT DICER1. Transgenic animal models are needed to delineate the oncogenic effects of DICER1 hotspot mutations in future studies.  The recurrent DICER1 hotspot mutations act more like oncogenic gain-of-function than loss-of-function mutations. Using a Dicer1/Pten double-knockout murine fallopian tumour cell model, we demonstrated that DICER1 hotspot mutations provide a growth advantage over Dicer1-null cells, in stark contrast to the repressive effects of WT DICER1. Genes de-repressed in DICER1 mutants, of which many are targets of the let-7 family miRNAs, were enriched in pathways associated with cell cycle progression (Table 2). Our data agree with previous speculation that loss of let-7, a well-known tumour suppressor, might have a profound impact on downstream pathways [4, 12].  More than 50000 women were diagnosed with endometrial cancer in the USA in 2014 (http://www.cancer.gov/). Although DICER1 hotspot mutations are found in ∼ 2% of endometrial cancers, they could affect a substantial number of patients (∼1000 cases/year). Our finding suggests that defects in the miRNA processing pathway may provide a new prospect for drug development and cancer therapy beyond rare paediatric cancers. Sex cord–stromal tumors and germ-cell tumors account for less than 10% of ovarian cancers.1 Unlike epithelial ovarian cancers, both sex cord–stromal tumors and germ-cell tumors can also occur in the testicle; testicular germ-cell tumors are the most common cancer in boys and men of European descent between the ages of 15 and 34 years.2,3 Other than a pathognomonic somatic mutation in FOXL2 in adult granulosa-cell tumors,4-6 little is known about the pathogenesis of ovarian sex cord–stromal tumors and germ-cell tumors. Recently, germline mutations in the microRNA processing gene DICER1 have been reported in probands with pleuropulmonary blastoma or the related familial tumor dysplasia syndrome, known as pleuropulmonary blastoma–family tumor and dysplasia syndrome (Online Mendelian Inheritance in Man [OMIM] number, 601200), which includes cystic nephroma, ovarian sex cord–stromal tumor (especially Sertoli–Leydig cell tumor), and multinodular goiter.7 MicroRNAs (miRNAs) are a functional class of noncoding RNA molecules that regulate translation and degradation of messenger RNA.8,9 MiRNA transcripts are processed from hairpin pre-miRNA precursors into short miRNA:miRNA* duplexes consisting of the miRNA targeting strand and the imperfectly complementary miRNA* strand (star strand, or inert carrier strand) by Dicer, an endoribonuclease with two RNase III–like domains. The RNase IIIb domain cuts the miRNA strand, whereas the RNase IIIa domain cleaves the miRNA* strand.10,11 The resultant RNA duplex is loaded into the RNA-induced silencing complex (RISC) containing an Argonaute protein. The miRNA* strand is then removed, leaving the miRNA strand, which targets messenger RNAs for degradation or interacts with the translation initiation complex to inhibit and destabilize translation of the targeted messenger RNAs.12,13 The expression levels of Dicer have global effects on the biogenesis of miRNA, and reduced expression correlates with a poor outcome in many cancers.7,14-18 In mouse models of cancer, the loss of a single Dicer1 allele (haploinsufficiency) reduced the time to tumor onset19 or survival time,20 as compared with control animals. Experimental data support the hypothesis that the pathogenicity of aberrations in Dicer function is dependent on the cellular context and that the activation or inhibition of pathways for tissue-specific development and differentiation are at least partially controlled by specific miRNAs or miRNA families.7,8,21 In mouse models with urogenital-specific knockout of Dicer1, there is evidence of apoptosis of germ cells and Sertoli cells22-24 but no induction of a tumorigenic phenotype. Outside the urogenital tract, impaired differentiation occurs with Dicer loss25-27 but requires a further challenge for oncogenesis.19,20 In this study, we report recurrent somatic missense mutations of DICER1 that implicate a defect in miRNA processing, not as a permissive event in tumor onset (as might be expected for loss of function in a tumor suppressor) but rather as an oncogenic event in the specific context of nonepithelial ovarian tumors. METHODS Experimental Data The methodologic details of all experiments that we conducted are provided in the Supplementary Appendix, available with the full text of this article at NEJM.org. Tumor Samples The 14 discovery samples included juvenile-type granulosa-cell tumors, primitive germ-cell tumors of the yolk-sac type, and Sertoli–Leydig cell tumors from the Children's Oncology Group at Nationwide Children's Hospital in Columbus, Ohio, and the Ovarian Cancer Research Program (OvCaRe) tissue bank in Vancouver, British Columbia, Canada. Both resources provide access to samples through specific application to studies approved by institutional review boards, including next-generation sequencing studies. We used 411 additional tumor samples for validation. Transcriptome, Exome, and Sanger Sequencing We carried out transcriptome sequencing and analysis as described previously.4,28,29 We performed exome capture of genomic coding regions through solution hybrid selection.30 Raw data were deposited at the European Genome–Phenome Archive (accession number, EGAS00001000135). We performed Sanger sequencing in the region of DICER1 encoding the RNase IIIb domain in DNA from samples in both the discovery and validation series and in germline DNA, when available. DICER1 Enzyme Assay We incubated wild-type DICER1 protein, mutants, and controls with 5′-32phosphate (32P)–labeled RNA oligonucleotide substrates and analyzed the cleavage products by means of gel electrophoresis and phosphorimaging. RESULTS Somatic Mutations in DICER1 Using a combination of whole-transcriptome and whole-exome sequencing, we looked for genomic aberrations in two Sertoli–Leydig cell tumors, four juvenile granulosa-cell tumors, and eight primitive germ-cell tumors of the yolk-sac type (Tables 1 and 2 in the Supplementary Appendix). We discovered four nonsynonymous missense mutations in DICER1 affecting the D1709 residue, one nonsense truncating mutation, and one insertion mutation that was predicted to result in a frameshift and truncation (Figure 1FIGURE 1 Mutations in DICER1.). All mutations were validated by means of Sanger sequencing. The D1709 residue is within a cluster of acidic residues that are responsible for metal binding10,11 in the catalytic center of the RNase IIIb domain (Fig. 1 in the Supplementary Appendix). Despite adequate sequence coverage, we observed no mutations that were predicted to affect the RNase IIIa domain (Fig. 2 in the Supplementary Appendix). Sequence-read data (Table 1 in the Supplementary Appendix) and Sanger traces from mutant DICER1 samples confirmed that all mutations were heterozygous. Copy-number analysis confirmed that only a single sample, PGCTYS-05 with wild-type DICER1, showed heterozygous loss around DICER1 (Fig. 3 in the Supplementary Appendix). DICER1 Mutations in the Validation Set The recurrent D1709 mutations in our discovery set seemed likely to have an effect on cleavage of miRNAs and metal (Mg2+/Mn2+) binding and are thus potentially pathogenic.10,11 We performed Sanger sequencing encompassing the regions encoding all four metal-binding residues within the RNase IIIb domain in a validation set that included 101 additional nonepithelial ovarian tumors (Table 1TABLE 1 Prevalence of Somatic Hot-Spot Mutations Affecting the RNase IIIb Domain of DICER1 in Nonepithelial Ovarian Tumors., and Tables 3, 4, and 5 in the Supplementary Appendix). We also screened 39 testicular tumors to determine whether they harbored the same mutations. Among the nonepithelial ovarian tumor types that were screened, mutations affecting any one metal-binding residue were present, with the highest prevalence in Sertoli–Leydig cell tumors: 26 of 43 (60%) of these tumors harbored a mutation. We designated these mutations affecting RNase IIIb metal-binding residues as hot spots. Apart from ovarian sex cord–stromal tumors and germ-cell tumors, 1 of 14 single, developmentally related, nonseminomatous testicular germ-cell tumors harbored a D1709N mutation (Table 3 in the Supplementary Appendix). To determine whether these mutations occur in other cancers with a so-called primitive phenotype, we analyzed 5 embryonal rhabdomyosarcomas; 2 of these tumors had hot-spot mutations, 1 of which occurred in a patient with a previously described germline mutation.31 We found hot-spot mutations in 33 of 159 tumors in the discovery and validation sets combined, including ovarian, testicular, and embryonal rhabdomyosarcoma tumor types. Among these mutations, the most prevalent were the 17 mutations affecting D1709. We confirmed that mutations were somatic in 16 cases for which we had corresponding germline DNA. We attempted to assess all hot-spot regions by performing Sanger sequencing of two independent amplicons spanning the metal-binding residues (for details, see the Methods section in the Supplementary Appendix). However, because of poor preservation of DNA in formalin-fixed, paraffin-embedded material, in some instances we could sequence only a single amplicon (4 mutation-positive samples and 18 mutation-negative samples) (Table 3 in the Supplementary Appendix). In cases in which both amplicon regions were successfully sequenced, mutations affecting any one RNase IIIb metal-binding residue were mutually exclusive with mutations affecting any other IIIb metal-binding residue (P<0.01 by Fisher's exact tests) (Fig. 4 in the Supplementary Appendix). We observed no mutations affecting the metal-binding regions of the RNase IIIa domain, despite targeted sequencing of 4 samples that were positive for an RNase IIIb mutation and 9 samples that were mutation-negative. Immunohistochemical analysis did not provide a proxy for hot-spot mutation status, since all tumors were positive for DICER1 expression (Fig. 5 in the Supplementary Appendix). The DICER1 hot-spot mutations are not present in the 1000 Genomes Project data or the public-data repository of the Cancer Genome Atlas consortium. No recurrent DICER1 mutations have been reported in the mutation database of the Catalogue of Somatic Mutations in Cancer (COSMIC), in which 4 of 938 cancers have somatic mutations but none in the RNase IIIb hot spots or RNase IIIa equivalents.32 Moreover, these mutations were not observed in any of the more than 1000 cancer-sequencing libraries that we studied or in 15 adult granulosa-cell tumors with FOXL2 mutations from our validation cohort. On the basis of reported DICER1 dysfunction in epithelial ovarian cancers,14 we further analyzed a series of 266 epithelial ovarian and endometrial cancers and observed a somatic hot-spot mutation (c.5676G→A, encoding an E1813G substitution) in a single ovarian carcinosarcoma (Table 4 in the Supplementary Appendix). Compound DICER1 Mutations Across our discovery and validation series, seven samples had a hot-spot mutation and an additional germline or somatic DICER1 mutation (Table 2TABLE 2 Patients with a Hot-Spot Mutation and an Additional Germline or Somatic DICER1 Mutation., and Table 3 in the Supplementary Appendix). To show that the specific hot-spot DICER1 somatic mutation was a separate hit on a different allele from the germline mutation, we cloned and sequenced complementary DNA (cDNA) from sample SLCT-19, which had a germline DICER1 deletion (5018-5021delTCAA). The primers for the reverse-transcriptase–polymerase-chain reaction (RT-PCR) assay were chosen to allow capture of both the germline and somatic changes in a single amplicon, should the two changes exist in cis with one another (Figure 2FIGURE 2 Germline Mutations of DICER1 in Trans with Somatic Hot-Spot Mutations.). Only the allele with the hot-spot mutation was observed, a finding consistent with a lymphoblastoid cell line derived from the same person in which transcripts containing the germline variant have been shown to undergo nonsense-mediated decay of messenger RNA.33 Using this RT-PCR strategy, we further showed that hot-spot mutations in three other samples were in trans with a separate DICER1 mutation, and expression of the allele with the hot-spot mutation was predominant (Table 2, and Fig. 6 in the Supplementary Appendix). Although it is possible that mutation of the non–hot-spot allele was present in other tumors from this study, because of the generally poor quality of DNA from formalin-fixed, paraffin-embedded samples, screening of the entire DICER1 sequence was not performed. Functional Characterization of DICER1 Mutants We carried out RNA cleavage assays to compare the activity of DICER1 proteins containing the hot-spot D1709N, D1709E, and E1705K substitutions with that of wild-type and control samples, including a D1709A mutant with greatly reduced RNase IIIb activity that shows total loss of DICER1 function only if a substitution in the RNase IIIa domain is also present10,11 (Figure 3FIGURE 3 Functional Characterization of DICER1 Mutations In Vitro.). Wild-type DICER1 (including ectopic, endogenous, and commercially obtained forms) generated products of 18 and 22 nucleotides (Figure 3B, and Fig. 7 in the Supplementary Appendix). These products correspond to those of enzymatic cleavage that produce a mature miRNA duplex (Figure 3A), whereas the D1709A enzyme generated only the 18-nucleotide product, as expected.10,11 This finding indicates that the 22-nucleotide band is derived from activity at the RNase IIIb site, whereas the 18-nucleotide band is related to activity at the RNase IIIa site. The hot-spot mutant DICER1 proteins yielded markedly low levels of the 22-nucleotide product and substantial levels of the 18-nucleotide product, suggesting that, like the D1709A mutation, the D1709N, D1709E, and E1705K substitutions result in reduced RNase IIIb activity but retention of RNase IIIa activity. Preliminary analysis of miRNA expression in four Sertoli–Leydig cell tumors with DICER1 hot-spot mutations had similar overall mature or processed miRNA expression levels as a Sertoli–Leydig cell tumor with no mutation in DICER1 (Fig. 8 in the Supplementary Appendix). Three of these Sertoli–Leydig cell tumors had second-allele non–hot-spot mutations, two of which were known to be truncating (Table 2). Thus, miRNA processing in the latter two Sertoli–Leydig cell tumors probably occurs through the hot-spot mutant DICER1, in which case tumors with mutant and nonmutant versions of DICER1 have similar levels of miRNA-processing activity. DISCUSSION The hot-spot DICER1 mutations that we report here are predominantly restricted to a specific subset of nonepithelial ovarian tumors and are highly prevalent (60%) in Sertoli–Leydig cell tumors. We also observed hot-spot mutations in a single high-grade ovarian carcinosarcoma, one testicular germ-cell tumor, and two embryonal rhabdomyosarcomas. The presence of these recurrent mutations in tumors that have an embryonal or primitive appearance suggests that their pathogenicity may be restricted to specific cell types or developmental settings. Generally, though, somatic mutations in DICER1 in patients with cancer are rare. A survey of cancer cell lines showed that 4 of 781 had truncating changes in DICER1.34 It is not clear whether these mutations were germline or somatic, but all occurred in microsatellite-unstable lines, suggesting limited relevance to carcinogenesis. Somatic mutations in DICER1 (albeit not the hot-spot mutations described here) have been reported in a single patient with prostate cancer35 and in two patients with epithelial ovarian cancers.14 Germline truncating DICER1 mutations have been reported in children with pleuropulmonary blastoma or a family history of the disease and related disorders, including families with Sertoli–Leydig cell tumors7 or ovarian sex cord–stromal tumors with or without a history of pleuropulmonary blastoma.36 Among 823 patients with a variety of primitive tumors, there was a low prevalence of germline DICER1 mutations that are likely to cause loss of function.34 None of these germline mutations involved the hot-spot locations that we report here or appeared to cluster within narrow regions. Retention of the wild-type DICER1 allele has been reported in the context of these germline loss-of-function mutations in all cases in which loss-of-heterozygosity studies were performed.7 It is possible that undetected hot-spot mutations were present on the ostensibly wild-type allele in these tumors. The recurrent and focal nature of the mutations and their restriction to nonepithelial ovarian tumors suggest a common oncogenic mechanism associated with a specific altered DICER1 function that is selected during tumor development in these cell types. This hypothesis is supported by several other observations. First, the in vitro biochemical data support impaired RNase IIIb activity and retention of RNase IIIa activity in tumors with hot-spot mutations. The fact that the RNase IIIa domain appears to be unaffected by mutations in the tumors that have been screened so far suggests that biologic activity in tumors with hot-spot mutations could be associated with the retained activity of the RNase IIIa site or the RISC loading function of DICER1. Second, DICER1 protein with hot-spot mutations appears to be capable of miRNA processing. We observed a second mutation in seven tumors with hot-spot mutations, and in six of these samples, the non–hot-spot mutated allele was predicted to be null. Thus, any DICER1-associated miRNA biogenesis in these six tumors was probably due to the activity of DICER1 with a hot-spot mutation. Although a global reduction in miRNA expression is one potential effect of the hot-spot mutation, our analysis suggests that a total loss of miRNA biogenesis is highly unlikely. Finally, DICER1 expression in tumors with hot-spot mutations argues against a role for DICER1 as a classic two-hit tumor suppressor. The localized and focal pattern of mutation is typical of dominantly acting oncogenes, like KRAS and BRAF. The absence of loss of heterozygosity that is seen in association with germline DICER1 mutations provides further evidence against a role for DICER1 as either a haploinsufficient or a two-hit recessive tumor suppressor. Our findings suggest that DICER1 alleles with hot-spot mutations produce viable protein. The hot-spot mutations appear to represent a new class in which the loss of the wild-type allele may occur in tandem with the retention of a hot-spot mutant DICER1 protein. We hypothesize that an oncogenic miRNA profile is derived from altered DICER1 activity in the RNase IIIb domain. Furthermore, such activity may result in a positive bias toward the processing or selection of the RNase IIIa–processed strand of the miRNA duplex (the miRNA* strand), which is consistent with the finding that the ability of DICER1 to load miRNA into RISC does not depend on the integrity of its RNase III domains.37 Recent studies suggest that miRNA* species may be important in gene regulation rather than simply being an inert strand.38,39 This hypothesis is consistent with the known importance of specific miRNAs in cell differentiation and cell-fate determination, although the mechanisms by which RNase IIIb hot-spot mutations affect miRNA processing or RISC loading in the context of cancer development warrants further investigation. We conclude that in a range of nonepithelial ovarian cancers, and potentially other cancers, aberrant miRNA processing resulting from DICER1 hot-spot mutations is a key oncogenic event.   : 3\n",
      "The receptor protein tyrosine phosphatase T (PTPRT/PTPρ) is the most frequently mutated tyrosine phosphatase in human cancer. PTPρ mediates homophilic cell-cell aggregation. In its extracellular region, PTPρ has cell adhesion molecule-like motifs, including a MAM domain, an immunoglobulin domain and four fibronectin type III (FNIII) repeats. Tumor-derived mutations have been identified in all of these extracellular domains. Previously, we determined that tumor-derived mutations in the MAM and immunoglobulin domains of PTPρ reduce homophilic cell-cell aggregation. In this manuscript, we describe experiments in which we evaluated the contribution of the FNIII repeats to PTPρ-mediated cell-cell adhesion. Our results demonstrate that deletion of the FNIII repeats of PTPρ result in defective cell-cell aggregation. Furthermore, all of the tumor-derived mutations in the FNIII repeats of PTPρ also disrupt cell-cell aggregation. These results further support the hypothesis that mutational inactivation of PTPρ may lead to cancer progression by disrupting cell-cell adhesion.  Keywords: Receptor protein tyrosine phosphatase, PTPRT, PTPρ, fibronectin type III repeat, cell-cell aggregation, cell-cell adhesion Go to: 1. Introduction Cancer progression and metastasis is in part influenced by changes in cell adhesion and tyrosine phosphorylation mediated by cell adhesion molecules (CAMs) (Paschos et al. 2009; Wheelock et al. 2008), as well as tyrosine kinases and tyrosine phosphatases, respectively (Freiss and Vignon 2004; Ostman et al. 2006). Receptor protein tyrosine phosphatases (RPTPs) comprise a family of proteins that have extracellular CAM domains coupled to cytoplasmic tyrosine phosphatase domains, and thus can impact cancer progression by regulating both cell adhesion and tyrosine phosphorylation (Brady-Kalnay 2001; Ostman et al. 2006).  The type IIB PTPμ-like subfamily of RPTPs includes PTPμ, PTPρ, PTPκ, and PCP-2 (PTPλ) (Aricescu et al. 2008; Brady-Kalnay 2001). The extracellular segment of the PTPμ-like RPTPs contain motifs found in CAMs, including a MAM domain, an immunoglobulin (Ig) domain and four fibronectin type III (FNIII) repeats (Aricescu et al. 2008; Brady-Kalnay 2001; Ostman et al. 2006). PTPμ subfamily members bind homophilically (i.e. the “ligand” for PTPμ is an identical PTPμ molecule on an adjacent cell) (Brady-Kalnay et al. 1993; Cheng et al. 1997; Gebbink et al. 1993; Sap et al. 1994; Yu et al. 2008; Zondag et al. 1995). The Ig domain of PTPμ is responsible for promoting homophilic interactions (Brady-Kalnay and Tonks 1994), and proper cell surface localization (Del Vecchio and Tonks 2005). However, both the MAM and Ig domains as well as the first two FNIII repeats are required for efficient cell-cell adhesion of PTPμ (Aricescu et al. 2006; Aricescu et al. 2007; Aricescu et al. 2008; Brady-Kalnay and Tonks 1994; Cismasiu et al. 2004; Zondag et al. 1995). PTPρ and PTPκ also mediate homophilic cell-cell adhesion (Sap et al. 1994; Yu et al. 2008). However, PCP2 has not yet been demonstrated to mediate cell-cell adhesion or aggregation although in vitro binding studies indicate it can bind homophilically (Cheng et al. 1997).  All type IIB RPTPs are potential tumor suppressor genes (Burgoyne et al. 2009a; Flavell et al. 2008; McArdle et al. 2001; Yan et al. 2006). Notably, PTPρ is the most frequently mutated phosphatase gene in human cancers including colon, lung, skin and stomach cancers (Forbes et al. 2008; Wang et al. 2004). Defective adhesion mediated by PTPρ may be at least one mechanism of cancer progression. In fact, cancer derived mutations in the MAM and Ig domains of PTPρ are defective in mediating cell-cell aggregation (Yu et al. 2008).  In addition to the tumor-derived mutations in the MAM and Ig domains of PTPρ previously described (Wang et al. 2004; Yu et al. 2008), the majority of extracellular domain mutations are located in the FNIII repeats of PTPρ (Wang et al. 2004). Given the importance of FNIII repeats in PTPμ-mediated adhesion and aggregation (Aricescu et al. 2007), we tested the contribution of the FNIII repeats of PTPρ to PTPρ-mediated adhesion. We determined that deletion constructs where each of the extracellular domains of PTPρ is removed, including the FNIII repeats, are defective in mediating cell-cell aggregation of Sf9 cells. Furthermore, when we engineered the cancer-derived mutations in the FNIII repeats of these PTPρ mutant proteins, they also result in defective cell-cell aggregation. These results demonstrate the importance of the FNIII repeats in contributing to proper cell-cell adhesion and suggest that adhesion mediated by PTPρ is likely important for the tumor suppressor activity of PTPρ.  Go to: 2. Materials and methods 2.1 Baculovirus generation  Baculoviruses were generated by transfecting cells with the BaculoGold™ Linearized Baculovirus DNA (BD Biosciences, San Diego, CA) according to the manufacturer’s protocol. The baculovirus for wild-type PTPρ was previously described (Yu et al. 2008).  2.2 Sf9 cell culture  Insect Sf9 cells from (BD Biosciences) were maintained at 27°C in Graces Insect Medium (BD Biosciences). The medium was supplemented with 10% FBS from (Hyclone, Logan, UT) and Gentamicin (MP Biomedicals, Solon, OH). Viruses were added to Sf9 cell medium to induce the expression of proteins of interest. Cells were harvested 30 hours post-infection for further analysis.  2.3 Generation of PTPRT deletion constructs  The PTPρ-ΔMAM, PTPρ-ΔIg, PTPρ-ΔFNIII-1, PTPρ-ΔFNIII-2, PTPρ-ΔFNIII-3 and PTPρ-ΔFNIII-4 constructs were generated by fusion PCR using the full-length PTPRT pVL1393-V5-His vector as the template. The primers 5’-GAAGATCTATGGCGAGCCTCGCCGCG-3’ and 5’-CCGGAATTCGGATCCTGGGGCGCTCTGAGCCCGGG-3’ were used to generate the PTPρ-ΔMAM construct; the primers 5’-GCGTGGGAGGCTCTCTGCATGGATGAGCAAGGAC-3’ and 5’-CAATGGGTCTTGCTCATCCATGCAGAGAGCCTCCCACGCCCATTG-3’ were used to generate the PTPρ-ΔIg construct; the primers 5’-CTGCACACTTGGTCCTGGTTTTCACGATCAGCTCCGCGTA-3’ and 5’-TACGCGGAGCGTCGTGAAAACCAGGACCAAGTGTGCAG-3’ were used to generate the PTPρ-ΔFNIII-1 construct; the primers 5’-CCGCCAGGGCCTCCCCTCACCGAGGAAGACGTTCCAGGAG-3’ and 5’-CTCCTGGAACGTCTTCCTCGGTGAGGGGAGGCCCTGGCGG-3’ were used to generate the PTPρ-ΔFNIII-2 construct; the primers 5’-GAGGAGCTGGTGGTGCAGACTAAAATTTCAGCTCCATCCA-3’ and 5’-TGGATGGAGCTGAAATTTTAGTCTGCACCACCAGCTCCTC-3’ were used to generate the PTPρ-ΔFNIII-3 construct and the 5’-GTCACCACTCGGATTGCCACCGGTGCCTCCACCCAGAATT-3’ and 5’-AATTCTGGGTGGAGGCACCGGTGGCAATCCGAGTGGTGAC-3’ were used to generate the PTPρ-ΔFNIII-4 construct.  2.4 Immunoblotting  Sf9 cells were lysed in a standard lysis buffer (20 mM Tris·HCl, pH 7.5, 1% Triton X-100, 150 mM sodium chloride, 2 mM Benzamidine, 2.5μg/ml Aprotinin, 2.5μg/ml Leupeptin, and 0.5μg/ml Pepstatin). Samples were solubilized in 2X SDS loading buffer and separated by SDS-PAGE. Samples were transferred to a nitrocellulose membrane (Whatman, Piscataway, NJ) and immunoblotted with SK18 (Brady-Kalnay et al. 1993), which cross-reacts with PTPρ.  2.5 Cell surface expression of PTPρ  Cell surface expression of PTPρ protein in baculovirus infected Sf9 cells was determined using a modified biotin-avidin reaction with the Cell Surface Protein Isolation Kit (catalog# 89991, Pierce, Rockford, IL), as previously described (Yu et al. 2008). Briefly, Sf9 cell media was removed and the flasks were incubated with rocking for 30 minutes at 4°C in 5ml of 0.25mg/ml Sulfo-NHS-SS-Biotin in 1X PBS. The cells were mechanically removed from the flask, centrifuged, washed, and lysed according to the manufacturer’s protocols. To isolate cell surface expressed PTPρ the cell lysate was added to a NeutrAvidin Gel column and incubated for 60 min at room temperature (RT) with rocking prior to washing and eluting proteins with 200μl of sample buffer containing a final concentration of 55mM DTT for 60min at RT. The flow through was run on 6% SDS-PAGE gels and transferred to nitrocellulose. Total cell surface protein levels were normalized by immunoblotting cell lysates with an anti-actin antibody (JLA20). The normalized cell surface protein was run on 6% SDS-PAGE gels, transferred to nitrocellulose and immunoblotted with antibodies to PTPμ that also recognized PTPρ (SK18) followed by secondary antibody conjugated to HRP. The HRP signal was detected using a Fluor-S MAX MultiImager (Bio-Rad Laboratories Inc., Hercules, CA).  2.6 Aggregation Assays  Sf9 cells were harvested by trituration. The cell suspensions were added to glass scintillation vials and incubated at 27°C at 90 rpm in a gyratory shaker for 30 minutes. For a given condition, the cell suspension was transferred to a 100mm petri dish either prior to (0 time point) or 30 minutes after aggregation. Images were captured at 10X magnification with a Nikon TE-200 inverted fluorescent microscope (Tokyo, Japan). To quantify cell aggregates, pictures of four randomly selected fields were taken per dish. The area of each object (single cells and aggregates) was determined by the Metamorph software (Molecular Devices) using an auto-threshold setting for light objects and appropriate size filters that allowed the counting of cells and aggregates but not debris. The four replicates were combined to yield average readings per condition. The percent aggregation was then calculated as the average aggregate area at 30 minutes minus the average aggregate area at zero minutes divided by the average aggregate area at 30 minutes [(N30-N0)/N30]. A minimum of three independent experiments was performed per condition. Statistical significance was determined in Microsoft Excel using Student’s T-test. Error bars indicate standard error.  2.7 Site-directed mutagenesis  The I395V, Y412F, R453C, Q479E, S492F, N510K, T605M, V648G, A707T, and L708P tumor-derived mutations (Wang et al. 2004) were generated by fusion PCR using the full-length PTPRT pVL1393-V5-His vector as the template. The primers 5’-TGGCCCACAGAACGTGGAAGTCGTAGACATCAGAGCC-3’ and 5’-CTTCCACGTTCTGTGGGCCA-3’ were used as the mutagenic primers for the I395V mutant; the primers 5’-CAGTGGGAGCCCTTCGGCTTCGCGGTGACCCGCTGCCAT-3’ and 5’-AAGCCGAAGGGCTCCCACTG-3’ were used as the mutagenic primers for the Y412F mutant; the primers 5’-CTACACCCTGCGAGGCCTGTGCCCCTTCATGACCATCCG-3’ and 5’-ACAGGCCTCGCAGGGTGTAG-3’ were used as the mutagenic primers for the R453C mutant; the primers 5’-GAGCGAGGAGCTGGTGGTGGAGACTGAGGAAGACGTTCC-3’ and 5’-CCACCACCAGCTCCTCGCTC-3’ were used as the mutagenic primers for the Q479E mutant; the primers 5’-TGTTCCTCTAGAATTCATCCAAGGGGGGCCCT TT-3’ and 5’-ATGAATTCTAGAGGAACA-3’ were used as the mutagenic primers for the S492F mutant; the primers 5’-TCCAGTGGAAACCTCCCAAGGAGACCAATGGGGTCATCA-3’ and 5’-CTTGGGAGGTTTCCACTGGA-3’ were used as the mutagenic primers for the N510K mutant; the primers 5’-CCATTGAATGAGACAGACATGACC ATCACAGTGATGCTG3-’ and 5’-ATGTCTGTCTCATTCAATGG-3’ were used as the mutagenic primers for the T605M mutant; the primers 5’-ATTATTGAGTGCTTTTC GGGGCCCGTGAGCTATCGGAAT-3’ and 5’-CCCGAAAAGCACTCAATAAT-3’ were used as the mutagenic primers for the V648G mutant; the primers 5’-CTACAGCATCTACTTCCAGACACTCAGCAAAGCCAATGG-3’ and 5’-TCTGGAAGTAGATGCTGTAG-3’ were used as the mutagenic primers for the A707T mutant; the primers 5’-AGCATCTACTTCCAGGCACCCAGCAAAGCCAATGGAGAG-3’ and 5’-GGTGCCTGGAAGTAGATGCT-3’ were used as the mutagenic primers for the L708P mutant.  2.8 Mapping of PTPρ mutations onto the PTPμ crystal structure  The sequence of PTPμ and PTPρ were aligned using CLUSTALW. Mutations probed in this study were mapped using this sequence alignment onto the PTPμ structure (PDB ID: 2V5Y). The figure was prepared utilizing PyMOL 1.2r1 (http://pymol.org).  Go to: 3. Results 3.1 Deletions in the MAM, Ig, and FNIII repeat domains of PTPρ result in defective cell-cell aggregation  To test whether a protein mediates cell-cell aggregation or adhesion, one has to express the protein of interest in the non-adhesive cells. Otherwise, one cannot measure the aggregation or adhesion of a single molecule in the sea of other adhesion molecules naturally expressed in adhesive cells. This is the main reason that the non-adhesive Sf9 or Drosophila S2 cells are widely used models for such studies. To our knowledge, nearly all cell lines of epithelial origin are adhesive cells and thus are not suitable for these studies. Introduction of putative adhesion molecules into non-adhesive Drosophila S2 insect cells has been used to demonstrate adhesive functions for fasciclin II, connectin, Dtrk, ARK, Neuroglian, and Capricious molecules directly (Bellosta et al. 1995; Hortsch et al. 1995; Nose et al. 1992; Pulido et al. 1992; Shinza-Kameda et al. 2006; Snow et al. 1989). In a similar approach, we and others demonstrated that the full-length form of PTPμ induced aggregation, via a homophilic binding mechanism, when expressed in nonadhesive Sf9 insect cells, which are derived from the Fall army-worm Spodoptera frugiperda (Brady-Kalnay et al. 1993; Gebbink et al. 1993; Zondag et al. 1995). We have also used this system for analyzing PTPRT/PTP rho (Yu et al., 2008). Examples of other cell adhesion molecules that have been studied in Sf9 cells include L1 (Gouveia et al. 2008), Galectin-3 (Inohara and Raz 1995) and C-CAM1 (Phan et al. 2001). Insect S2 and Sf9 cells are extremely useful systems to analyze adhesive mechanisms of cell adhesion molecules.  The MAM domain, Ig domain, and first two-FNIII repeats are essential for PTPμ cell-cell aggregation (Brady-Kalnay and Tonks 1994; Zondag et al. 1995). To test whether these domains are the minimal essential regions required for PTPρ cell-cell aggregation, baculoviral PTPρ deletion constructs of the MAM domain, Ig domain, and each of the four individual FNIII repeats were made and expressed in non-adhesive Sf9 insect cells (Figure 1). All the PTPρ deletion constructs were expressed and appropriately trafficked to the cell surface (Figure 1). To test whether these domains contribute to PTPρ-dependent adhesion, Sf9 cells were evaluated for their ability to mediate cell-cell aggregation when expressing the deletion constructs. Aggregation assays with Sf9 cells expressing the deletion constructs were conducted alongside aggregation assays using Sf9 cells expressing wild-type PTPρ, which mediates cell-cell aggregation (Figure 2) and serves as a positive control (Yu et al. 2008). Deletion of any of the extracellular domains of PTPρ resulted in defective cell-cell aggregation compared to aggregation of wild-type PTPρ expressing Sf9 cells (Figure 2). Deletion of the fourth FNIII repeat was the least severe where ~23% aggregation was observed. Deletion of the MAM domain, Ig domain and FNIII repeats 1, 2 or 3 resulted in 5–16% aggregation. When the level of aggregation was quantified, we determined that all of the deletion constructs resulted in a large and statistically significant reduction in cell-cell aggregation (Figure 2, p-value<0.005). Together with our previous study, these results demonstrate that all of the extracellular domains of PTPρ are necessary for PTPρ-mediated cell-cell aggregation.  Figure 1 Figure 1 PTPρ deletion constructs are all expressed at the cell surface. Constructs containing deletions of one extracellular domain of PTPρ each were generated and expressed in Sf9 cells. Total cellular protein and cell surface protein were isolated ... Figure 2 Figure 2 Deletion of any of the extracellular domains of PTPρ disrupts cell-cell aggregation. Sf9 cells expressing deletion constructs of the individual extracellular domains of PTPρ were allowed to aggregate for 30 minutes prior to imaging. Scale ... 3.2 Tumor-Derived PTPρ Mutations in the FNIII repeats result in defective cell-cell aggregation  We next tested the effects of human tumor-derived mutations in the FNIII repeats of PTPρ on PTPρ-mediated cell-cell aggregation. We recently demonstrated that tumor-derived mutations in the MAM and Ig domains of PTPρ are defective in cell-cell aggregation using the Sf9 aggregation assay (Yu et al. 2008), and given the above results, we hypothesized that mutations in the FNIII domains would likewise result in defective aggregation. Ten human tumor-derived mutations are located in the second, third and fourth FNIII repeats of PTPρ (Wang et al. 2004). To test whether these mutations also affect cell-cell aggregation, we engineered the ten human mutations by site-directed mutagenesis using the full-length PTPρ baculoviral construct as a template (see Figure 3). The ten mutations are expressed in Sf9 cells at the expected molecular weight and at similar levels to full-length human PTPρ (Figure 3). All of these mutants are expressed at the cell surface (Figure 3). When these mutant PTPρ proteins were expressed in Sf9 cells and tested in aggregation assays, we observed that all of the mutant PTPρ proteins resulted in fewer and smaller aggregates than wild type PTPρ (Figure 4). When the level of aggregation was quantified, we determined that all of the FNIII mutations significantly reduced cell-cell aggregation (16–43%) compared to cells expressing wild type PTPρ (68% aggregation, p-value<0.008, Figure 4).  Figure 3 Figure 3 PTPρ proteins containing tumor-derived mutations in their FNIII domains are expressed at the cell surface. Ten mutations in the FNIII domains of PTPρ were generated by site-directed mutagenesis and expressed in Sf9 cells. Total cellular ... Figure 4 Figure 4 Tumor derived mutations in FNIII domains of PTPρ are defective in cell-cell aggregation. Sf9 cells expressing the mutated PTPρ proteins were allowed to aggregate for 30 minutes prior to imaging. Scale bar equals 100μm. The percentage ... Go to: 4. Discussion PTPRT/PTPρ is the most highly mutated tyrosine phosphatase in human colon carcinomas. We and others found that PTPRT/ PTPρ is also mutated in lung, gastric and skin (melanoma) cancers (Forbes et al. 2008; Wang et al. 2004). The spectrum of mutations, which included nonsense mutations and frameshifts, suggested that these mutations were inactivating (Wang et al. 2004). Biochemical analyses demonstrated that missense mutations in the catalytic domains of PTPRT diminished its phosphatase activity and overexpression of PTPRT inhibited colon cancer cell growth. Moreover, we showed previously that the tumor-derived mutations in the MAM and Ig domains of PTPρ are defective in cell-cell adhesion (Yu et al. 2008). All these data suggest that PTPRT/PTPρ normally functions as a tumor suppressor gene. The experiments presented in this manuscript evaluate the contribution of the individual extracellular domains of PTPρ to PTPρ-mediated adhesion. We demonstrate that in addition to the MAM and Ig domains, the FNIII repeats are required for PTPρ-mediated adhesion. Importantly, we also show that PTPρ-mediated adhesion is impaired by human tumor-derived mutations in those FNIII repeats. These studies indicate that all the tumor-derived mutations located in the extracellular domain of PTPRT are loss-of-function mutations, thereby providing further evidence that PTPRT is a tumor suppressor. This notion is further supported by our recent study showing that PTPRT knockout mice are highly susceptible to colon specific carcinogen azoxymethane (AOM)-induced colon cancer (Zhao et al.).  PTPμ-mediated cell-cell aggregation requires the MAM domain, Ig domain, and the first two FNIII repeats (Aricescu et al. 2006; Aricescu et al. 2007; Brady-Kalnay and Tonks 1994; Cismasiu et al. 2004; Zondag et al. 1995). The entire extracellular domain of PTPμ is predicted to make a rigid conformation with extensive interfaces between each of the domains (Aricescu et al. 2006). The minimal adhesive unit for functional cell-cell aggregation is the MAM domain, Ig domain and the first two FNIII repeats (Aricescu et al. 2006; Brady-Kalnay and Tonks 1994; Zondag et al. 1995). However, the FNIII repeats appear to play a role in spacing the key homophilic interaction domain(s) a certain distance from the plasma membrane. Deletion of the FNIII repeats changed the intercellular spacing of two apposing membranes (Aricescu et al. 2007). Cell-cell aggregation assays of the FNIII deletion constructs were not performed in that study (Aricescu et al. 2007). The amino acid residues of the interfaces between the extracellular domains are highly conserved between members of the type IIB RPTP subfamily (Aricescu et al. 2006; Aricescu et al. 2007) therefore it is likely that the PTPρ adhesive interface resembles that of PTPμ. Together our data support the hypothesis that the essential regions for PTPρ-mediated adhesion resemble those of PTPμ.  To gain insight into how the point mutations may alter the adhesive function of PTPρ, we modeled the human tumor derived PTPρ mutations onto the crystal structure of PTPμ (Figure 5). The crystal structure of PTPμ includes only the first three FNIII repeats (Aricescu et al. 2007). Four of these mutations fall within the second FNIII (FNIII-2) repeat (I395V, Y412F, R453C, Q479E). It is important to note that FNIII-2 is required for efficient cell-cell adhesion (Aricescu et al. 2006). Two of the mutations fall in the third FNIII (FNIII-3) repeat (S492F, N510K). The mutation within the linker region between FNIII-1 and FNII-2 (Y412F) could alter the flexibility or positioning between the two domains. The other mutations (I395V R453C, Q479E, S492F, N510K) appear to be clustered in surface exposed regions and/or near the linker regions between FNIII-2 and FNIII-3 that could alter flexibility, position of individual FNIII repeats and/or protein-protein interactions. While these mutations do not correspond to the adhesive interface hypothesized for PTPμ based upon a single static crystal structure (Aricescu et al. 2007), some of the mutations fall within the FNIII-2 repeat which is required for adhesion, and all of these surface modifications would likely alter either the three-dimensional topology and/or cis/trans interactions of PTPρ.  Figure 5 Figure 5 Structural modeling of the point mutations of PTPρ. The point mutations (indicated in red) evaluated in this manuscript were modeled onto the equivalent sites in the crystal structure of PTPμ. The PTPρ mutations likely alter cell-cell ... Other type IIB RPTPs also function as tumor suppressor genes, by regulating adhesion and/or tyrosine phosphorylation of effector proteins. Expression of PTPμ is reduced in human glioblastomas (GBM) and cell lines (Burgoyne et al. 2009a). Reduction in full-length PTPμ expression with a concomitant increase in a proteolytically processed cytoplasmic fragment of PTPμ is observed in GBM cells and both are correlated with decreased adhesion and increased migration observed in the disease (Burgoyne et al. 2009a; Burgoyne et al. 2009b). Over-expression of the full-length form of PTPμ suppressed migration of GBM cells and reduces their survival (Burgoyne et al. 2009b). Of note, at the same time that stable cell adhesion is reduced due to the loss of full-length PTPμ expression in GBM cells, a catalytically active cytoplasmic fragment of PTPμ is released and translocates to the nucleus (Burgoyne et al. 2009b). Catalytic activity of this fragment is also necessary for cell migration and viability of GBM cells (Burgoyne et al. 2009b). Therefore, both the tyrosine phosphatase activity of the cytoplasmic fragment of PTPμ and the loss of PTPμ-mediated adhesion may contribute to the invasiveness of GBM cells.  PTPκ is also implicated in tumor progression. PTPκ expression is reduced in melanoma cell lines and human tissue biopsies (McArdle et al. 2001), and in Hodgkin’s lymphoma cells (Flavell et al. 2008). Over-expression of PTPκ in Hodgkin lymphoma cells reduces cellular proliferation and survival (Flavell et al. 2008). Like PTPμ, PTPκ is proteolytically processed (Anders et al. 2006), potentially as a result of aberrant glycosylation (Kim et al. 2006). It is not clear whether adhesion or catalytic activity of PTPκ is required for its tumor suppressor activity (Flavell et al. 2008; Kim et al. 2006). Of note, the cleaved cytoplasmic domain fragment of PTPκ is catalytically active, translocates to the nucleus and promotes β-catenin-mediated transcription (Anders et al. 2006). Signaling via the PTPκ cytoplasmic fragment might augment the loss of stable cell-cell adhesion and may promote tumor progression.  Although speculative, it is interesting to postulate that tyrosine phosphatase activity and or cleavage of PTPρ will also be an important element of its tumor suppressor activity, as has been demonstrated for PTPμ and PTPκ. Interestingly, we identified and validated STAT3 as a direct substrate of PTPRT (Zhang et al. 2007). STAT3 is latent transcription factor that translocates from the cytoplasm to the nucleus after being phosphorylated at the Y705 residue. It is possible that the cleaved intracellular fragment of PTPρ, which contains the phosphatase domains, translocates to nucleus and dephosphorylates STAT3. Besco and colleagues found that PTPRT associates with the adhesion molecule E-cadherin and its binding partner p120 catenin, which are PTPRT substrates (Besco et al. 2006). p120 also translocates to the nucleus and associates with the Kaiso transcription factor (Daniel and Reynolds 1999). Given the frequency and distribution of mutations in PTPRT/PTPρ in human colon and other cancers, a more thorough understanding of its biological function is warranted.  : 3\n",
      "Tyrosine phosphorylation, regulated by protein tyrosine phosphatases (PTPs) and kinases (PTKs), is important in signaling pathways underlying tumorigenesis. A mutational analysis of the tyrosine phosphatase gene superfamily in human cancers identified 83 somatic mutations in six PTPs (PTPRF, PTPRG, PTPRT, PTPN3, PTPN13, PTPN14), affecting 26% of colorectal cancers and a smaller fraction of lung, breast, and gastric cancers. Fifteen mutations were nonsense, frameshift, or splice-site alterations predicted to result in truncated proteins lacking phosphatase activity. Five missense mutations in the most commonly altered PTP (PTPRT) were biochemically examined and found to reduce phosphatase activity. Expression of wild-type but not a mutant PTPRT in human cancer cells inhibited cell growth. These observations suggest that the mutated tyrosine phosphatases are tumor suppressor genes, regulating cellular pathways that may be amenable to therapeutic intervention. Phosphorylation of tyrosine residues is a central feature of many cellular signaling pathways, including those affecting growth, differentiation, cell cycle regulation, apoptosis, and invasion (1, 2). This phosphorylation is coordinately controlled by protein tyrosine kinases (PTKs) and phosphatases (PTPs). Although a variety of PTK genes have been directly linked to tumorigenesis through somatic activating mutations (3–6), only a few PTP genes have been implicated in cancer (7–10). Moreover, it is not known how many or how frequently members of the PTP gene family are altered in any particular cancer type. We have systematically addressed these issues by comprehensive mutational analysis of the PTP gene superfamily in colorectal tumors.  The PTP gene superfamily is composed of three main families: (i) the classical PTPs, including the receptor PTPs (RPTPs) and the nonreceptor PTPs (NRPTPs); (ii) the dual specificity phosphatases (DSPs), which can dephosphorylate serine and threonine in addition to tyrosine residues; and (iii) the low molecular weight phosphatases (LMPs) (1). Using an approach similar to that described in a recent bioinformatic analysis of PTPs in the mouse genome (11), we employed a combination of Hidden Markov Models representing catalytic domains of members of the PTP superfamily to identify 53 classical PTPs (21 RPTPs and 32 NRPTPs), 33 DSPs, and one LMP in the human genome (12). This analysis revealed a set of genes representing all known human PTPs (13) as well as seven putative PTPs.  As an initial screen to evaluate whether these phosphatases are genetically altered in human cancer, we analyzed the coding exons of all 87 members of this gene superfamily in 18 colorectal cancers. A total of 1375 exons from all annotated RPTPs, NRPTPs, DSPs, and LMPs were extracted from genomic databases (12). These exons were amplified by polymerase chain reaction from cancer genomic DNA samples and directly sequenced with dye terminator chemistry (12). Whenever a presumptive mutation was identified, we attempted to determine whether it was somatically acquired (i.e., tumor specific) by examining the sequence of the gene in genomic DNA from normal tissue of the relevant patient.  From the 3.3 Mb of sequence information obtained, we identified six genes containing somatic mutations, including three members of the RPTP subfamily (PTPRF, PTPRG, and PTPRT) and three members of the NRPTP subfamily (PTPN3, PTPN13, and PTPN14). These six genes were then further analyzed for mutations in another 157 colorectal cancers. Through this strategy, we identified 77 mutations in the six genes, in aggregate affecting 26% of the colorectal tumors analyzed (table S1 and Fig. 1). Examination of these six genes in seven other tumor types identified PTPRT mutations in two of 11 (18%) lung cancers and two of 12 gastric cancers (17%), and PTPRF mutations in one of 11 (9%) lung cancers and one of 11 (9%) breast cancers. No mutations were identified in 12 pancreatic cancers, 12 ovarian cancers, 12 medulloblastomas, or 12 glioblastomas (table S1 and Fig. 1). In total, 83 nonsynonymous mutations were observed, all of which were somatic in the cancers that could be assessed (12).    Download high-res image  Open in new tab  Download Powerpoint Fig. 1. Distribution of mutations in PTPRT, PTPN13, PTPN14, PTPRG, PTPRF, and PTPN3. Black arrows indicate location of missense mutations, red arrows indicate location of nonsense mutations or frameshifts, and boxes represent functional domains (B41, band 41; CA, carbonic anhydrase; FN3, fibronectin type III; IG, immunoglobulin; MAM, meprin/A5/PTPμ; PDZ, postsynaptic density, discs large, zonula occludans; PTPc, catalytic phosphatase domain). Black stars indicate PTPRT mutants evaluated for phosphatase activity (Fig. 2), and the red star indicates PTPRT mutant evaluated for effects on cell proliferation (Fig. 3).  Fifteen of the 83 mutations were nonsense, frameshift, or splice-site alterations, all of which were predicted to result in aberrant or truncated proteins. In 16 tumors both alleles of the phosphatase gene appeared to be mutated, a characteristic often associated with tumor suppressor genes. The majority of tumors with PTP gene mutations also contained mutations in KRAS or BRAF, and nine tumors contained alterations in previously reported tyrosine kinase genes (table S1). Thus, the mutant phosphatases identified in this study are likely to operate through cellular pathways distinct from those associated with previously identified mutant kinases.  Analysis of mutations in tumors is complicated by the fact that mutations can arise either as functional alterations affecting key genes underlying the neoplastic process or as nonfunctional “passenger” changes. The multiple waves of clonal expansion and selection that occur throughout tumorigenesis lead to fixation of any mutation that had previously occurred in any predecessor cell, regardless of whether the mutation was actually responsible for the clonal expansion. Two independent lines of evidence suggest that the sequence alterations we observed are functional. First, the ratio of nonsynonymous to synonymous mutations provides an indication of selection, as synonymous alterations usually do not exert a growth advantage. There were no somatic synonymous mutations detected in the colorectal cancers analyzed, resulting in a ratio of nonsynonymous to synonymous mutations of 77 to 0, much higher than the expected 2:1 ratio for nonselected passenger mutations (P < 1 × 10–6). Second, the prevalence of mutations in the coding regions of the analyzed genes was ∼19 per Mb of tumor DNA, similar to the prevalence of functional somatic alterations observed in other gene families [e.g., the tyrosine kinome (6)] and significantly higher than the prevalence of nonfunctional alterations previously observed in the cancer genomes (∼1 per Mb, P < 0.01) (14). These data support the idea that these mutations were the targets of selection during tumorigenesis.  The great majority of the nonsense and frameshift mutations (Fig. 1) would result in polypeptides devoid of C-terminal phosphatase catalytic domains, thereby leading to inactivation of the phosphatase. To evaluate whether tumor-specific point mutations alter phosphatase activity, we biochemically tested mutant versions of PTPRT, the most frequently mutated PTP in the superfamily. Mutations in both intracellular PTP domains (D1 and D2) were evaluated. His-tagged versions of the catalytic region of wild-type PTPRT, two D1 mutants (Q987K and N1128I), and three D2 mutants (R1212W, R1346L, and T1368M) were produced in bacteria and analyzed for phosphatase activity by using 6,8-difluoro-4-methylumbelliferyl phosphate (DiFMUP) as a substrate (Fig. 2) (12). All D1 and D2 mutants had reduced phosphatase activity compared with the wild-type protein (Fig. 2). The kinetic parameter Kcat was reduced in both D1 mutants, while Km was increased in all three D2 mutants, suggesting that mutations in the two domains may have different effects on enzymatic activity. Although it has been thought that the D2 domain is usually catalytically inactive (1), our results are consistent with recent studies that show that the D2 domain is important for phosphatase activity in some receptor phosphatases (15).    Download high-res image  Open in new tab  Download Powerpoint Fig. 2. Evaluation of phosphatase activity of mutant PTPRT. (A) Saturation kinetics of wild-type and mutant PTPRT. His-tagged versions of PTPRT protein segments comprising the two catalytic domains containing wild-type (WT) and tumor-specific mutant sequences were expressed in bacteria and purified using nickelaffinity chromatography. Equal amounts of WT and mutant proteins were used to evaluate enzyme kinetics. The rate of hydrolysis of substrate (DiFMUP) is plotted against increasing substrate concentration. Data were fitted to the Michaelis-Menton equation, and the resulting kinetic parameters of WT and mutant proteins are indicated in (B).  These biochemical data on missense mutations, coupled with the large number of truncating mutations, suggested that PTPRT functions as a tumor suppressor gene. To determine whether PTPRT expression can inhibit tumor cell growth, we transfected wild-type PTPRT into HCT116 colorectal cancer cells (12). An identical expression vector containing an R632X mutant of PTPRT was used for comparison. Wild-type PTPRT potently inhibited cell growth in this assay, as seen by the substantial decrease in the number of neomycin-resistant colonies compared with the R632X mutant or with vector alone (Fig. 3, A and B). Similar results with wild-type and mutant PTPRT were also observed in DLD1 colorectal cancer cells.    Download high-res image  Open in new tab  Download Powerpoint Fig. 3. PTPRT overexpression suppresses growth of human cancer cells. (A) HCT116 colorectal cancer cells were transfected with wild-type (WT) PTPRT construct, truncated R632X mutant PTPRT construct, or empty pCI-Neo vector. The photographs show colonies stained with crystal violet after 14 days of geneticin selection. (B) Number of resistant colonies (mean of two 25 cm2 flasks) for WT PTPRT, mutant PTPRT, and empty vector.  The combination of these genetic, biochemical, and cellular data suggest that PTPRT and the other identified phosphatases are likely to act as tumor suppressors. This is consistent with the function of other phosphatases implicated in tumorigenesis (7, 8, 16) and with the general role of phosphatases in inhibiting various growth-promoting signaling pathways (2). The absence of biallelic mutations in a subset of the analyzed tumors suggests that some alterations may act in a dominant-negative fashion or may affect gene dosage, mechanisms that have been previously involved in inactivation of other tumor suppressor genes (17, 18).  Little is known about the functional role of the tyrosine phosphatases discussed here. PTPN13 appears to be involved in apoptosis (19) and may be partly responsible for the antitumor effects of tamoxifen (20). Overexpression of PTPN3 inhibits growth of NIH/3T3 cells, possibly through interaction with valosin-containing protein (VCP/p97) (21). PTPN14 and PTPRF are thought to play a role in cell adhesion by regulating tyrosine phosphorylation of adherens junction proteins (22, 23). Because increased phosphorylation of adherens junctions has been shown to increase cell motility and migration (22, 24), mutational inactivation of these genes may be an important step in cancer cell invasion and metastasis. PTPRG maps to chromosome 3p14.2, a region frequently lost in lung, renal, and early-stage breast tumors, and is thought to be a target of the translocation at 3p14 in familial renal cell carcinoma (25–27). However, no point mutations in PTPRG (28) or any of the other genes identified here have been previously described in any cancer. PTPRT is expressed in the developing central nervous system and in the adult cerebellum (29) and had not been thought to play a role in the growth or differentiation of other tissues. We have found that PTPRT is expressed in a variety of human tissues, including normal colon epithelium as well as cells derived from colorectal cancers (fig. S1).  Phosphatases affect signaling pathways that may be amenable to therapeutic intervention in cancer cells (2). Although reactivation of incapacitated phosphatases is likely to be pharmacologically challenging, identification of the corresponding kinases that phosphorylate substrates normally regulated by the mutant phosphatases could provide novel therapeutic targets. Like the analysis of genetic alterations in tyrosine kinases (6), the present study suggests the possibility of individualized therapy based on the mutant phosphatases present in specific tumors. Conceivably, this approach would have broad therapeutic use, because more than 50% of colorectal tumors analyzed to date have alterations in at least one member of the tyrosine phosphatome or kinome. Protein tyrosine phosphatase receptor-type T (PTPRT) is the most frequently mutated tyrosine phosphatase in human cancers. However, the cell signaling pathways regulated by PTPRT largely remain to be elucidated. Here, we show that paxillin is a direct substrate of PTPRT and that PTPRT specifically regulates paxillin phosphorylation at tyrosine residue 88 (Y88) in colorectal cancer (CRC) cells. We engineered CRC cells homozygous for a paxillin Y88F knock-in mutant and found that these cells exhibit significantly reduced cell migration and impaired anchorage-independent growth, fail to form xenograft tumors in nude mice, and have decreased phosphorylation of p130CAS, SHP2, and AKT. PTPRT knockout mice that we generated exhibit increased levels of colonic paxillin phosphorylation at residue Y88 and are highly susceptible to carcinogen azoxymethane-induced colon tumor, providing critical in vivo evidence that PTPRT normally functions as a tumor suppressor. Moreover, similarly increased paxillin pY88 is also found as a common feature of human colon cancers. These studies reveal an important signaling pathway that plays a critical role in colorectal tumorigenesis.  Keywords: colorectal cancer Reversible tyrosine phosphorylation, which is coordinately controlled by protein tyrosine kinases (PTKs) and phosphatases (PTPs), governs numerous signaling pathways that regulate cell proliferation, apoptosis, adhesion, and migration. Over the last two decades, many PTKs have been found to be mutated in a variety of different tumor types (reviewed in ref. 1). In contrast to PTKs, the role of PTPs in tumorigenesis is underexplored. To systematically evaluate possible roles of PTPs in tumorigenesis, we used a high throughput molecular and bioinformatics approach to detect genetic alterations of the tyrosine phosphatase gene family in colorectal cancers (CRCs) (2). Among the six mutated PTPs that we identified, protein tyrosine phosphatase receptor-type T (PTPRT), also known as PTPρ, was the most frequently mutated (2). In addition, we and others found that PTPRT is also mutated in lung, stomach, and skin cancers (2, 3). The spectrum of mutations, which includes nonsense mutations and frameshifts, suggested that these mutations were inactivating (2). Biochemical analyses demonstrated that missense mutations in the catalytic domains of PTPRT diminished its phosphatase activity, whereas overexpression of PTPRT inhibited CRC cell growth (2). Taken together, these studies suggest that PTPRT normally acts as a tumor suppressor gene. In light of these data, it is important to identify the functionally significant substrates of PTPRT as well as to elucidate the signal transduction pathways regulated by this phosphatase. Here, we report that paxillin, an adaptor protein involved in cell adhesion, migration, proliferation, and apoptosis (4, 5), is a direct substrate of PTPRT and that phospho-paxillin has oncogenic properties. We further demonstrate that in an in vivo model, PTPRT is both a potent tumor suppressor gene and a key regulator of colonic phospho-paxillin levels.  Given its roles in cell adhesion and migration, paxillin is thought to play an important role in tumor migration, invasion, and metastasis (6). Moreover, a recent study found that paxillin is mutated and amplified in roughly 20% of lung cancers and that mutant paxillin promotes cancer cell growth and enhances cell adhesion, suggesting that paxillin is a proto-oncogene (7). Paxillin directly interacts with a variety of cellular proteins, including FAK, Gab1, p120RasGAP, and others (4, 8 –10). Some of these interactions are tyrosine phosphorylation (pY)-dependent, wherein the pY residues recruit SH2 domain-containing proteins (4). A recent mass spectrometric analysis indicates that tyrosine phosphorylation occurs at multiple sites on paxillin including tyrosine residues 31, 40, 76, 88, 118, 182, 377, and 436 (11). Although paxillin Y31 and Y118 phosphorylation has been implicated in cell adhesion and migration (12), the physiological relevance of tyrosine phosphorylation at other residues remains to be determined.  In this study, we demonstrate that PTPRT dephosphorylates paxillin specifically at Y88 residue both in vitro and in CRC cells, and, by using paxillin Y88F mutant knock-in CRC cells, that regulation of paxillin Y88 phosphorylation plays a critical role in colorectal tumorigenesis.  Go to: RESULTS Paxillin Is a Direct Substrate of PTPRT. As reported previously, we used a proteomic approach to identify potential substrates of PTPRT (13). We generated two HEK 293T cell lines: one expressing the intracellular part containing the two phosphatase domains of PTPRT and the other expressing the extracellular portion of the protein (13). We used the proteomic approach developed by Rush et al. (14) to globally profile tyrosine phosphopeptides in the two cell lines as well as parental HEK293T cells. Phosphopeptides that are present in parental HEK 293T cells and in cells expressing the extracellular portion of PTPRT, but not in cells expressing the intracellular portion, are potential PTPRT substrates. Through this approach, we identified five candidate substrates of PTPRT, including paxillin and STAT3. We validated that STAT3 is indeed a direct substrate of PTPRT (13). Given that PTPRT plays important roles in cell proliferation and adhesion (2, 15), two processes that paxillin is also involved, we set out to determine whether paxillin was a direct substrate of PTPRT. To test this hypothesis, we employed a substrate-trapping assay modified from that described for investigation of unrelated phosphatase (PTP1B) (16, 17). Three GST fusion proteins were constructed for this purpose as reported (13). All contained both phosphatase domains of PTPRT, but two of the three (TRAP-1 and TRAP-2) contained mutations predicted to result in substrate trapping (the D1074A single mutant and the C1106S D1074A double mutant, respectively). The recombinant GST fusion proteins were expressed in Escherichia coli and equal amounts of each were attached to beads (Fig. S1A). The beads were then incubated with lysates from CRC cells. Paxillin proteins bound abundantly to both of the two substrate trapping mutants, but only minimally to the wild-type (WT) PTPRT protein and not at all to the control GST protein (Fig. 1A).  Fig. 1. Fig. 1. Paxillin is a direct substrate of PTPRT. (A) Paxillin is pulled down by substrate-trapping mutants of PTPRT. Colon cancer cell lysates were incubated with beads bound to the indicated GST-fusion proteins. Western blots were performed with an anti-paxillin ... The target site of PTPRT on paxillin, as predicted by our proteomic data (13), is the previously uninvestigated phospho-Y88 residue. To characterize phosphorylation at this tyrosine, we successfully generated an antibody specifically recognizing phospho-tyrosine 88 (pY88) paxillin as described in detail in Materials and Methods. This antibody is highly specific as it recognized a single pY88 paxillin band in lysates of parental CRC cells by Western blot analysis, but produced no signal in cell lysates of paxillin Y88F homozygous mutant cells (Fig. S2C). To test whether PTPRT directly dephosphorylates paxillin in vitro, we incubated phospho-paxillin immunoprecipitated from HEK 293T cells as a substrate with equal amounts of WT PTPRT, TRAP-2 mutant PTPRT, or GST protein, with or without the phosphatase inhibitor Na3VO4. Paxillin was dephosphorylated at the Y88 residue by WT PTPRT but not by the phosphatase-dead mutant (TRAP-2) or GST alone (Fig. 1B). This activity was also inhibited by Na3VO4, indicating that it was phosphatase-dependent (Fig. 1B). Interestingly, PTPRT failed to dephosphorylate paxillin at either the Y31 residue or the Y118 residue (Fig. S1B), suggesting that PTPRT is a paxillin pY88-specific phosphatase.  PTPRT Regulates Paxillin Y88 Phosphorylation in CRC Cells. Although Y88 paxillin phosphorylation has been reported in several studies (11, 18 –21), the biological significance of this phosphorylation and the extracellular cues that activate it remain to be determined. To address these issues, we used the CRC cell line DLD1 to test whether growth factors and cytokines could stimulate paxillin Y88 phosphorylation. As shown in Fig. S3A, PDGF-AA robustly activated this phosphorylation, whereas PDGF-AB, PDGF-BB, PDGF-CC, EGF, interleukin-6 (IL-6), and FGF-stimulated paxillin Y88 phosphorylation to varying, lesser extents. No stimulation was observed with VEGF. Furthermore, PDGF-AA activated paxillin Y88 phosphorylation in a variety of CRC cell lines (Fig. S3B). Thus, we chose PDGF-AA for in-depth study.  Given that PDGF plays direct roles in development of gastrointestinal villi (22) and in promoting the growth of CRC cells (23), it is important to determine whether PTPRT regulates paxillin Y88 phosphorylation in CRC cells activated by PDGF-AA. As shown in Fig. 1C, infection of DLD1 CRC cells with an adenovirus expressing PTPRT resulted in reduced pY88 paxillin levels in comparison with cells infected with a control GFP adenovirus at each of 45, 60, and 120 min after PDGF-AA stimulation. Similar results were also observed with HCT116 CRC cells (Fig. S3C). Interestingly, phosphorylation of paxillin Y31 and Y118 was not altered by PDGF-AA or PTPRT (Fig. 1C). Note that only the pY88 paxillin, not its total levels, was affected by PTPRT, thereby excluding protein degradation as a cause of these effects (Fig. 1C).  Generation of Paxillin Y88F Knock-In Colon Cancer Cells. To rigorously test whether regulation of paxillin Y88 phosphorylation is critical to colorectal tumorigenesis, we set out to engineer paxillin Y88F knock-in (KI) CRC lines. The adeno-associated virus (AAV) targeting system was used to generate the KI cell lines because of its high homologous recombination frequency in somatic cells (24, 25). We first chose to knock in the paxillin Y88F mutant allele into the human colon cancer cell line HCT116, because we have shown that paxillin Y88 phosphorylation could be stimulated by PDGF-AA in HCT116 cells and because HCT116 has been widely used for successful gene targeting by homologous recombination (26). The exon 3 sequences of the paxillin gene, which encodes the Y88 residue, were replaced by mutant alleles by homologous recombination (Fig. S2A). This exon was sequenced in the targeted clones and the parental cells. As expected, both WT paxillin alleles were replaced by the paxillin Y88F mutants in the homozygous KI clones (Fig. S2B). Furthermore, Western blot analyses showed that paxillin proteins were expressed in the homozygous KI (Y88F/Y88F) cells, but they remained unphosphorylated at residue 88 after PDGF-AA stimulation. Moreover, phosphorylation at Y88 was heavily induced in the parental cells, but it was attenuated in the heterozygous KI (WT/Y88F) cells in response to PDGF-AA stimulation (Fig. S2C). These data indicated that we had successfully engineered paxillin Y88F mutant cells. To ensure that what we observe with HCT116 cells can be generalized to other colon cancer cells, we used the same method to generate paxillin Y88F KI mutant DLD1 cells (Fig. S2C).  Paxillin Y88F Mutant Cells Exhibit Impaired Anchorage-Independent Growth. When grown under normal tissue culture conditions (McCoy's 5A supplemented with 10% FBS), there was no obvious proliferation difference between the heterozygous KI clones and the parental cells; however, the homozygous Y88F mutant cells grew a little slower (Fig. S4A). Consistently, we observed a subtle increase of apoptotic cell population in the homozygous Y88F mutant clones (Fig. S5). To test whether the paxillin Y88F mutant affects transformation, we performed colony formation and soft agar assays with the paxillin mutant KI cells. Compared to the parental cells, homozygous paxillin Y88F KI HCT116 cells displayed reduced colony-formation ability (Fig. S4B). In the case of DLD1 cells, however, only subtle differences were observed between parental and paxillin Y88F KI cells (Fig. S4B). In contrast, all homozygous Y88F KI clones formed ≈3-fold (P < 0.001) fewer foci in the soft agar assay than their WT counterparts in both HCT116 and DLD1 CRC cells (Fig. 2A). Interestingly, all of the heterozygous KI clones also displayed a significant (P < 0.05) reduction in the number of soft-agar foci with respect to WT cells (Fig. 2A), suggesting that paxillin Y88F protein may act as either a dominant-negative or haploid-insufficient mutant. Taken together, these results demonstrated that regulation of paxillin Y88 phosphorylation plays an important role in transformation.  Fig. 2. Fig. 2. Paxillin Y88F mutant CRC cells are less tumorigenic. (A) Anchorage-independent growth. CRC cells of the indicated clones were mixed in 0.4% soft agar and plated in six-well plates in triplicates. Cells were grown for 30 days, and colony foci were counted ... Paxillin Y88F Mutant Cells Fail to Form Xenograft Tumors in Nude Mice. To test tumorigenicity of the paxillin Y88F KI cells, paxillin Y88F homozygous clones, heterozygous clones, or parental HCT116 and DLD1 cells were injected s.c. into nude mice. Tumor formation and size were assessed by weekly caliper measurements. After 35 days of growth, WT cells formed tumors in all injected mice, yielding an average size of 2,300 mm3 in HCT116 cells and 1,800 mm3 in DLD1 cells. In contrast, homozygous HCT 116 KI cells failed to form tumors in any of the five mice (Fig. 2B and Fig. S4C), and only one of the five mice injected with DLD1 paxillin Y88F homozygous mutant cells formed a tumor, but it was tiny, only 18 mm3 (Fig. 2 B and C). Notably, the average tumor sizes of paxillin Y88F heterozygous KI clones were also significantly (P < 0.05) smaller than those formed by the parental cells (Fig. 2C).  Paxillin Y88F Mutant Cells Exhibit Reduced Cell Migration. It is well documented that paxillin is involved in cell adhesion and migration (4). The paxillin Y88F KI cells showed no defect in cell adhesion. To test the ability of paxillin Y88F KI cells to undergo migration, a Boyden chamber assay was employed. As shown in Fig. S6A, both HCT116 and DLD1 Y88F homozygous mutant cells exhibited a ≈4-fold decreased ability to migrate through a porous membrane coated with fibronectin in comparison with parental cells (P < 0.001). Consistent with this result, the paxillin Y88F homozygous mutant cells also migrated slower in wound healing assays (Fig. S6 B and C). Compared to parental cells, heterozygous paxillin KI cells also displayed a significant defect in cell migration in these assays (P < 0.05).  Paxillin Y88F Mutant Affects AKT, p130CAS, and SHP2 Signaling. We demonstrated that phosphorylation of the paxillin Y88 residue plays a critical role in colorectal tumorigenesis. To gain insights into the effects of this phosphorylation on downstream signaling, we examined how the paxillin Y88F KI affects phosphoyrlation of signaling molecules in CRC cells after PDGF-AA stimulation. It is well documented that PDGF receptors (PDGFRs), once they are engaged by their ligands, activate multiple well-characterized signaling pathways, including Ras-MAPK, PI3K-AKT, and PLC-γ (22). We tested the phosphorylation status of 27 sites on 17 proteins that could be potentially modulated by PDGF signaling (Table S1). In both parental HCT116 and DLD1 cells, PDGF-AA activated AKT phopshorylation at both threonine residue 308 and serine residue 473, two posttranslational modifications that are known to be critical for AKT activation (27). However, no activation of these AKT phosphorylations could be induced in the paxillin Y88F homozygous mutant CRC cells (Fig. 3). PDGFRs also crosstalk with integrins and modulate cell adhesion signaling (22). Among the cell adhesion molecules tested, PDGF-AA stimulated the phosphorylation of p130CAS at tyrosine 165 (Y165) (Fig. 3). This stimulation of pY165 p130CAS was also attenuated in paxillin Y88F mutant CRC cells. Lastly, phospho-Y542 of SHP2 could also be moderately stimulated by PDGF-AA and the levels of this phosphorylation were consistently lower in the paxillin Y88F mutant CRC cells than in the parental cells. The paxillin Y88F mutant did not affect the phosphorylation of MAP kinase (Thr-202/204), PLC-γ (Tyr783 and Ser1248), several STAT proteins (STAT1, STAT3, and STAT5), FAK, and CRKL in response to PDGF-AA in CRC cells (Fig. 3 and Table S1). These results suggest that phosphorylation of paxillin Y88 plays a role in transducing the PDGF-AA signal to pathways involving AKT, p130CAS, and SHP2.  Fig. 3. Fig. 3. Paxillin Y88F mutant affect cell signaling through altering phosphorylation of AKT, p130CAS, and SHP2. Parental (WT) and paxillin Y88F homozygous KI CRC cells were serum-starved for 16 h and stimulated with PDGF-AA for the indicated times. Western blot ... Paxillin Is an in Vivo Substrate of PTPRT. To determine whether paxillin is an in vivo substrate of PTPRT, we generated PTPRT knockout mice. Exon 22 of PTPRT, which encodes the phosphatase catalytic core motif of the first catalytic domain, was targeted (Fig. S7A). This strategy produced a truncated PTPRT protein devoid of the two phophatase domains in 129/Ola mouse embryonic stem (ES) cells. Three clones were aggregated with eight-cell stage embryos. Chimeras were mated with C57BL/6J females, and germ-line transmission of the disrupted PTPRT allele was achieved. Interbreeding of F1 PTPRT+/− mice yielded F2 offspring at the normal Mendelian ratio; a representative Southern blot of genomic DNA from PTPRT+/+, +/−, and −/− mice is shown in Fig. S7B. Furthermore, PTPRT−/− mice showed no obvious developmental abnormalities.  To test whether PTPRT regulates paxillin Y88 phosphorylation under in vivo physiological conditions, Western blots were performed with colon tissue lysates from WT and PTPRT−/− mice. As shown in Fig. 4A, pY88 paxillin, but not pY31 or pY118, was up-regulated in the knockout mouse colons in comparison with colons from the WT littermates, suggesting that PTPRT is a key regulator of paxillin Y88 phosphorylation in vivo.  Fig. 4. Fig. 4. Increased tyrosine phosphorylation of paxilin correlates with azoxymethane (AOM) induces colon tumors in PTPRT knockout mice. (A) PTPRT regulates pY88 paxillin in vivo. Colon lysates were made from PTPRT+/+ (WT) and PTPRT−/− mouse littermates. ... PTPRT Suppresses Azoxymethane-Induced Colon Tumors. Our biochemical and cellular data indicate that PTPRT normally acts as a tumor suppressor gene. To test this hypothesis in vivo, we bred the PTPRT knockout allele for more than 10 generations onto the C57BL/6J mouse strain, which has been shown to be highly resistant to colon tumor induced by azoxymethane (AOM) (28, 29). Intercrossing heterozygous PTPRT knockout mice generated littermates of PTPRT genotypes that were +/+ and −/−. Twenty littermates of each genotype were treated with six doses of AOM by i.p. injection. As reported previously, the +/+ mice were highly resistant to AOM-induced colon cancer and none developed any colon tumor (Fig. 4B). In contrast, 17 of the 20 PTPRT homozygous knockout mice developed visible colon tumors, averaging 1.5 tumors per mouse. Representative tumors that developed in a PTPRT−/− mouse colon are shown in Fig. 4C. Histological analyses indicated that those tumors were adenomas (Fig. 4D) and that the dysplastic cells were highly proliferative as indicated by Ki-67 immunohistochemistry (IHC) staining (Fig. 4E).  Paxillin Y88 Phosphorylation Is Up-Regulated in Human Colon Cancer Tissues. Our data demonstrated that PTPRT is a key regulator of murine colonic pY88 paxillin and that dephosphorylation of paxillin pY88 is strongly tumor suppressive. To explore the role of this pY88 paxillin pathway in human colon cancers, we performed IHC staining of formalin-fixed paraffin-embedded (FFPE) human colon carcinoma sections and their matched normal colon tissues with our anti-pY88 paxillin antibody. These tumors were all low-grade moderately differentiated adenocarcinomas. As shown in Fig. 5A Upper, pY88 paxilin was highly expressed in tumor epithelial cells but not in the matched normal colon epithelial cells. Interestingly, total paxillin levels were comparable between the tumor and normal cells (Fig. 5A Lower), indicating that paxillin Y88 phosphorylation was specifically up-regulated in human colon cancers. Of the 10 colon tumor and normal pairs examined, pY88 paxillin was up-regulated in 8 of the tumors but not the matched normal tissues (Fig. 5B). Staining of colon cancer cells could be completely blocked by competition with pY88 paxillin peptides, but not by competition with the corresponding nonphosphorlated peptides (Fig. S7C). In stroma, we noted some nonspecific staining that could not be competed by the pY88 paxillin peptides. Nonspecific staining that could not be competed was also noted in mouse epithelial cells, thus restricting IHC analysis to human samples.  Fig. 5. Fig. 5. Paxillin Y88 phosphorylation is up-regulated in human colon cancers. (A) Representative images of IHC staining of human colon carcinomas and matched normal colon tissues with the indicated antibodies. (Scale bar: 100 μm.) (B) Quantitative IHC ... Go to: DISCUSSION These results provide critical in vivo evidence that PTPRT normally functions as a tumor suppressor gene. Moreover, they establish paxillin as a functionally important direct substrate of PTPRT. Specifically, we showed that regulation of paxillin phosphorylation at the Y88 residue, the target site of PTPRT, plays an important role in CRC tumorigenesis and CRC cell migration, and that phosphorylation of the paxillinY88 residue affects signaling through AKT, p130CAS, and SHP2.  Although paxillin Y88 phosphorylation was observed more than a decade ago (18), it was thought that this was a minor phosphorylation site of SRC kinase (20). Phosphorylation of Y31 and Y118 residues on paxillin was recognized as the major tyrosine phosphorylation events (18 –20). Here, we demonstrated that pY88 paxillin was strongly induced by multiple growth factors and cytokines including PDGFs, EGF, FGF, and IL-6. Consistent with our observation that EGF activated paxillin Y88 phosphorylation in CRC cells, a recent proteomic study showed that EGF stimulates pY88 paxillin in a head and neck squamous carcinoma cell line (21). Among the tested growth factors, PDGF-AA activates robust paxillin Y88 phosphorylation in multiple CRC cell lines. In fact, stimulation of paxillin tyrosine phosphorylation by PDGF was first observed more than 15 years ago (30), but the identity of the pY sites remained elusive. Our data clearly demonstrated that PDGF-AA predominantly activated paxillin phosphorylation at Y88 residue but not the Y31 or Y118 residues (Fig. 2C). Although a body of literature demonstrated the involvement of paxillin Y31 and Y118 phosphorylation in cell adhesion (4), we did not observe any major defects of paxillin Y88F mutant CRC cells in cell adhesion. Our studies showed that the unphosphorylatable paxillin Y88F mutant CRC cells display significantly reduced anchorage-independent growth, although the proliferation rate of these mutant cells is only slightly decreased compared to their parental cells when grown on plastics. Surprisingly, both paxillin Y88F mutant HCT116 and DLD1 CRC cells largely failed to form xenograft tumors in nude mice, suggesting that paxillin Y88 phosphorylation plays a prominent role in tumor growth in the rich in vivo environment. All these results support the hypothesis that PTPRT-regulated paxillin Y88 phosphorylation is critical for colorectal tumorigenesis.  Increasing evidence suggests that receptor protein tyrosine phosphatases (RPTPs) play critical roles in tumorigenesis. Since our first identification of RPTPs, including PTPRT, PTPRF, and PTPRG as potential tumor suppressor genes, several laboratories recently discovered that PTPRD was mutationally inactivated in brain, colon, head and neck, lung, and skin cancers (31, 32). Interestingly, PTPRD also dephosphorylates STAT3 (33), another proven PTPRT substrate (13). This study suggests that PTPRT and PTPRD may have overlapping tumor suppression functions. The target site of PTPRT on STAT3 is the Y705 residue, and phosphorylation of Y705 STAT3 is a key event to its activation. Interestingly, neither STAT3 nor paxillin forms stable complexes with PTPRT. Besco and colleagues (34) found that PTPRT associated with adhesion molecules such as E-cadherin and different catenin isofroms. It is yet to be determined whether paxillin or STAT3 cross-talks with those adhesion proteins.  Finally, our findings are significant for human disease, as demonstrated by the up-regulation of pY88 paxillin in ≈80% of human colon carcinomas. Unfortunately, currently available PTPRT antibodies do not work for immnuohistochemistry, which prohibits us from determining whether there is inverse correlation between PTPRT levels and paxillin pY88 levels in human colon tumors. Even though PTPRT is mutated in ≈11% of colon cancers, epigenetic silence could be another mechanism by which tumors inactivate PTPRT, which may provide potential explanation to the up-regulation of paxillin pY88 in a majority of colon cancer specimens. These results have potential therapeutic and diagnostic implications. Given that paxillin Y88F mutant CRC cells largely failed to form xenograft tumors (Fig. 2), inhibition of the corresponding kinase(s) should mimic this mutation and, thus, lead to arrest of in vivo tumor growth. It will be of great interest to identify such kinase(s).  Although protein tyrosine kinases, such as JAK (Janus kinase), BCR-Abl and Src [1,2], have been long known to serve as oncogenes the role that their cellular counterpart – PTPs (protein tyrosine phosphatases) – could play in cancer progression had not been extensively explored. The PTP superfamily, which regulates a multitude of signalling pathways, is divided into either ‘receptor-like’ [RPTP (receptor protein tyrosine phsophatase)] or ‘non-transmembrane’ [3]. Recently, a high-throughput mutational analysis of all PTPs identified six PTPs to be mutated in colon cancer [4]. Among them, PTPRT (protein tyrosine phosphatase receptor-T), also known as PTPρ, is the most frequently mutated tyrosine phosphatase. In the present review, we focus on the physiological and pathological functions of PTPRT as well as the cellular pathways regulated by this phosphatase.  DOMAIN STRUCTURE OF PTPRT  PTPRT belongs to the type IIB RPTP subfamily that consists of PTPRK, PTPRM, PTPRT and PTPRU (protein tyrosine phosphatase receptor-K, -M, -T and -U; PTPRU is frequently referred to as PCP-2). All four type IIB RPTPs share common domain architecture: an extracellular domain, a transmembrane domain, a juxtamembrane region and two phosphatase domains (Figure 1). The extracellular domains of type II RPTPs have high-sequence identities [3]: all consist of an MAM (meprin/A5/PTP μ) domain, an Ig domain and four FNIII (fibronectin type III) repeats (Figure 1). The MAM domain is suggested to play a role in protein dimerization. The Ig domain is a disulfide structure that is found in many cell-surface proteins and has been shown to mediate homophilic and heterophilic interactions between cell adhesion molecules. The FNIII motif was originally identified in the extracellular matrix protein fibronectin and later found to be present in many cell adhesion molecules [3]. Intracellular domains include the juxtamembrane domain, which has similarity to cadherin domains [3,5], and two phosphatase domains, D1 and D2. The juxtamembrane domain was specifically identified as being responsible for many of the sequence differences between the four type IIb family members, providing specificity in signal transduction [5] and potentially needed for catalytic activity in some family members [6,7]. The phosphatase domain D1 is actively responsible for the phosphatase activity of type IIb members, while the ‘pseudophosphatase’ domain D2, although catalytically inactive, is thought to be important for regulation [6].  Figure 1 Download figureOpen in new tabDownload powerpoint Figure 1 Structure and mutation distribution of PTPRT Black arrows indicate missense mutations and red arrows indicate nonsense or frame-shift mutations. Boxes represent functional domains. Fn3, FNIII; PTPc D1, first phosphatase catalytic domain; PTPc D2, second phosphatase catalytic domain PTPRT AS A TUMOUR SUPPRESSOR  PTPRT is mutated in colon, lung, stomach and skin (melanoma) cancers. Several lines of genetic evidence suggest that PTPRT normally functions as a tumour suppressor gene: first, a significant fraction of the mutations found in tumour samples are nonsense and insertion and deletion mutations, resulting in premature truncation of PTPRT and therefore loss of its function [4]. Secondly, consistent with Knudson's Two-Hit Hypothesis for the inactivation of tumour suppressor activity [2,8,9], approximately one-third of the tumours with PTPRT mutations harbour two different mutations in one tumour [4]. Thirdly, many of the tumour-derived missense mutations in either the D1 or the D2 phosphatase domains lead to reduced phosphatase activity of PTPRT [4]; similarly, such mutations in the extracellular domain result in defective cell adhesion [10,11]. Fourthly, overexpression of PTPRT activity inhibited cell growth, acting as a putative tumour suppressor in cancer cell culture [4]. Most importantly, knockout mice lacking both alleles of PTPRT, grown in a strain normally resistant to AOM (azoxymethane)-induced colon cancer, became highly susceptible to AOM-induced adenomas (Figure 2) [12]. Another large-scale screen also independently identified PTPRT as a putative tumour suppressor: after mobilizing transposable elements in mouse somatic cells, the authors documented which insertion sites led to tumour formation, ultimately including PTPRT in its list [13].  Figure 2 Download figureOpen in new tabDownload powerpoint Figure 2 PTPRT-knockout mice are hypersensitive to AOM-induced colon tumours (A) Schematic of PTPRT targeting strategy. The exon 22, which encoded the phosphatase catalytic motif of PTPRT, was deleted. (B) Southern blotting of genomic DNA from mice with indicated PTPRT genotypes. Arrowhead indicates the targeted alleles. (C) Gross morphology of two tumours (arrowheads) in a PTPRT−/− mouse (right) compared with a colon of a PTPRT+/+ wild-type (WT) mouse. (D) Representative images of haematoxylin and eosin staining of AOM-induced tumours. (E). Numbers of colon tumours developed in each of the PTPRT+/+, +/− and −/− C57BL/J6 mice (n=20 in each group). Each circle, square or triangle represents one mouse. (C) and (E) are reproduced from [12] with permission. ROLE OF PTPRT IN CELL ADHESION  Similar to PTPRM and PTPRK, we demonstrated that the extracellular domain of PTPRT also mediated homophilic cell–cell adhesion [10]. This homophilic interaction is very specific, as the extracellular part of PTPRT does not interact with PTPRM, PTPRK or PTPRU [10,14]. The specificity of the homophilic cell–cell adhesion seems to be determined by the MAM and Ig domains, because chimaeric PTPRT that substituted its own MAM or Ig domains for those of PTPRU would not aggregate with cells expressing wild-type PTPRT [14]. Consistent with the notion that the MAM and Ig domains are critical for PTPRT's cell adhesion function, deletion mutants devoid of either domain abolish its ability to mediate cell aggregation [10]. Surprisingly, we found that the four FNIII domains of PTPRT are also required for its cell–cell adhesion function [11]. Importantly, all the tumour-derived mutations located in the extracellular domains of PTPRT show defects in cell–cell adhesion [10,11].  The crystal structures of PTPRM revealed some clues to how PTPRT mutations affect cell–cell adhesion, as these mutated residues in PTPRT were able to be correlated with key amino acids in PTPRM [15,16]. According to these data, some of these mutations were predicted to disturb the structure as a whole, affecting the overall folding patterns, the stability of the extracellular portion of the protein or the ability for the protein to correctly localize to the membrane [16]. However, one mutated residue in the Ig domain was thought to be specifically responsible for changing the capability for protein–protein interaction, underscoring the importance of this function in normal physiology [15]. Moreover, crystal structures also inform the importance of the ectodomain in dimerization as well as ensuring correct spacing between adjacent cells at adherens junctions so that interactions with substrates can occur properly [16].  Consistent with the role of PTPRT in cell–cell adhesion, Besco et al. [17] showed that PTPRT interacts with E-cadherin (endothelial cadherin). The authors then showed that this interaction leads to E-cadherin dephosphorylation, affecting the stability of junctional complexes [17].  PTPRT IN SIGNALLING PATHWAYS  Identification of the substrates of PTPRT is an important step to understand its tumour suppressor function. Since PTPRT's phosphatase activity is often attenuated in cancers, proteins that retain phosphorylated tyrosine residues when PTPRT is mutated in the crucial D1 and D2 domains may be involved in tumorigenesis. Studies to this end ultimately identified proteins that had been previously speculated to be proto-oncogenes, specifically paxillin and STAT3 (signal transducer and activator of transcription 3) [12,18]. In each case, a phosphopeptide profile was collected from wild-type cancer cells and cells from the same lineage that express either the intracellular fragment of PTPRT (which contains both the phosphatase domains needed for enzymatic activity) or the extracellular fragment (which lacks such activity). Those phosphopeptides that still had tyrosine phosphorylation in the absence of a functional copy of PTPRT warranted further review as possible substrates.  This finding of paxillin as a putative substrate for PTPRT complements the canonical function of the type IIB family members, which often interact with adherens junction proteins [17], as paxillin is known to be involved in cell–cell adhesion [19]. We have shown that PTPRT was responsible for dephosphorylating Tyr88 [12]. The importance of Tyr88 phosphorylation was especially seen in xenograft models, where cancer cells with mutated Tyr88 residues (a ‘knock-in’ cell line with phenylalanine substituted for Tyr88 – Y88F – prevented phosphorylation from occurring) were not able to grow xenografts [12]. Similarly, the Y88F mutants in cell lines experienced reduction in anchorage-independent growth and cell migration, as well as changes in Akt (also known as protein kinase B), SHP2 (short heterodimer partner 2) and p130CAS signalling due to alterations in their respective phosphorylation levels [12]. Consistent with the premise that phospho-Tyr88 paxillin plays an oncogenic role, this phosphorylated residue is up-regulated in majority of human colon cancer tissues in comparison with the matched adjacent normal tissues [12].  Similarly, PTPRT was also found to reverse Tyr705 phosphorylation on STAT3, a modification associated with STAT3 activation [18]. Substrate-trapping assays reinforced this finding. Then, we showed that PTPRT's phosphatase activity was crucial in regulating STAT3's translocation into the nucleus and activation of target genes, many of which are involved in cell survival and tumorigenesis [especially BCL-XL and SOCS3 (suppressor of cytokine signalling 3)] [18].  ROLE OF PTPRT IN NEUROLOGICAL DEVELOPMENT AND ITS REGULATION  McAndrew et al. [20] discovered PTPRT due in part to the role that many other RPTPs play in neural physiology [21]. The authors demonstrated that expression of PTPRT was largely limited to the brain and spinal cord. In situ hybridization indicated the hippocampus and dentate gyrus as specific areas of the brain, especially in post-migratory cells instead of the dividing and mobile neurons of prenatal nervous system development. Therefore, the authors then hypothesized that PTPRT might be more involved in synaptic regulation rather than nervous system growth and development. Lim et al. [22] delved into this speculation in their study about PTPRT and its regulation. They showed that PTPRT fosters synaptic formation by interacting with a series of synaptic-related proteins (including neuroligin family members and neurexin) and binding with other PTPRT molecules on other cells; both interactions occur via PTPRT's extracellular domains. This paper was also crucial as it explored PTPRT's regulation, as the authors demonstrate that this activity is attenuated by expression of Fyn kinase (a Src kinase family member), decreasing synapse formation. Specifically, Fyn phosphorylates PTPRT in its catalytic domain, causing inactivating interactions between PTPRT molecules on the same cell, a loss of phosphatase activity and ultimately an inability of PTPRT to interact with its target proteins.  OTHER TYPE IIB FAMILY MEMBERS AS TUMOUR SUPPRESSORS  PTPRT is not the only type IIB family member implicated as a tumour suppressor, as PTPRK, PTPRM and PTPRU are also implicated in tumorigenesis [9]. After first being associated with a known tumour suppressor region, loss of PTPRK function gradually became associated with a variety of cancers. PTPRK was first identified as a possible tumour suppressor in a study that mapped it to the long arm of the sixth chromosome, specifically at 6q22.2–q22.3, a region that had already been known to be deleted in a variety of neoplasms [23]. These data were followed by a microsatellite study of 6q22–6q23 in primary CNS (central nervous system) lymphomas, confirming that PTPRK was the responsible gene [24]. The authors in that paper further investigated the possibility of PTPRK acting as a tumour suppressor, demonstrating that mRNA transcripts were absent and protein expression levels were decreased in tumour samples versus control tissue [24]. Their results also had clinical significance in that decreased PTPRK expression correlated with decreased patient survival, furthering the case that PTPRK is a tumour suppressor [24]. Another study associated PTPRK with HL (Hodgkin Lymphoma): PTPRK [transcription of which depends on TGF-β (transforming growth factor-β)] antagonized the growth of HL cells positive for the Epstein–Barr virus, but this specific subset of HL overcomes PTPRK's inhibition by down-regulating TGF-β signalling [25]. Moreover, this tumour suppressor role can be extended to skin cancers, as transcriptional levels of PTPRK and PTPRU were decreased in melanoma tumour samples according to data generated from RT–PCR (reverse transcription–PCR) [26] and microarray [27].  Much work has been done to elucidate the role that PTPRM plays in suppressing GBM (glioblastoma multiforme). After identifying a decrease in PTPRM expression in GBM samples (a high-grade astrocytoma) versus low-grade astrocytomas, Burgoyne et al. [28] decided to investigate the migratory abilities of cells without functional PTPRM, since the classification of GBM as ‘high grade’ depends on the dispersal of its constituent cells. In a wound-scratch assay, cells without PTPRM expression due to shRNA (small-hairpin RNA) knockdown experienced greater migratory capability than cells that expressed PTPRM [28]. A complementary paper showed that, although overexpression of PTPRM was key in decreasing cellular migration, similar results were seen when PTPRM fragments were also knocked down [29]. Other RPTPs are known to be cleaved proteolytically, and the authors identified PTPRM cleavage products in GBM cell lines, one of which translocates to the nucleus; cleavage products were seen preferentially instead of full-length PTPRM in tumour samples [29]. Then, these fragments – still catalytically active – were shown to be important for GBM cell proliferation and movement [29]. Moreover, a subsequent study demonstrated that these fragments are useful diagnostically, as probes against the extracellular fragment were used to detect GBM tumours in vivo [30].  Although one of least studied phosphatases in the RPTP type IIB family, evidence suggests that PTPRU might also be a tumour suppressor. This RPTP is notable for its role in obstructing colon carcinoma development [31]. Particularly, PTPRU antagonizes β-catenin's transcriptional activation activity, preventing transcription of target genes. As a result, when expressed in a colon carcinoma cell line with increased β-catenin stability, PTPRU resulted in decreased cellular movement and replication, acting as a tumour suppressor. It is worth noting that PTPRU is also mutated in colon cancer tumour samples [32].  CONCLUSIONS  Recent studies have elaborated on the role that RPTP type IIB family members play in suppressing tumour growth, especially PTPRT. Although originally discovered as a primarily neurological protein, PTPRT has been shown to play integral roles in cell adhesion and intracellular signalling, due in part to extensive studies done on its extracellular and intracellular domains respectively. When this normal physiological role becomes abrogated, however, cellular transformation often occurs, demonstrating its importance as a tumour suppressor. Further studies need to be made on PTPRT and its type IIB family members to identify additional phosphatase targets to obtain a better understanding of progression to cancer.  : 3\n"
     ]
    }
   ],
   "source": [
    "from joblib import load\n",
    "pipes = load('pipeUSECASE.joblib')\n",
    "\n",
    "preds = pipes.predict(X_test)\n",
    "\n",
    "#print(\"results:\")\n",
    "#for (sample, pred) in zip(X_test, preds):\n",
    "#    print (sample, \":\", pred)\n",
    "\n",
    "#predicted = pipe.predict(X_test)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### 8. Test fitting "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Logistic Regression Accuracy:  0.0\n"
     ]
    }
   ],
   "source": [
    "from sklearn import metrics\n",
    "\n",
    "print(\"Logistic Regression Accuracy: \", metrics.accuracy_score(y_test, preds))\n",
    "#print(\"Logistic Regression Precision: \", metrics.precision_score(y_test, predicted))\n",
    "#print(\"Logistic Regression Recall: \", metrics.recall_score(y_test, predicted))"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "***Questions***:\n",
    "\n",
    "1. How this can be executed in HPC, e.g. training a new model for the corpus?\n",
    "2. Is there a maximum lenght for the spacy object nlp?\n",
    "3. Can we use the Noun Phrase to clean the text?"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.7.3"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
